# Gut microbiome of mealworms (Tenebrio molitor Larvae) show similar responses to polystyrene and corn straw diets

## Abstract

### Background
Some insects can degrade both natural and synthetic plastic polymers, their host and gut microbes play crucial roles in this process. However, there is still a scientific gap in understanding how the insect adapted to the polystyrene (PS) diet from natural feed. In this study, we analyzed diet consumption, gut microbiota responses, and metabolic pathways of Tenebrio molitor larvae exposed to PS and corn straw (CS).

### Results
T. molitor larvae were incubated under controlled conditions (25 ± 1°C, 75 ± 5% humidity) for 30 days by using PS foam with weight-, number-, and size-average molecular weight (Mw, Mn, and Mz) of 120.0, 73.2, and 150.7 kDa as a diet, respectively. The larvae exhibited lower PS consumption (32.5%) than CS (52.0%), and these diets had no adverse effects on their survival. The gut microbiota structures, metabolic pathways, and enzymatic profiles of PS- and CS-fed larvae showed similar responses. The gut microbiota of larvae analysis indicated Serratia sp., Staphylococcus sp., and Rhodococcus sp. were associated with both PS and CS diets. Metatranscriptomic analysis revealed that xenobiotics, aromatic compounds, and fatty acid degradation pathways were enriched in PS- and CS-fed groups; laccase-like multicopper oxidases, cytochrome P450, monooxygenase, superoxide, and dehydrogenase were involved in lignin and PS degradation. Furthermore, the upregulated gene lac640 in both PS- and CS-fed groups was overexpressed in E. coli and exhibited PS and lignin degradation ability.

### Conclusions
The high similarity of gut microbiomes adapted to biodegradation of PS and CS indicated the plastics-degrading ability of the T. molitor larvae originated through an ancient mechanism that degrades the natural lignocellulose.

### Keywords
Tenebrio molitor, Polystyrene, Corn straw, Microbial community, Metabolic pathway, Biodegradation

*Correspondence: Huwen Han hanhw@lzu.edu.cn Xiangkai Li xkli@lzu.edu.cn Full list of author information is available at the end of the article

# Graphical Abstract

![Graphical Abstract](image/graphical_abstract.png)

## Background

Plastic contamination is continuously increasing due to rapid urbanization and industrialization, and produced about 368 million tons of plastic as reported in 2019 [1]. The widespread usage, low recycling rates, and non-degradable nature of plastics have resulted in their accumulation in the environment [2]. Although mechanical, chemical, and physical techniques have been applied to the management of plastic waste, these methods are ineffective and can cause pollution through the emission of different gases and additive chemicals [3]. Therefore, biological technologies have been developed as an eco-friendly alternative for eliminating plastic waste. Several bacteria and fungi are capable of degrading plastic materials at low rates [4, 5]. Recently, numerous studies highlighted the role of some insects and their gut microbes in plastic degradation [6–8]. For example, yellow worms (Tenebrio molitor) [9], wax moths (Galleria mellonella) [10], and super worms (Zophobas atratus) [11] can degrade and metabolize polystyrene (PS), polyethylene (PE), and polyvinyl chloride (PVC) [12]. Other soil invertebrates, land snails (Achatina fulica) are capable of ingesting and degrading PS plastics [13], and gut bacteria isolated from earthworms (Lumbricus terrestris) can decay LDPE microplastics [14].

Insect's diet selection is a long process of adaptation, most insect diets are plants with lignocellulose as the main component [15]. Lignocellulose-feeding insects including termites, locusts and beetles have a natural microbial community in their gut acting as a lignin degrader [16, 17]. As a model insect for screening lignocellulose-degrading microorganisms, many pure cultured microbes, such as Bacteroides, Staphylococcus, Streptococcus, and Bacillus, have been isolated from termites [18, 19]. Carbohydrate-active enzymes (CAZymes) were reported to be produced by gut bacteria for lignocellulose


The degradation, thereby contributing to the growth of insects [20]. The diet of great wax moths (G. mellonella) and mealworms (T. molitor) typically contains wax and wheat bran, respectively (Biology: Mealworm-HandWiki) [21, 22], whereas mealworms can digest lignocellulose-rich crops, including rice straw, rice husk, and corn straw (CS) [23]. Intriguingly, these insects are also capable of ingesting and degrading synthetic plastic, and several plastic-degrading bacteria, such as Enterobacter asburiae, Exiguobacterium sp., and Bacillus sp., have been identified within their guts [7, 10]. The degradation of natural polymers by insects is a natural mechanism that has a long evolutionary history [24]; however, plastics were invented a century ago; this raises questions about how synthetic plastics are metabolized by insects.

Most plastics are derived from crude oil, and the chemical bonds connecting the monomers resemble those found in natural polymers [25, 26]. The fungi Trametes versicolor and Phanerochaete chrysosporium have been reported can degrade lignin by several extracellular enzymes such as lignin peroxidase (Lip), manganese peroxidase (Mn), and laccase (Lac) [27, 28]. Synthetic plastic PE, polypropylene (PP), and PS share similar chemical structures with lignin, such as aromatic rings and carbon skeletons [26]. These similarities make it possible for lignin-modifying enzymes to degrade these plastics [25, 29]. Some ligninolytic enzymes were produced by Bacillus sp. PE3 in the PE containing media [30]. Another study also found that the PS degrading enzyme, hydroquinone peroxidase, was secreted by lignin-degrading bacterium Azotobacter beijerinckii HM121 [31]. Moreover, the well-known lignocellulose-degrading fungi Aspergillus flavus were isolated from wax moth, and two laccase-like multicopper oxidases (LMCOs) were upregulated under plastic stress [32]. Thus, insect gut microbes may use these natural polymer-degrading mechanisms to break down synthetic plastic polymers as well.

Synthetic plastics and natural lignocellulosic polymers are resistant to the degradation process because of their long chain of inert C–C bonds [33]. The insect T. molitor larvae can digest synthetic plastics and natural lignocellulose polymers, their host enzymes and gut microbes play important roles in this process. Thus, we hypothesized that (1) the gut microbe responses and metabolic pathways of PS and CS degradation in larvae are similar against the natural polymer CS and the synthetic polymer PS; (2) some enzymes in the larval host and gut can degrade both PS and CS; and (3) the ability of mealworms to degrade plastics is attributed to their capacity to degrade lignin. In this study, the larvae were fed with PS and CS diets to evaluate their influences on survival and gut microbiota, and cabbage (CB) was used as a control due to the less lignocellulose component rather than wheat bran. The degradation intermediates of PS/ CS were detected and characterized via gas chromatography–mass spectrometry (GC–MS) and metabolomics analysis. The metabolic pathways and potential functional enzymes associated with PS and CS degradation were also analyzed by comparative metatranscriptomic sequencing. Furthermore, the differentially expressed genes (DEGs) related to PS and CS degradation were verified by real-time polymerase chain reaction (RT-PCR), and the highly expressed gene lac640 in both PS- and CS-fed group was overexpressed in E. coli to determine its PS and CS degradation ability.

Methods Test materials and mealworm sources Three different feedstocks, namely, cabbage (CB), PS foam, and CS, were used as a diet for mealworms. The PS foam was purchased from local suppliers (frozen sample packs). According to gel permeation chromatography (GPC) analysis [6], the weight- and number-average molecular weight of the PS was 120,000 Da and 73,178 Da (Additional file 1: S1, Table S1). CS was obtained from corn-cultivating farmlands in Yuzhong County (Lanzhou, China), and CB was purchased from a supermarket as a control. The main characteristics of the CB and CS were shown in Table S1. T. molitor larvae were obtained from Insect Breeding Plant (Lanzhou, China). The average weight and length of the larvae were 75–90 mg and 2 cm, respectively.

Mealworm survival and PS/CS consumption Primary tests were performed to observe the PS and CS consumption by larvae and their effects on larval survival accordingly [7]. T. molitor larvae (n=200) were incubated in a rectangular food-grade polypropylene container (L×W×D: 13×7×5 cm) under controlled conditions [25±1 °C, 75±5% humidity, and 16:8 (light/dark) photoperiod] with PS (2.0 g) or CS (2.0 g) as the sole diet. Unfed and CB-fed larvae were used as a control (Additional file 1: S1, Fig. S1). To minimize the effects of the previous diet, the larvae were fed CB for at least 3 days, then removed into new containers and kept starved for 48 h [9]. The larval survival rates (SRs), pupation rates (PRs), and the mass loss of all diets were measured for 30 days with an interval of 5 days. Dead larvae and molts were removed from the containers at each measurement point to prevent the dead worms from being eaten by the remaining larvae. Furthermore, the mass balances of PS and CS degradation were calculated accordingly [23, 34], details were shown in the Additional file 1: (S1, M1).

The effects of antibiotic on the PS, CS and CB consumption of larvae were tested using a combination of gentamicin, vancomycin, and ampicillin accordingly [8,

To eliminate the gut microbes of larvae, the above antibiotics were supplemented with the diet at a ratio of 5:3:3 (55 mg/g of bran food) fed to the larvae ($n$=300 per group, $n$=3) for 8 days, whereas a control group was fed with bran without antibiotics. The ten mealworms were randomly selected and disinfected with 75% ethanol for 1 min, then washed with sterile water. The gut samples were drawn out and put into the 2 mL tube with 1 mL PBS. After shaking in the vortex for 10 min, the bacte- rial suspension was diluted to $10^{-1}$, $10^{-2}$, $10^{-3}$ with PBS, coated on the plate, and cultured in the tryptic soy agar (TSA) medium for 24 h. The number of active gut bacteria colonies was counted at 0, 2, 4, 6, and 8 days, respectively [8]. For the uncultured bacteria, normal polymerase chain reaction (PCR, 27F: AGAGTTTGA TCCTGGCTCAG, 1492R: GGTACCTTGTACG ACTT) and real-time quantitative PCR (RT-qPCR, F: AGAGTTTGATCCTGGCTCAG; R: CTGCTGCCT CCCGTAGGAGT) were determined, respectively. Subsequently, the remaining antibiotic-treated mealworms were divided into three groups for feeding on PS, CS, and CB, whereas the untreated larvae were also fed the same feedstocks as the control. The SRs and consumption rates (CRs) of with or without antibiotics larvae fed with PS, CS and CB were measured after 5 days, all tests were performed in duplicate.

Chemical characterization of the frass of PS-/CS-fed larvae and comparative metabolomic analysis At the end of the 30-day test, frass samples of larvae from each group were collected and stored at $-80^{\circ}$C for analysis. Fourier transform infrared spectroscopy (FTIR) (Thermo Nicolet NEXUS 670 FTIR, USA) was performed to characterize major functional groups of PS/CS feedstocks (control) and frass from the PS-, CS-fed and unfed larvae in the range of 4000 $-$ 500 cm$^{-1}$. The samples were prepared accordingly [12]. The characterization of thermal variation between the control PS/CS samples and frass from the PS- and CS-fed larvae were performed by thermogravimetric analysis (TGA) using a TG analyzer (Linseis TGA PT1600, Germany). All samples (each group: 5 mg) for TGA were tested from room temperature to 600 $^{\circ}$C at a heating rate of 10 $^{\circ}$C/min, under a nitrogen atmosphere ($>99.9\%$) with a flow rate of 10 mL/min [36]. GPC (Waters 1515, U.S.A.) was used to characterize depolymerized polymer molecular weight (number- [Mn], weight- [Mw] and Size- [Mz] averaged molecular weight) in the frass according to the previous methods [36], details were shown in the Additional file 1: (S1, M2). The lignocellulose component of CB, CS feedstocks, and frass of CS-,CB-fed larvae was measured by using Van Soest methods (ANKOM220 Fiber analyzer) [37], as shown in Additional file 1: (S1, M3). Scanning electron microscopy (SEM) and SEM-energy dispersive spectrometry (SEM-EDS, Apreo S, ThermoFisher, U.S.A) were used for observing micro-morphological images of raw CS, PS feedstocks, and frass of PS-, CS-fed larvae.

GC-MS (ThermoFisher, San Jose, California, U.S.A.) was applied to detect the intermediates of CS and PS metabolism. All samples, including control PS, CS and CB feedstocks, intestinal and frass samples of PS-, CS-, CB-fed and unfed larvae, were pretreated with a slightly modified method as previously described [10, 38], details were presented in the Additional file 1: (S1, M4). To further identify the metabolites and metabolic pathways of PS and CS degradation in the larvae, comparative metabolomic analysis was conducted by liquid chromatography-tandem mass spectrometry (LC-MS/MS), as shown in Additional file 1: (S1, M5) [34, 39]. The metabolites by matching the exact m/z of the samples were annotated using the KEGG databases and human metabolome. The metabolites with $p$-value $<0.05$ and VIP $>1$ were considered significantly different.

Microbial community analysis To evaluate the differences in community structure, the gut microbial community of the CB-, CS-, and PS-fed larvae were analyzed. At the end of 30-day of feeding, fifteen larvae were randomly selected from the PS-, CS-, CB-fed groups and sterilized for 1 min with 75% ethanol, followed by washing twice with 0.9\% (w/v) sterile saline [7]. Next, the guts were taken out and placed in a 2 mL centrifuge tube. Total DNA was extracted using the SPINeasy DNA Kit for Feces (MP Biomedicals, LLC, Singapore) based on the protocols. The PCR amplicon sequencing was performed as previously [10, 40]. Briefly, the hypervariable regions V3/V4 of the 16S rRNA gene were chosen for the PCR amplicon sequencing with the primers 338F/806R (F: ACTCCTACGGGA GGCAGCAG, R: GGACTACHVGGGTWTCTAAT) by using PCR amplifier (ABI GeneAmp® 9700). The unqualified sequences and adapters were removed from the raw sequencing data, and high-quality data existed in pair-end reads using the Illumina Hiseq 6000 platform. Microbial community structure, alpha diversity, hierarchical clustering, and the relative abundance of differential species were performed on Majorbio Cloud Platform (https://cloud.majorbio.com/). All groups (CB, PS, CS) were analyzed in four replicates.

Metatranscriptome analysis RNA extraction, Illumina sequencing and assembly, and functional genes annotation were conducted accordingly [41], details were presented in the Additional file 1: (S1, M6 &M7). The DEGs of comparisons of PS vs. CK (CB),

and CS vs. CK groups were analyzed using the R package edgeR [42]. The Benjamini–Hochberg method was applied to adjust \( P \)-value of DEGs analysis with the false discovery rate (FDR) for multiple comparisons [43]. The selection criteria for DEGs were as follows: \( \left| \log _ { 2 } ^ { \mathrm { F C } } \right| \geq 1 \), FDR \(\leq 0.05\), and transcripts per million (TPM) \(\geq 50\) in at least one sample. The assembled sequences of DEGs were compared to the NCBI Non-redundant Protein (Nr) database using the BLASTP algorithm based on a threshold of E-value \(<10^{-5}\) for assigning predicted gene descriptions [44].

RT-PCR validation

The seventeen DEGs from the PS and CS groups metatranscriptomic sequencing were confirmed via RT-PCR. cDNA was synthesized from total RNA using a reverse-transcription kit (RR047A, Takara) based on the manufacturer's instructions. The primers for RT-PCR were designed by Primer3Plus (https://www.prime r3plus.com), as shown in Additional file 1: (S1, Table S2). RT-PCR was performed using TB Green Premix Ex Taq \(^{\text {TM }}\) II (RR820A, Takara) following the manufacturer's protocol. The ribosomal protein L27a of \( T\). molitor (TmL27a) was used as an endogenous gene [45]. RT-PCR included the following cycles: denaturation at \(95^{\circ} \mathrm{C}\) for 5 min, followed by 40 cycles of \(95^{\circ} \mathrm{C}\) for 15 s, \(60^{\circ} \mathrm{C}\) for 15 s, and \(72^{\circ} \mathrm{C}\) for 10 s. RT-PCR assays of each gene were performed in triplicate.

Expression, purification, and functional assay of potential PS-/CS-degrading enzyme

To find potential enzymes that can degrade both PS and CS, according to the metatranscriptomic and RT-PCR results, we selected lac640 (LMCOs, k97_48640 gene_4_1) that was highly upregulated in both PS- and CS-fed groups. Previous research also reported laccase can degrade lignin and up-regulated in plastic degradation [32, 46]. Therefore, the enzyme was heterogeneously expressed in \( E\). coli Rosetta (DE3) and purified as previously [46], details were shown in the Additional file 1: (S1, M8). The laccase activity of the protein Lac640 was assayed by detecting the oxidation of 2,2'-azino-bis (3-ethylbenzothiazoline-6-sulfonic acid) (ABTS) at 420 nm. The reaction volume of 3.2 mL included 3 mL of sodium acetate buffer (0.1 M) containing 10 mM \(\mathrm{CuSO}_{4}\), \(160 \mu \mathrm{L}\) of ABTS (20 mM), and \(40 \mu \mathrm{L}\) of Lac640 solution. The optimal enzyme activities were detected with different \(\mathrm{Cu}^{2+}\) concentrations (1-1000 mM), pH values (3-7), and temperatures \((30-80^{\circ} \mathrm{C})\). The kinetics of Lac640 was measured at different initial ABTS concentrations. The effects of Lac640 in degrading PS/CS were detected in a solution including \(0.1 \mathrm{M}\) sodium acetate buffer (10 mM \(\mathrm{CuSO}_{4}\)), enzyme solution (0.1 mg/mL), and PS/CS powder or film at \(40^{\circ} \mathrm{C}\) for 72 h. FTIR and SEM analyses were performed to observe the changes in surface morphology and functional groups between Lac640-treated and untreated PS/CS samples [47]. After completion of the reaction, all samples were successively washed with \(1 \%\) SDS, distilled water, and ethanol, then completely dried at \(60^{\circ} \mathrm{C}\) for FTIR analysis, or freeze-dried for SEM.

Statistical analysis

The mean \pm SD was used for all experimentally obtained data. Student's t-test was used for two-group comparisons; for more than two groups, the significance of differences was analyzed using one-way ANOVA with the post hoc Tukey test. Significance was set at \({ }^{*} P<0.05,{ }^{* * } P<0.01\), \(* * * P<0.001\), and \(* * * * P<0.0001\). The significance of differences \((P<0.05)\) of each set of gut microbiota among the larval groups were analyzed using ANOVA with Benjamini–Hochberg FDR multiple tests [43].

Results and discussion

Survival of \( T\). molitor larvae and consumption of PS and CS

The mealworms were fed with PS and CS feedstocks (Fig. 1A, B), and the larvae were developed into pupae during the 30 days of the experiment (Fig. 1C). For mealworms initially fed 2 g of PS and CS, the total mass loss of PS was \(0.67 \pm 0.02 \mathrm{~g}\) and the CRs was \(32.5 \pm 1.5 \%\), whereas the total mass loss of the CS was \(1.05 \pm 0.04 \mathrm{~g}\) and CRs was \(52.0 \pm 1.8 \%\) (Fig. 1D). The SRs of larvae fed with CB, PS, CS and unfed were \(93.3 \%, 84.4 \%, 86.8 \%\) and \(64.6 \%\), respectively (Fig. 1E); their corresponding PRs were \(9.8 \%, 3.7 \%, 4.8 \%\), and \(2.8 \%\) (Fig. 1F). The mass balance of PS and CS were estimated according to the CRs of feedstock, generated frass mass and frass extraction, and lignocellulose components in frass. The larvae were digested \(63.5 \%\) (PS removal rate) of consumed-PS feedstocks, while digested \(44.4 \%, 26.6 \%\), and \(33.3 \%\) of cellulose, hemicellulose, and lignin component of consumed-CS feedstocks, respectively (Additional file 1: S1, Table S3). The average weight of larvae was increased \(23.3 \%\) in the CB-fed group, while PS-, CS-fed and unfed larvae were decreased \(18.6 \%, 9.6 \%\), and \(26.8 \%\) of body weight, respectively (Additional file 1: S1, Fig. S2). GPC analysis showed that the Mw, Mn, and Mz of PS residues extracted from the PS-fed larvae frass were significantly reduced at a rate of \(21.0 \%, 19.2 \%\) and \(32.9 \%\) than the PS feedstocks (Fig. 1L). Specifically, Mw decreased from 120,000 to 94,466 Da, Mn decreased from 73,178 to 59,151 Da, and Mz decreased from 150,774 to 101,198 Da (Additional file 1: S1, Table S4).

The antibiotics test showed that gentamicin, vancomycin, and ampicillin had strong inhibition ability on the growth of larvae gut flora by making prominent halos whether used alone or in combination (Additional

# Figure 1

## Panels A-C
Images of exposure of the yellow worms to PS and CS. A: Day 1 vs Day 30, B: Day 1 vs Day 30, C: After 30 Days.

## Panels D-F
PS/CS consumption rates (D), survival rates (E), and pupation rates (F) of larvae fed with PS, CS, CB and unfed. D: Consumption rate, E: Survival rate, F: Pupation rate.

## Panel G
The gut bacteria of larvae after antibiotic treatment on TSA medium and electrophoretogram of conventional 16S rRNA PCR products. H: Total visible colony counts of larvae gut flora within 8 days of antibiotic treatment and I: the absolute quantification of 16S rRNA copies in guts of the control and antibiotic-treated groups.

## Panels J-L
Consumption and survival rates of antibiotics-treated and untreated larvae (control) fed with different feedstocks for 5 days. J: Consumption rate, K: Survival rate, L: The molecular changes (Mw, Mn and Mz) of PS residues extracted from frass of antibiotics-treated and untreated larvae fed PS vs. PS feedstock. Styrofoam (PS), Corn straw (CS), Cabbage (CB)


file 1: S1, Fig. S3). After 6-day of antibiotic treatment, there was no visible gut microbial colonies on TSA medium (Fig. 1G), the number of CFUs of the larval gut active flora was (0.1×10<sup>6</sup>) significantly decreased than control (4×10<sup>6</sup>) (Fig. 1H). The RT-qPCR results also showed that the copy number of gut microbes from the antibiotic-treated larvae (4×10<sup>3</sup>) was significantly decreased compared with the control (2.5×10<sup>5</sup>) (Fig. 1I). The PS/CS consumption rates of antibiotic-treated larvae were decreased 3.0% and 2.6% compared with the control, respectively (Fig. 1J). Similarly, the SRs of antibiotic-treated larvae fed with PS and CS were slightly decreased 6.8% and 3.0% than that of the control after 5 days (Fig. 1K). Antibiotic treatment had no significant impact on the SRs and CRs of CB-fed larvae. The GPC results showed that Mw, Mn and Mz values of PS residues extracted from frass of antibiotic-treated larvae fed with PS were also slightly decreased (Mw: 6065 Da, Mn: 5392 Da, Mz: 8339 Da) than raw PS feedstocks (Fig. 1L) (Additional file 1: S1, Table S4).

The PS consumption rate (32.5%) of this study was consistent with the range of rates (31%–41.5%) reported previously by other researchers [7, 9]. The CS consumption of larvae (52.0%) was lower than that in a previous study which was 90% [48], this disparity can be attributed to different lignocellulose contents or the hardness of CS materials. The SRs of PS- and CS-fed larvae were ~10% higher than that of the unfed group (P<0.01), but there was no significant difference between CB-, CS- and PS-fed groups (P>0.05) (Fig. 1E), which was also supported by previous studies [9, 48]. The number of pupae is widely used to evaluate the healthy development of larvae [36], the PRs of larvae fed PS and CS were higher than the unfed group, but lower than the CB-fed group, during the 30 days. The mass balance results revealed that the larvae cannot remove or digest all consumed-PS/CS feedstocks, and partially consumed-feedstocks excrete with feces, which was consistent with previous reports [34, 37, 48]. The CB feedstocks conversion rate by larvae was higher than PS and CS feedstocks, this resulting in an increased weight of CB-fed larvae than other diets groups. Except for CB-fed larvae, the PS-, CS-fed and unfed larvae lost their weights, the unfed group was the one with the higher weight loss (Additional file 1: S1, Fig. S2). In a previous study, the average weight of mealworms fed with only PS or PVC foam was also decreased after 31-day tests [12, 49], which was consistent with the present study. Besides, the mealworms fed with CS diet increased their body weights by 2.6% [48], but our studies observed that the CS-fed larvae lost weight, this may be attributed to the components and structure of CS materials. These findings imply that larvae can obtain some energy from PS and CS for survival, but lack sufficient nutrients for healthy growth when unfed or only fed PS and CS. Several studies have reported that the Mn and Mw of PS residues extracted from only PS-fed mealworm feces were lower than that of the raw PS feedstocks, and the reduced range of Mw and Mn were 7.5~38.9% and 8.75%~34.6%, respectively [7, 9, 34, 49, 50]. However, the variable reduction rate depends on the differential molecular weight of the raw PS materials. In our study, the Mw and Mn of PS feedstocks were 120.0 kDa and 73.2 kDa, and the reduced rate were 21% (25.1 kDa) and 19.2% (14.0 kDa), respectively. For the Mw and Mn of PS materials similar with our study, their reduction rates of Mn and Mw in the PS-fed larvae feces were 20.8% (124.2 kDa~98.3 kDa) and 20.2% (40. 4 kDa~32.3 kDa) [7]. Nonetheless, the depolymerization observed in PS-fed larvae was within the range of previous observations.

The antibiotic treatment impaired the ability to digest PS due to suppression of the larval gut microbiota, thereby decreasing the SRs of mealworms compared with that of the control group. Previous studies reported that the larval gut microbes depressed by antibiotic resulted inhibited PS degradation, but not LDPE plastic [7, 36]. Our findings observed the Mw, Mn and Mz of PS residues from antibiotic-treated larvae frass were slightly decreased compared with the raw PS foam, indicating that host enzymes also contributed in the PS depolymerization. This synergetic role of the mealworms host and its gut microbiome for plastic biodegradation has been proven [51]. However, there has no study on the role of larval gut microbiota in the CS degradation. Our antibiotics test showed that the degradation of CS was not significantly dependent on gut microbiota as PS foam, but antibiotic treatment had slight effect on CS consumption rates of larvae compared with the antibiotic-untreated larvae.

Depolymerization and comparative metabolomic analysis of PS and CS FTIR spectra of the frass from PS-fed larvae showed new functional groups of C=O (1700 cm<sup>-1</sup>) and C-O stretches (1050–1150 cm<sup>-1</sup>) (Fig. 2A). The peak of R-OH (2500–3500 cm<sup>-1</sup>) stretching of the hydrophobic group was broader than that in the control raw PS. Similar FTIR spectra appeared in the CS-fed larvae frass, including oxygen- and hydrophobic group associated peaks C-O (800–1250 cm<sup>-1</sup>), C=O stretches (1500–1800 cm<sup>-1</sup>), and alcohol groups (R-OH stretching, 3200–3700 cm<sup>-1</sup>) (Fig. 2B). TGA analysis indicated that the weight loss of control PS was 98.1% at 280 °C–450 °C, with the maximum decomposition rate at 368 °C (Fig. 2C); meanwhile, the frass from PS-fed larvae contained three decomposition stages, with a total of 70% weight loss occurring at 100 °C–500 °C.

The control CS sample had two decomposition stages: the weight loss in the first stage was 46.2% at 200 \textdegree C–310 \textdegree C, and 32.5% weight loss occurred at 310 \textdegree C–410 \textdegree C (Fig. 2D). In contrast, the frass from CS-fed larvae was decomposed in four stages, with corresponding weight loss of 19.5%, 21.4%, 11.8%, and 4.7% at 256 \textdegree C, 306 \textdegree C, 378 \textdegree C, and 559 \textdegree C, respectively. The SEM images revealed that the surface of CS/PS feedstocks and excreted feces by larvae fed with PS and CS were different, suggesting that the PS and CS were digested and converted into another monomers by larvae (Additional file 1: S1, Fig. S4). The EDS mapping showed the frass of PS- and CS-fed larvae containing C, O and N, indicating the consumed PS and CS by larvae were converted to carbon–oxygen substance (Additional file 1: S1, Fig. S4).

To determine the degradation intermediates of PS and CS, the gut and frass samples from the PS-, CS-, CB-fed and unfed larvae were profiled by GC–MS (Additional file 1: S1, Table S5). Long-chain fatty acids, such as undecanoic acid (C11H23O), oleic acid, 3-(octadecyloxy) propyl ester (C39H76O3), 3,6-octadecadiynoic acid (C19H30O2), and hexadecanoic acid (C18H36O2), were found in the frass or intestinal samples of PS-fed larvae (Fig. 2E), whereas the relative abundances of benzene, 1,3-dimethyl-(C8H10), cyclohexene (C6H12), and phthalic acid (C26H42O4) were decreased. Similarly, in the frass or intestinal samples of CS-fed larvae, hexadecanoic acid (C18H36O2), 9-octadecenoic acid (C19H36O2), and tetradecanoic acid (C15H30O2) were also found (Fig. 2F), along with decreases of complex long-chain hydrocarbons or ring structures such as eicosane, 2-methyl-(C21H44), cyclohexane (C6H12), and phenol,2,2-methylenebis (C23H32O2). In addition, the gut samples of CB-fed and unfed larvae presented some similar peaks, as well as the frass samples of CB-fed and unfed larvae; this may be due to gut tissue or frass characterization. Despite the similarities in the gut or frass samples of CB- and unfed, the results of the GC–MS analysis were distinct from those obtained from the PS- and CS-fed larvae samples (Additional file 1: S1, Fig. S5).

Comparative metabolomic analysis was performed to characterize differential metabolites of CB-, PS-, CS-fed and unfed larvae by using LC–MS/MS. In total, 2489 metabolites were identified in the gut of larvae fed with CB-, PS-, CS-fed and unfed. Most of the metabolites located in organic acids and derivatives (25.39%), lipids and lipid-like molecules (21.54%), organoheterocyclic compounds (13.218%) and benzenoids (11.089%) (Additional file 1: S1, Fig. S6A). PCA (Principal Component Analysis) analysis of gut samples metabolites of larvae fed with CB, PS, CS feedstocks and unfed were significantly distinct from each-other (Fig. 2G). The hierarchical clustered heat maps of significantly differential metabolites showed that the PS and CS clustered one small branch, indicating PS and CS metabolites expression patterns were similar (Fig. 2H-I). The volcano map analysis showed the metabolites of PS- and CS-fed larvae were differentially expressed compared with the control group CB-fed and unfed group (Additional file 1: S1, Fig. S6B). A total of 706, 655, 561 and 581 differential metabolites were detected in the comparisons between PS vs. CB, CS vs. CB, PS vs. unfed, CS vs. unfed, respectively (Additional file 1: S2 & S3). Among them, 321 and 296 metabolites were upregulated in PS- and CS-fed groups compared with the CB-fed group, while 154 and 182 metabolites were upregulated in PS- and CS-fed groups compared with the unfed group. The relative abundance of differentially expressed metabolites, such as 9,12-octadecadiynoic acid, 2-methylbenzoic acid, 2,4-dichlorobenzoic acid, 4-hydroxybenzaldehyde and homocitrate were higher in PS-fed larvae than another groups (Fig. 2J). The tetradecanedioic acid, 3-phenoxymethyl alcohol and D-arabinose were enriched in the CS-fed group, whereas the high relative abundance of pyruvate and 2-acetic acid in the CB or unfed group.

In previous studies, peaks of C=O (1700 cm\(^{-1}\) ) and C-O (1050–1200 cm\(^{-1}\) ) stretching appeared and R-OH (2600–3600 cm\(^{-1}\) ) stretching broadened in the FTIR

# Figure 2

## (See legend on previous page.)

### A
- **Graph Title:** Transmission (\%) vs. Wavelength (nm)
- **Description:** This graph shows the transmission of light through different materials at various wavelengths. The x-axis represents the wavelength in nanometers, and the y-axis represents the transmission percentage. The graph includes multiple curves representing different materials, with each curve showing the transmission pattern at different wavelengths.

### B
- **Graph Title:** Transmission (\%) vs. Wavelength (nm)
- **Description:** Similar to graph A, this graph also shows the transmission of light through different materials at various wavelengths. The x-axis represents the wavelength in nanometers, and the y-axis represents the transmission percentage. The graph includes multiple curves representing different materials, with each curve showing the transmission pattern at different wavelengths.

### C
- **Graph Title:** Weight (\%) vs. Temperature (°C)
- **Description:** This graph shows the weight percentage of a material as a function of temperature. The x-axis represents the temperature in degrees Celsius, and the y-axis represents the weight percentage. The graph includes multiple curves representing different materials, with each curve showing the weight percentage change at different temperatures.

### D
- **Graph Title:** Weight (\%) vs. Temperature (°C)
- **Description:** Similar to graph C, this graph also shows the weight percentage of a material as a function of temperature. The x-axis represents the temperature in degrees Celsius, and the y-axis represents the weight percentage. The graph includes multiple curves representing different materials, with each curve showing the weight percentage change at different temperatures.

### E
- **Graph Title:** Intensity (mV) vs. Retention time (min)
- **Description:** This graph shows the intensity of a signal as a function of retention time. The x-axis represents the retention time in minutes, and the y-axis represents the intensity in millivolts. The graph includes multiple peaks representing different retention times and intensities.

### F
- **Graph Title:** Intensity (mV) vs. Retention time (min)
- **Description:** Similar to graph E, this graph also shows the intensity of a signal as a function of retention time. The x-axis represents the retention time in minutes, and the y-axis represents the intensity in millivolts. The graph includes multiple peaks representing different retention times and intensities.

### G
- **Graph Title:** PCA analysis
- **Description:** This graph represents a principal component analysis (PCA) plot. The x-axis and y-axis represent the first two principal components (PC1 and PC2), respectively. The plot includes different colored dots representing different groups or samples. The PCA analysis helps in visualizing the relationships and differences between the groups.

### H
- **Graph Title:** Heatmap
- **Description:** This heatmap represents the abundance of different compounds in various groups. The x-axis represents the compounds, and the y-axis represents the groups. The color intensity indicates the abundance level of each compound in each group. The heatmap helps in identifying the compounds that are significantly different between the groups.

### I
- **Graph Title:** Heatmap
- **Description:** Similar to graph H, this heatmap represents the abundance of different compounds in various groups. The x-axis represents the compounds, and the y-axis represents the groups. The color intensity indicates the abundance level of each compound in each group. The heatmap helps in identifying the compounds that are significantly different between the groups.

### J
- **Graph Title:** Heatmap
- **Description:** This heatmap represents the abundance of different compounds in various groups. The x-axis represents the compounds, and the y-axis represents the groups. The color intensity indicates the abundance level of each compound in each group. The heatmap helps in identifying the compounds that are significantly different between the groups.


The Shannon index showed lower species richness of the gut microbiome of PS- and CS-fed larvae than in the CK (CB-fed larvae) group (Fig. 3A). A principal coordinate analysis (PCoA) based on the OTU revealed that the microbial composition of PS and CS group was distinct from that of CK group, but there was no clear distinction between PS and CS group (Fig. 3B). Furthermore, hierarchically clustered heatmap analyses of CB-, PS-, and CS-fed larvae at the genus level were cluster I comprised CK-4 and CK-3, whereas cluster II had two branches, one including control samples CK-2 and CK-1, another included CS and PS group (Fig. 3E). The gut microbial community of PS and CS group was significantly separated from that of the CB-fed group, but the PS and CS group was clustered into one branch. At the phylum level, the relative abundances of Firmicutes in PS (81.2%) and CS (73.5%) groups were greater than in the CK (56.1%), whereas the relative abundances of Proteobacteria of PS (18.7%) and CS (26.4%) groups were lower compared to the CK group (42.5%) (Fig. 3C). At the genus level, *Spiroplasma, Kluyvera*, and *Enterobacter* were the predominant genera among the larval gut microbiota in all diet groups (Fig. 3D). Unclassified *Enterobacteriaceae, Serratia*, and *Rubellimicrobium* were dominant in the PS-fed group, whereas *Staphylococcus* and *Weissella* were enriched in the CS-fed group. The relative abundances of *Spiroplasma* (PS: 74.5%, CS: 64.8%) and *Escherichia–Shigella* (PS: 4.8%, CS: 6.3%) increased in both PS- and CS-fed groups compared with those in the CK group (38.5%, 1.4%). To further evaluate the particular OTUs related to the PS and CS diets, differential abundance analyses of the PS vs. CB group and CS vs. CB groups were conducted (Fig. 3F, G). *Rubellimicrobium* sp. and *Pseudomonas* sp. exhibited significant differences in the gut of PS-fed larvae, whereas *Kluyvera* sp., *Weissella* sp., and *Bacillus* sp. maintained high abundances in the gut of CS-fed larvae. Furthermore, *Spiroplasma* sp., *Escherichia–Shigella* sp., unclassified *Enterobacteriaceae* sp., *Serratia* sp., *Staphylococcus* sp., *Prevotella* sp., and *Rhodococcus* sp. exhibited higher abundances in both PS- and CS-fed groups than in CB-fed larvae.

The alpha diversity results indicated that the richness of gut flora was dropped after PS and CS feeding compared with the CB diet, but species diversity was not significantly affected. The PCoA and hierarchical clustered heatmap analysis suggested that most members of the gut microbial community of larvae did not differ dramatically between the PS and CS diet groups, indicating that the gut microbial community structures of PS- and CS-fed larvae were similar. Previous studies reported that the phylum Firmicutes and the two genera *Spiroplasma* and *Escherichia–Shigella* presented at high abundance in the gut microbiota of PS-fed larvae, and *Spiroplasma* helped

# Gut Microbial Community Analysis of Larvae Fed with Different Diets

## Figure 3

### A
Shannon index of OTU level from larvae fed PS, CS, and CB (CK).

### B
PCoA analysis of gut microbial community (OTU level) from larvae fed PS, CS, and CB (CK).

### C-D
Relative abundances of larval gut microbes with different diets at the (C) phylum and (D) genus level.

### E
Hierarchical clustered heatmap analysis of larval gut microbes at the genus level.

### F, G
Differential abundance analyses at the OTU level [PS-fed vs. CK-fed (F); CS-fed vs. CK-fed (G)]. The direction of log2 (fold change) implies that the OTU is highly related to each diet.


Metatranscriptomic analysis of mealworms fed PS and CS Metatranscriptomic sequencing was applied to determine the metabolically active bacteria and the mechanism of plastic and lignocellulose degradation. A total of 87,468,710 clean reads with 11.3G clean bases, 89,290,414 clean reads with 11.5G clean bases, and 89,150,052 clean reads with 11.4G clean bases were obtained from the CK (CB), PS, and CS groups, respectively (Additional file 1: S1, Table S7). Overall, there were 28,413, 31,636, and 46,494 unigenes with N50 (N90) lengths of 1138 bp (399 bp), 1148 bp (392 bp), and 1421 bp (423 bp) in the CK, PS, and CS groups. There were most unigenes with lengths of 1-700 bp/701-1400 bp in all samples (Additional file 1: S1, Fig. S7). The relative abundance of larval gut microbiota was changed at the RNA level after feeding on the different diets. At the phylum level, PS and CS exposure altered the structure of larval gut microbiota (Additional file 1: S1, Fig. S8A). Specifically, the relative abundances of Tenericutes, Firmicutes, and Bacteroidetes were increased, whereas Proteobacteria and Cyanobacteria were decreased in the PS- and CS-fed groups, which was consistent with the previous study [40]. The genus level analysis showed that Spiroplasma was predominant in the gut of mealworms in all the diet groups (Additional file 1: S1, Fig. S8B). These results were almost consistent with the Illuma sequencing of 16S rRNA sequences (Fig. 3C, D). Streptococcus, Halobacterium, Bacillus, Pediococcus, Desulfurobacterium, and Aspergillus exhibited high abundance in the PS- and CS-fed larvae (Additional file 1: S1, Fig. S8C, D). These genera are known as plastic- or lignocellulose-degrading bacteria or fungi [23, 32, 49, 63], and these microbes might be related to PS and CS degradation in the larval gut.

In the comparative PS vs. CK and CS vs. CK metatranscriptomic libraries, 8082 and 8186 of DEGs were obtained, respectively. A total of 443 and 370 genes [log2(FC)≥1 and FDR≤0.05, P≤0.05] showed significant changes after feeding with PS and CS, respectively (Fig. 4A, B). Among them, 103 and 91 DEGs were upregulated, 340 and 279 genes were downregulated in the PS-/CS-fed groups, respectively, whereas 30 DEGs were upregulated in both PS- and CS-fed groups (Fig. 4D). A heatmap of the DEGs of the three different diet groups generated two clusters; PS and CS clustered together, whereas CK clustered alone, indicating that the gene responses of larvae were similar against the PS and CS diets (Fig. 4C). According to the KEGG gene annotation, the dominated relative abundances of DEGs in the PS- and CS-fed groups were mainly located in clusters of carbohydrates, lipids, cofactors and vitamins, biosynthesis of other secondary metabolites, and xenobiotic metabolism compared with the CK group in KEGG level 2 (Fig. 4E). Metabolism of regarding the annotated xenobiotic bio-degradation: benzoate degradation, styrene degradation, chloroalkane and chloroalkene degradation, metabolism of xenobiotics by cytochrome P450, and drug metabolism other enzymes; and global and overview maps: degradation of aromatic compounds were enriched in both PS- and CS-fed groups (Fig. 4F). The alcohol dehydrogenase [ADH; EC:1.2.1.10], cytochrome P450 family 6 [CYP6; EC:1.14.-.-], alkylglycerol monooxygenase [AGMO; EC:1.14.16.5], aldehyde dehydrogenase family [ALDH; EC:1.2.1.3], peroxidase [PO; EC:1.11.1.7], superoxide dismutase (Fe-Mn) family [SOD; EC:1.15.1.1], and long-chain-fatty-acid-CoA ligase [ACSL; EC:6.2.1.3] were found in the DEGs of PS-and CS-fed groups (Additional file 1: S1, Table S8). As typical lignocellulose- and plastic-degrading enzymes, CAZymes, such as laccase-like multicopper oxidase [AA1; EC 1.10.3.-], carboxyl esterase [CE10; EC 3.1.1.3], lipase [CE1; EC 3.1.1.-], endo-beta-1,3(4)-glucanase [GH12; EC 3.2.1.6], acetyl xylan esterase [CE1; EC 3.1.1.72], and feruloyl esterase [CE1; EC 3.1.1.73] were found in the PS-/CS-fed larval groups. Collectively, the genes involved in lignin and plastic degradation were classified into the AA or CE family, whereas hemicellulose and cellulose were classified into the GH/CE families (Additional file 1: S1, Fig. S9).

The corresponding bacterial species and their relative abundances for those enzyme sources in the 16S rRNA and metatranscriptomic taxonomy were analyzed by Nr database (Additional file 1: S1, Table S9 & Fig. S10). Although most enzymes were derived from the host genes, eight enzymes were resulted from larval gut

# Figure 4

## A, B Volcano map of the DEGs in response to PS and CS
The volcano plots show the differential expression of genes (DEGs) in response to PS and CS. The x-axis represents the log2 fold change, and the y-axis represents the p-value. The red dots represent upregulated genes, and the blue dots represent downregulated genes. The volcano plots indicate that there are more upregulated genes than downregulated genes in response to PS and CS.

## C Heat map of DEGs in response to three different diets (PS, CS and CB)
The heat map shows the expression levels of DEGs in response to three different diets (PS, CS, and CB). The colors represent the expression levels, with red indicating high expression and blue indicating low expression. The heat map indicates that there are differences in gene expression between the three diets.

## D Venn diagram of up- or down-regulated gene responses to PS or CS
The Venn diagram shows the overlap of upregulated and downregulated genes in response to PS and CS. The diagram indicates that there are more upregulated genes in response to PS than CS, and vice versa.

## E, F KEGG pathway enrichment analysis of DEGs in response to PS and CS at KEGG level 2 (E) and level 3 (F)
The KEGG pathway enrichment analysis shows the enrichment of DEGs in specific pathways. The x-axis represents the log2 fold change, and the y-axis represents the p-value. The blue bars represent the upregulated genes, and the red bars represent the downregulated genes. The analysis indicates that there are significant enrichments in specific pathways in response to PS and CS.


(Eb-G) can hydrolyze wheat straw or celluloses [66, 71]. Our transcriptomic analysis also found a series of oxidase (Peroxidase, LMCOs, SOD, CYP4/6, AGMO) and hydro-lase (Carboxylesterase, Lipases and AXE1) related to the degradation pathways of lignocellulose and PS.

Previous studies confirmed fatty acid-, drug-, and xenobiodegradation metabolism enriched in PS- and CS-fed mealworms [64], whereas drug metabolism-other enzymes, fatty acid biosynthesis and xenobiodegradation pathways enriched in the lignin degradation strain Aspergillus sydowii MS-19 [72]. Our findings showed that benzoate degradation, styrene degradation, chloroalkane and chloroalkene degradation, metabolism of xenobiotics by cytochrome P450, drug metabolism, and degradation of aromatic compounds were enriched in the DEGs of PS- and CS-fed mealworms. (Fig. 4F), which was in accordance with previous studies [64, 65, 72]. These results suggest that the metabolism of long-chain fatty acids, xenobiotics, styrene, and aromatic compounds degradation were related to the plastic and lignin degradation metabolism, and the above-mentioned enzymes may participate in these pathways.

Analysis of the potential metabolic pathways of PS/CS in T. molitor larvae

The PS and CS are metabolized by the larvae, and produce energy for growth and development, but how the larvae metabolize these refractory synthetic and natural polymers? this needs to further investigation. The metatranscriptomic analysis revealed that the DEGs responses, some metabolic pathways in KEGG, and enriched enzymes were similar in both PS- and CS-fed groups. Moreover, the GC-MS and metabolomic analysis revealed some similar intermediates in the gut or frass samples of PS- and CS-fed larvae. Based on present and previous results, we assumed that the degradation of these two polymers in the larvae have overlapped metabolic pathways, and therefore, we proposed the potential PS and CS metabolic pathways in the larvae (Fig. 5). The plastic- and lignocellulose-degrading enzymes from the DEGs were validated by RT-PCR, their expression levels in the PS-/CS-fed groups were calculated by the 2−ΔΔCt (Fig. 5A-D). The expression levels of CYP6/4, Lac640, and SOD were upregulated in the PS- and CS-fed groups, especially, the gene Lac640 were two times higher rather than other enzymes (Fig. 5A). These enzymes can oxidize the side chain or break C-C bonds of PS and lignin, and convert them into aromatic residues or monomers. The monomer of PS is styrene, and the styrene degradation pathway was found in the KEGG analysis of the PS-fed group (Additional file 1: S1, Fig. S11). After the oxidation of PS and lignin, the aromatic benzene ring was cleaved driven by alkylglycerol monooxygenase, aldehyde dehydrogenase, alcohol dehydrogenase, and peroxidase (Fig. 5B). Then, the products were further hydrolyzed to generate long-chain fatty acids via carboxylesterase (CES1), long-chin fatty acid-CoA ligase (ACSL) and lipase, and these enzymes were upregulated in both PS- and CS-fed groups (Fig. 5C). These fatty acids are subsequently stored in the host or undergo β-oxidation for the TCA cycle and produce metabolic energy. The metabolism of lignocellulose involves the lignin, hemicellulose, and cellulose degradation pathways. After lignin modification, exos of cellulose and hemicellulose fibers were converted into glucose or xylose by AXE1, FAEB, CES1, and Eb-G, respectively. The enzymes AXE1 and Eb-G exhibited higher expression levels in the CS-fed group than PS group (Fig. 5D), indicating that they are involved in cellulose and hemicellulose degradation.

The PS and lignin polymers are primarily comprise carbon chain and an aromatic ring side chain [26], and their degradations can be characterized by main-chain (C-C) or side-chain (benzene ring) oxidation, aromatic compound cleavage, and long-chain fatty acid degradation [35, 64, 65]. Previous studies found that the enzymes CYP4/6, monooxygenase, and aldehyde dehydrogenase were involved in PS metabolism of G. mellonella larvae [39], and reported CYP4/6 plays crucial role in the main-chain cleavage [73]. Another research reported that the PE plastic and beeswax degradation had similar metabolic approaches in G. mellonella due to the long-chain hydrocarbon as their major components [35]. Lac-, Mn-, and Lip-peroxidase can degrade lignin [67, 74]. In particular, some genes encoding LMCOs upregulated during plastic degradation [32], indicating a role in the cleavage of C-C bonds during the first step of PS and CS degradation. Representative enzymes for benzene ring opening, monooxygenase and peroxidase, and other key enzymes ADH, ALDH, and ACSL, were closely related to the process of long-chain fatty acid degradation [65, 75]. Our GC-MS and metabolomic analysis also detected some long-chain fatty acids, such as octadecenoic acid and hexadecenoic acid as the intermediates of PS/CS degradation in the larval gut or frass samples (Fig. 2E, F, J). Metatranscriptomics analysis, RT-PCR, GC-MS, and metabolomic analysis further suggested that the degradation pathways between PS and lignin are similar due to the analogous C-C backbone.

Laccase enzyme activity and PS-/CS-degrading ability of Lac640

The metatranscriptomic data indicated that lac640 encodes LMCOs, which belong to the AA1 family based on the CAZy database annotation. The protein Lac640 was highly upregulated in the PS- and CS-fed larvae groups (Fig. 5A), its corresponding bacterial

# Figure 5

A proposed metabolic pathways of lignocellulose (CS) and PS plastic degradation in mealworms, and validation of the expression levels of related PS- and CS-degrading enzymes by RT-PCR (A-D). The 60S ribosomal protein 27a (Tml27a) of T. molitor was used as a reference gene; expression levels were calculated by the \(2^{-\Delta \Delta C t}\)

## Figure 5 Description

### A
- **Title:** Relative expression level of PS and CS-degrading enzymes
- **X-axis:** Enzymes (CYP4, SOD, LMCO-1, LMCO-2, Lac640)
- **Y-axis:** Relative expression level
- **Legend:** Red (PS), Blue (CS)
- **Data Points:** Bars represent the mean \pm standard deviation of three replicates

### B
- **Title:** Relative expression level of PS and CS-degrading enzymes
- **X-axis:** Enzymes (PO, AGMO, ADH, ALDH)
- **Y-axis:** Relative expression level
- **Legend:** Red (PS), Blue (CS)
- **Data Points:** Bars represent the mean \pm standard deviation of three replicates

### C
- **Title:** Relative expression level of PS and CS-degrading enzymes
- **X-axis:** Enzymes (ACSL, Lip, CESI, ACOX1)
- **Y-axis:** Relative expression level
- **Legend:** Red (PS), Blue (CS)
- **Data Points:** Bars represent the mean \pm standard deviation of three replicates

### D
- **Title:** Relative expression level of PS and CS-degrading enzymes
- **X-axis:** Enzymes (AXE1, FAEB, CESI)
- **Y-axis:** Relative expression level
- **Legend:** Red (PS), Blue (CS)
- **Data Points:** Bars represent the mean \pm standard deviation of three replicates

## Figure 5 Explanation

The figure illustrates the proposed metabolic pathways of lignocellulose (CS) and PS plastic degradation in mealworms. The pathways are divided into several stages, including the degradation of lignocellulose, the formation of aromatic monomers, ring opening, the formation of long-chain fatty acids, and the β-oxidation process. The figure also shows the relative expression levels of related PS- and CS-degrading enzymes in mealworms, validated by RT-PCR. The 60S ribosomal protein 27a (Tml27a) of T. molitor was used as a reference gene, and the expression levels were calculated by the \(2^{-\Delta \Delta C t}\) method.


![Figure 6](https://i.imgur.com/3Q5Q5Q5.png)

Fig. 6 A Phylogenetic tree of Lac640 protein. B SDS-PAGE analysis of Lac640. Lane M, molecular size markers; lane Lac640, purified enzyme. C Laccase activity of Lac640 in the different concentrations of Cu²⁺. D Kinetic analysis of Lac640 at different concentrations of ABTS. E, G FTIR spectra for PS/CS samples treated with or without Lac640 enzyme; F, H The surface morphology of PS/CS treated with or without Lac640 enzyme


## Supplementary Information

The online version contains supplementary material available at [https://doi.org/10.1186/s40168-023-01550-w](https://doi.org/10.1186/s40168-023-01550-w).

### Additional file 1.
### Additional file 2.
### Additional file 3.

## Acknowledgements

We thank the School of Life Sciences, Lanzhou University, for providing instruments and equipment. We also want to thank the Electron Microscopy Center of Lanzhou University for helping with the microscopy and microscopic analysis of our specimens.

## Authors' contributions

Tursunay Mamtimin: Experiment setup, Data analysis, Writing-original draft, and editing. Huawen Han: Investigation, Supervision, Writing-review & editing. Aman Khan: Methodology, Writing-review & editing. Pengya Feng, Qing Zhang, Xiaobiao Ma, Yitian Fang: Conceptualization, Data curation, Project administration. Pu Liu, Saurabh Kulshresthac, Toshiro Shigaki: Conceptualization, Supervision, Writing-review & editing. Xiangkai Li: Conceptualization, Supervision, Validation, Writing-review & editing, Funding acquisition. The author(s) read and approved the final manuscript.

## Funding

This work was supported by the National Natural Science Foundation of China (No. 32070117, 32200080, and 31870082), International Science and Technology Cooperation Project of Gansu Province (No. 2021–0204-GHC-0019), and Gansu Science and Technology Association Project (No. GXX20210506-16–5).

## Availability of data and materials

The raw data on microbial diversity, metatranscriptomic sequences of *T. molitor*, and lac640 have been submitted to NCBI, with accession numbers PRJNA857507, PRJNA858987, and OP020932, respectively.

## Declarations

### Ethics approval and consent to participate

Not applicable.

### Consent for publication

Not applicable.

### Competing interests

The authors declare that there are no conflicts of interest associated with the publication of this paper.

### Author details

1. Ministry of Education Key Laboratory of Cell Activities and Stress Adaptations, School of Life Science, Lanzhou University, Lanzhou, China.
2. State Key Laboratory of Grassland Agro-Ecosystems, Center for Grassland Microbiome, Lanzhou University, Lanzhou, China.
3. State Key Laboratory of Microbial Metabolism, School of Life Sciences and Biotechnology, Shanghai Jiao Tong University, Shanghai, China.
4. Faculty of Applied Sciences and Biotechnology, Shoolini University of Biotechnology and Management, Chandigarh, India.
5. School of Life Sciences, Lanzhou University, Lanzhou, China.
6. School of Life Sciences, Lanzhou University, Lanzhou, China.
7. School of Life Sciences, Lanzhou University, Lanzhou, China.
8. School of Life Sciences, Lanzhou University, Lanzhou, China.
9. School of Life Sciences, Lanzhou University, Lanzhou, China.
10. School of Life Sciences, Lanzhou University, Lanzhou, China.
11. School of Life Sciences, Lanzhou University, Lanzhou, China.
12. School of Life Sciences, Lanzhou University, Lanzhou, China.
13. School of Life Sciences, Lanzhou University, Lanzhou, China.
14. School of Life Sciences, Lanzhou University, Lanzhou, China.
15. School of Life Sciences, Lanzhou University, Lanzhou, China.
16. School of Life Sciences, Lanzhou University, Lanzhou, China.
17. School of Life Sciences, Lanzhou University, Lanzhou, China.
18. School of Life Sciences, Lanzhou University, Lanzhou, China.
19. School of Life Sciences, Lanzhou University, Lanzhou, China.
20. School of Life Sciences, Lanzhou University, Lanzhou, China.
21. School of Life Sciences, Lanzhou University, Lanzhou, China.
22. School of Life Sciences, Lanzhou University, Lanzhou, China.
23. School of Life Sciences, Lanzhou University, Lanzhou, China.
24. School of Life Sciences, Lanzhou University, Lanzhou, China.
25. School of Life Sciences, Lanzhou University, Lanzhou, China.
26. School of Life Sciences, Lanzhou University, Lanzhou, China.
27. School of Life Sciences, Lanzhou University, Lanzhou, China.
28. School of Life Sciences, Lanzhou University, Lanzhou, China.
29. School of Life Sciences, Lanzhou University, Lanzhou, China.
30. School of Life Sciences, Lanzhou University, Lanzhou, China.
31. School of Life Sciences, Lanzhou University, Lanzhou, China.
32. School of Life Sciences, Lanzhou University, Lanzhou, China.
33. School of Life Sciences, Lanzhou University, Lanzhou, China.
34. School of Life Sciences, Lanzhou University, Lanzhou, China.
35. School of Life Sciences, Lanzhou University, Lanzhou, China.
36. School of Life Sciences, Lanzhou University, Lanzhou, China.
37. School of Life Sciences, Lanzhou University, Lanzhou, China.
38. School of Life Sciences, Lanzhou University, Lanzhou, China.
39. School of Life Sciences, Lanzhou University, Lanzhou, China.
40. School of Life Sciences, Lanzhou University, Lanzhou, China.
41. School of Life Sciences, Lanzhou University, Lanzhou, China.
42. School of Life Sciences, Lanzhou University, Lanzhou, China.
43. School of Life Sciences, Lanzhou University, Lanzhou, China.
44. School of Life Sciences, Lanzhou University, Lanzhou, China.
45. School of Life Sciences, Lanzhou University, Lanzhou, China.
46. School of Life Sciences, Lanzhou University, Lanzhou, China.
47. School of Life Sciences, Lanzhou University, Lanzhou, China.
48. School of Life Sciences, Lanzhou University, Lanzhou, China.
49. School of Life Sciences, Lanzhou University, Lanzhou, China.
50. School of Life Sciences, Lanzhou University, Lanzhou, China.
51. School of Life Sciences, Lanzhou University, Lanzhou, China.
52. School of Life Sciences, Lanzhou University, Lanzhou, China.
53. School of Life Sciences, Lanzhou University, Lanzhou, China.
54. School of Life Sciences, Lanzhou University, Lanzhou, China.
55. School of Life Sciences, Lanzhou University, Lanzhou, China.
56. School of Life Sciences, Lanzhou University, Lanzhou, China.
57. School of Life Sciences, Lanzhou University, Lanzhou, China.
58. School of Life Sciences, Lanzhou University, Lanzhou, China.
59. School of Life Sciences, Lanzhou University, Lanzhou, China.
60. School of Life Sciences, Lanzhou University, Lanzhou, China.
61. School of Life Sciences, Lanzhou University, Lanzhou, China.
62. School of Life Sciences, Lanzhou University, Lanzhou, China.
63. School of Life Sciences, Lanzhou University, Lanzhou, China.
64. School of Life Sciences, Lanzhou University, Lanzhou, China.
65. School of Life Sciences, Lanzhou University, Lanzhou, China.
66. School of Life Sciences, Lanzhou University, Lanzhou, China.
67. School of Life Sciences, Lanzhou University, Lanzhou, China.
68. School of Life Sciences, Lanzhou University, Lanzhou, China.
69. School of Life Sciences, Lanzhou University, Lanzhou, China.
70. School of Life Sciences, Lanzhou University, Lanzhou, China.
71. School of Life Sciences, Lanzhou University, Lanzhou, China.
72. School of Life Sciences, Lanzhou University, Lanzhou, China.
73. School of Life Sciences, Lanzhou University, Lanzhou, China.
74. School of Life Sciences, Lanzhou University, Lanzhou, China.
75. School of Life Sciences, Lanzhou University, Lanzhou, China.
76. School of Life Sciences, Lanzhou University, Lanz

University, Solan, India. \({ }^{2}\) Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo, Japan.

Received: 3 November 2022 Accepted: 16 April 2023 Published online: 05 May 2023

\section*{References}

1. Tiess, Global production volume of thermoplastics by type 2020-2050. 2021. <https://www.statista.com/statistics/1192886/thermoplastics-production-volume-by-type-globally/>. Accessed 12 Aug 2022.

2. Sharma S, Basu S, Shetti NP, Nadagouda MN, Aminabhavi TM. Microplastics in the environment: occurrence, perils, and eradication. Chem Eng J. 2021;408:127317. <https://doi.org/10.1016/j.cej.2021.127317>

3. Bahl S, Dolma J, Singh JJ, Sehgal S. Biodegradation of plastics: a state of the art review, materials today. Proceedings. 2021;39:31-34. <https://doi.org/10.1016/j.matpr.2020.06.096>

4. A'Paço A, Duarte K, da Costa JP, Santos PS, Pereira R, Pereira ME, Freitas AC, Duarte AC, Rocha-Santos TA. Biodegradation of polyethylene microplastics by the marine fungus Zalerion maritimum. Sci Total Environ. 2017;586:10-15. <https://doi.org/10.1016/j.scitotenv.2017.02.017>

5. Jadau JS, Bansal S, Sonthalia A, Rai AK, Singh SP. Biodegradation of plastics for sustainable environment. Bioresource Technol. 2022;347:126697. <https://doi.org/10.1016/j.biortech.2022.126697>

6. Yang J, Yang Y, Wu W-M, Zhao J, Jiang L. Evidence of polyethylene biodegradation by Bacterial Strains from the guts of plastic-eating waxworms. Environ Sci Technol. 2014;48(23):13776-84. <https://doi.org/10.1021/es504038a>

7. Yang Y, Yang J, Wu WM, Zhao J, Song Y, Gao L, Yang R, Jiang L. Biodegradation and mineralization of polystyrene by plastic-eating mealworms: part 2 role of gut microorganisms. Environ Sci Technol. 2015;49(20):12087-93. <https://doi.org/10.1021/acs.est.5b02663>

8. Yang Y, Wang J, Xia M. Biodegradation and mineralization of polystyrene by plastic-eating superworms Zophobas atratus. Sci Total Environ. 2020;708:135233. <https://doi.org/10.1016/j.scitotenv.2019.135233>

9. Brandon AM, Gao S-H, Tian R, Ning D, Yang S-S, Zhou J, Wu W-M, Criddle CS. Biodegradation of polyethylene and plastic mixtures in mealworms (Larvae of Tenebrio molitor) and effects on the gut microbiome. Environ Sci Technol. 2018;52(15):6526-33. <https://doi.org/10.1021/acs.est.8b02301>

10. Lou Y, Ekaterina P, Yang SS, Lu B, Liu B, Ren N, Corvini PF, Xing D. Biodegradation of polyethylene and polystyrene by greater wax moth larvae (Galleria mellonella L.) and the effect of co-diet supplementation on the core gut microbiome. Environ Sci Technol. 2020;54(5):2821-2831. <https://doi.org/10.1021/acs.est.9b07044>

11. Peng BY, Li Y, Fan R, Chen Z, Chen J, Brandon AM, Criddle CS, Zhang Y, Wu WM. Biodegradation of low-density polyethylene and polystyrene in superworms, larvae of Zophobas atratus (Coleoptera: Tenebrionidae): broad and limited extent depolymerization. Environ Pollut. 2020;266:115206. <https://doi.org/10.1016/j.envpol.2020.115206>

12. Peng BY, Chen Z, Chen J, Yu H, Zhou X, Criddle CS, Wu WM, Zhang Y. Biodegradation of Polyvinyl Chloride (PVC) in Tenebrio molitor (Coleoptera: Tenebrionidae) larvae. Environ Int. 2020;145:106106. <https://doi.org/10.1016/j.envint.2020.106106>

13. Song Y, Qiu R, Hu J, Li X, Zhang X, Chen Y, Wu WM, He D. Biodegradation and disintegration of expanded polystyrene by land snails Achatina fulica. Sci Total Environ. 2020;746:141289. <https://doi.org/10.1016/j.scitotenv.2020.141289>

14. Lwanga EH, Thapa B, Yang X, Gertsen H, Salänki T, Geissen V, Garbeva P. Decay of low-density polyethylene by bacteria extracted from earthworm's guts: a potential for soil restoration. Sci Total Environ. 2018;624:753-757. <https://doi.org/10.1016/j.scitotenv.2017.12.144>

15. Xie S, Lan Y, Sun C, Shao Y. Insect microbial symbionts as a novel source for biotechnology. World J Microbiol Biotechnol. 2019;35(2):25. <https://doi.org/10.1007/s11274-019-2599-8>

16. G. Handique, A. Phukan, B. Bhattacharyya, A.A. Baruah, S.W. Rahman, R. Baruah. Characterization of cellulose degrading bacteria from the larval gut of the white grub beetle Lepidiota mansueta (Coleoptera: Scarabaei- dae). Arch Insect Biochem Physiol. 2017;94(2). <https://doi.org/10.1002/arch.21370>

17. Ceja-Navarro JA, Karaoz U, Bill M, Hao Z, White RA 3rd, Arellano A, Raman- culova L, Filley TR, Berry TD, Conrad ME, Blackwell M, Nicora CD, Kim YM, Reardon PN, Lipton MS, Adkins JN, Pett-Ridge J, Brodie EL. Gut anatomical properties and microbial functional assembly promote lignocellulose deconstruction and colony subsistence of a wood-feeding beetle. Nat Microbiol. 2019;4(5):864-75. <https://doi.org/10.1038/s41564-019-0384-y>

18. Tsegaye B, Balomajumder C, Roy P. Isolation and characterization of novel lignolytic, cellulolytic, and hemicellulolytic bacteria from wood-feeding termite cryptodermes brevis. Int Microbiol. 2019;22(1):29-39. <https://doi.org/10.1007/s11238-018-0024-z>

19. Ni J, Tokuda G. Lignocellulose-degrading enzymes from termites and their symbiotic microbiota. Biotechnol Adv. 2013;31

36. Yang L, Gao J, Liu Y, Zhuang G, Peng X, Wu WM, Zhuang X. Biodegradation of expanded polystyrene and low-density polyethylene foams in larvae of Tenebrio molitor Linnaeus (Coleoptera: Tenebrionidae): broad versus limited extent depolymerization and microbe-dependence versus independence. Chemosphere. 2021;262:127818. https://doi.org/10.1016/j.chemosphere.2020.127818.

37. Yang SS, Kang JH, Xie TR, He L, Xing DF, Ren NQ, Ho SH, Wu WM. Generation of high-efficient biochar for dye adsorption using frass of yellow mealworms (larvae of Tenebrio molitor Linnaeus) fed with wheat straw for insect biomass production. J Clean Prod. 2019;227:33–47. https://doi.org/10.1016/j.jclepro.2019.04.005.

38. Cui T, Yuan B, Guo H, Tian H, Wang W, Ma Y, Li C, Fei Q. Enhanced lignin biodegradation by consortium of white rot fungi: microbial synergistic effects and product mapping. Biotechnol Biofuels. 2021;14(1):162. https://doi.org/10.1186/s13068-021-02011-y.

39. Wang S, Shi W, Huang Z, Zhou N, Xie Y, Tang Y, Hu F, Liu G, Zheng H. Complete digestion/biodegradation of polystyrene microplastics by greater wax moth (Galleria mellonella) larvae: Direct in vivo evidence, gut microbiota independence, and potential metabolic pathways. J Hazard Mat. 2022;423:127213. https://doi.org/10.1016/j.jhazmat.2021.127213.

40. Lou Y, Li Y, Lu B, Liu Q, Yang SS, Liu B, Ren N, Wu WM, Xing D. Response of the yellow mealworm (Tenebrio molitor) gut microbiome to diet shifts during polystyrene and polyethylene biodegradation. J Hazard Mat. 2021;416:126222. https://doi.org/10.1016/j.jhazmat.2021.126222.

41. Pei Y, Tao C, Ling Z, Yu Z, Ji J, Khan A, Mamtimin T, Liu P, Li X. Exploring novel Cr(VI) remediation genes for Cr(VI)-contaminated industrial waste water treatment by comparative metatranscriptomics and metagenomics. Sci Total Environ. 2020;742:140435. https://doi.org/10.1016/j.scitotenv.2020.140435.

42. Robinson MD, McCarthy DJ, Smyth GK. edgeR: A Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics. 2009;26(1):139–40. https://doi.org/10.1093/bioinformatics/bxp616%28Bioinformatics%29.

43. Benjamin Y, Hochberg Y. Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. J Royal Stat Soc B (Methodo- logical). 1995;57(1):289–300. https://doi.org/10.1111/j.2517-6165.1995. tb02031.x.

44. Conesa A, Götz S, García-Gómez JM, Terol J, Talón M, Robles M. Blast2GO: a universal tool for annotation, visualization and analysis in func- tional genomics research. Bioinformatics. 2005;21(18):3674–6. https://doi.org/10.1093/bioinformatics/bti106%28Bioinformatics%29.

45. Jang HA, Park KB, Kim BB, Ali Mohammadie Kojour M, Bae YM, Balasingh S, Lee YS, Han YS, Jo YH. Bacterial but not fungal challenge up-regulates the transcription of Coleopterinc genes in Tenebrio molitor. Entomol Res. 2020;50(9):440–449. https://doi.org/10.1111/1748-5967.12465.

46. Zhang W, Wang W, Wang J, Shen G, Yuan Y, Yan L, Tang H, Wang W. Isola- tion and characterization of a novel laccase for lignin degradation, LacZ1. App Environ Microbiol. 2021;87(23):e035521. https://doi.org/10.1128/ aem.03555-21.

47. Kim HW, Jo JH, Kim YB, Le TK, Cho CW, Yun CH, Chi WS, Yeom SJ. Biodegra- dation of polystyrene by bacteria from the soil in common environments. J Hazard Mat. 2021;416:126239. https://doi.org/10.1016/j.jhazmat.2021. 126239.

48. Yang SS, Zhang Y, Zhou HM, Ji XY, He L, Xing DF, Ren NQ, Ho SH, Wu WM. A novel clean production approach to utilize crop waste residues as co-diet for mealworm (Tenebrio molitor) biomass production with bio- char byproduct for heavy metal removal. Environ Poll. 2019;252:1142– 53. https://doi.org/10.1016/j.envpol.2019.06.028.

49. Peng BY, Su Y, Chen Z, Chen J, Zhou X, Benbow ME, Criddle CS, Wu WM, Zhang Y. Biodegradation of Polystyrene by Dark (Tenebrio obscurus) and Yellow (Tenebrio molitor) Mealworms (Coleoptera: Tenebrionidae). Environ Sci Technol. 2019;53(9):5256–65. https://doi.org/10.1021/acs.est. 8b00963.

50. Yang SS, Brandon AM, Flanagan JC, Yang J, Ning D, Cai SY, Fan HQ, Wang ZY, Ren J, Benbow E, Ren NQ, Waymouth RM, Zhou J, Criddle CS, Wu WM, Biodegradation of polystyrene wastes in yellow mealworms (larvae of Tenebrio molitor Linnaeus): factors affecting biodegradation rates and the ability of polystyrene-fed larvae to complete their life cycle. Chemosphere. 2018;191:979–89. https://doi.org/10.1016/j.chemosphere.2017.10. 117.

51. Brandon AM, Garcia AM, Khlystov NA, Wu W-M, Criddle CS. Enhanced Bioavailability and Microbial Biodegradation of Polystyrene in an Enrichment Derived from the Gut Microbiome of Tenebrio molitor (Mealworm Larvae). Environ Sci Technol. 2021;55(3):2027–36. https://doi.org/10.1021/ acs.est.0c04952.

52. Chen H, Liu J, Chang X, Chen D, Xue Y, Liu P, Lin H, Han S. A review on the pretreatment of lignocellulose for high-value chemicals. Fuel Proc Technol. 2017;160:196–206. https://doi.org/10.1016/j.fuproc.2016.12.007.

53. Weng C, Peng X, Han Y. Depolymerization and conversion of lignin to value-added bioproducts by microbial and enzymatic catalysis. Biotech- nol Biofuels. 2021;14(1):84. https://doi.org/10.1186/s130

69. Rashid GM, Taylor CR, Liu Y, Zhang X, Rea D, Fülöp V, Bugg TD. Identification of manganese superoxide dismutase from sphingobacterium sp. T2 as a novel bacterial enzyme for lignin oxidation. ACS Chem Biol. 2015;10(10):2286–94. <https://doi.org/10.1021/acschembio.5b00298>.

70. Raychoudhury R, Sen R, Cai Y, Sun Y, Lietze VU, Boucias DG, Scharf ME. Comparative metatranscriptomic signatures of wood and paper feeding in the gut of the termite Reticulitermes flavipes (Isoptera: Rhinotermitidae). 2013;22(2):155–71. <https://doi.org/10.1111/jmb.12011>.

71. Kumar A, Naraian R. Chapter 6 - Differential Expression of the Microbial β-1,4-Xylanase, and β-1,4-Endoglucanase Genes, in: H.B. Singh, V.K. Gupta, S. Jogaiah (Eds.), New and future developments in microbial biotechnology and bioengineering, Amsterdam: Elsevier; 2019. p. 95–111. <https://doi.org/10.1016/B978-0-444-63503-7.00006-1>.

72. Cong B, Wang N, Liu S, Liu F, Yin X, Shen J. Isolation, characterization and transcriptome analysis of a novel Antarctic Aspergillus sydowii strain MS-19 as a potential lignocellulosic enzyme source. BMC Microbiol. 2017;17(1):129. <https://doi.org/10.1186/s12866-017-1028-0>.

73. Hou L, Majumder EL. Potential for and distribution of enzymatic biodegradation of polystyrene by environmental microorganisms. Materials (Basel, Switzerland). 2021;14(3). <https://doi.org/10.3390/ma14030503>.

74. Tom LM, Aulillo M, Wu Y-W, Deng K, Gao Y, Xiao N, Rodriguez BG, Louime C, Northen TR, Eudes A, Mortimer JC, Adams PD, Scheller HV, Simmons BA, Ceja-Navarro JA, Singer SW. Low-abundance populations distinguish microbiome performance in plant cell wall deconstruction. Microbiome. 2022;10(1):183. <https://doi.org/10.1186/s40168-022-01377-x>.

75. Chenprakhon P, Wongnate T, Chaiyen P. Monoxygenation of aromatic compounds by flavin-dependent monoxygenases. Protein Sci. 2019;28(1):8–29. <https://doi.org/10.1002/pro.3525>.

76. Avanthi A, Banerjee R. A strategic laccase mediated lignin degradation of lignocellulosic feedstocks for ethanol production. Industr Crops Prod. 2016;92:174–85. <https://doi.org/10.1016/j.indcrop.2016.08.009>.

77. Granja-Travez RS, Bugg TD. Characterization of multicopper oxidase CopA from pseudomonas putida KT2440 and pseudomonas fluorescens PF-S: involvement in bacterial lignin oxidation. Arch Biochem Biophys. 2018;660:97–107. <https://doi.org/10.1016/j.jabb.2018.10.012>.

78. Sulaiman S, Yamato S, Kanaya E, Kim JJ, Koga Y, Takano K, Kanaya S. Isolation of a novel cutinase homolog with polyethylene terephthalate-degrading activity from leaf-branch compost by using a metagenomic approach. Appl Environ Microbiol. 2012;78(5):1556–62. <https://doi.org/10.1128/aem.00725-11>.

79. Sagong HY, Son HF, Seo H, Hong H, Lee D, Kim KJ. Implications for the PET decomposition mechanism through similarity and dissimilarity between PETases from Rhizobacter gummiophilus and Ideonella sakaiensis. J Hazard Mat. 2021;416:126075. <https://doi.org/10.1016/j.jhazmat.2021.126075>.

80. Zhang Z, Peng H, Yang D, Zhang G, Zhang J, Ju F. Polyvinyl chloride degradation by a bacterium isolated from the gut of insect larvae. Nat Commun. 2022;13(1):5360. <https://doi.org/10.1038/s41467-022-32903-y>.

**Publisher's Note**

Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

---

**Ready to submit your research? Choose BMC and benefit from:**

- fast, convenient online submission
- thorough peer review by experienced researchers in your field
- rapid publication on acceptance
- support for research data, including large and complex data types
- gold Open Access which fosters wider collaboration and increased citations
- maximum visibility for your research: over 100M website views per year

**At BMC, research is always in progress.**

**Learn more** biomedcentral.com/submissions