# Journal of Hazardous Materials 465 (2024) 133446

## Contents lists available at ScienceDirect
### Journal of Hazardous Materials

[Journal homepage: www.elsevier.com/locate/jhazmat](https://www.elsevier.com/locate/jhazmat)

---

## Biodegradation of polyethylene terephthalate by *Tenebrio molitor*: Insights for polymer chain size, gut metabolome and host genes

### Authors
- Lei He \({ }^{a, b}\)
- Shan-Shan Yang \({ }^{b, *}\)
- Jie Ding \({ }^{b}\)
- Cheng-Xin Chen \({ }^{b}\)
- Fan Yang \({ }^{c}\)
- Zhi-Li He \({ }^{d}\)
- Ji-Wei Pang \({ }^{e}\)
- Bo-Yu Peng \({ }^{f}\)
- Yalei Zhang \({ }^{f}\)
- De-Feng Xing \({ }^{b}\)
- Nan-Qi Ren \({ }^{b}\)
- Wei-Min Wu \({ }^{g, *}\)

### Affiliations
- \({ }^{a}\) CAS Key Laboratory of Urban Pollutant Conversion, Department of Environmental Science and Engineering, University of Science and Technology of China, Hefei 230026, China
- \({ }^{b}\) State Key Laboratory of Urban Water Resource and Environment, School of Environment, Harbin Institute of Technology, Harbin 150090, China
- \({ }^{c}\) School of Life Science and Technology, Harbin Institute of Technology, Harbin 150090, China
- \({ }^{d}\) Southern Marine Science and Engineering Guangdong Laboratory (Zhuhai), Zhuhai 519080, China
- \({ }^{e}\) China Energy Conservation and Environmental Protection Group, Beijing 100089, China
- \({ }^{f}\) State Key Laboratory of Pollution Control and Resource Reuse, College of Environmental Science and Engineering, Tongji University, Shanghai 200092, China
- \({ }^{g}\) Department of Civil and Environmental Engineering, William \& Clay Codiga Resource Recovery Center, Stanford University, Stanford, CA 94305, USA

---

## Highlights
- High-crystallinity PET biodegradation was confirmed by testing change of \(\delta^{13} \mathrm{C}\).
- Depolymerization is independent of gut microbes.
- Polymer molecular size influences degradation rates.
- Gut microbiomes synergistically biodegrade intermediate products and provide nitrogen sources.
- Upregulation of host genes for PET depolymerization and metabolism.

## Graphical Abstract
![Graphical Abstract](https://i.imgur.com/3Q5Q5Q5.png)

## Article Info
### Keywords
- Polyethylene terephthalate

### Abstract
Polyethylene terephthalate (PET or polyester) is a commonly used plastic and also contributes to the majority of plastic wastes. Mealworms (Tenebrio molitor larvae) are capable of biodegrading major plastic polymers but their biodegradation mechanisms are not well understood. This study investigated the biodegradation of high-crystallinity PET by mealworms and explored the roles of polymer chain size, gut metabolome, and host genes. The results showed that high-crystallinity PET could be biodegraded by mealworms, and the degradation rate was influenced by polymer molecular size. The gut microbiomes of mealworms played a synergistic role in the biodegradation of PET intermediates and provided nitrogen sources. Upregulation of host genes for PET depolymerization and metabolism was observed. These findings provide insights into the biodegradation mechanisms of high-crystallinity PET by mealworms and could contribute to the development of biodegradable plastics.

---

*Abbreviations: ABC, ATP binding cassette; BD, Broad extent depolymerization; CAZY, Carbohydrate-active enzymes; CDS, Coding regions; CFU, Colony forming units; GO, Gene ontology; FA, Fatty acid; FT-IR, Fourier transform infrared spectroscopy; GPC, Gel permeation chromatography; HFIP, 1,1,1,3,3-hexafluoropropanol; \({ }^{1} \mathrm{H}\) NMR, \({ }^{1} \mathrm{H}\) nuclear magnetic resonance; KEGG, Kyoto encyclopedia of genes and genomes; LD, Limited extent depolymerization; LDPE, Low-density polyethylene; LDA, Linear discriminant analysis effect size; MHET, Monohydroxyethyl terephthalate; MPS, Microplastics; \(\mathrm{M}_{\mathrm{n}}\), Number-average molecular weight; \(\mathrm{M}_{\mathrm{w}}\), Weight-average molecular weight; MW, Molecular weight; MWD, Molecular weight distribution; \(\mathrm{M}_{\mathrm{s}}\), Size-average molecular weight; NPs, Nanoplastics; PCoA, Principal coordinate analysis; PE, Polyethylene; PET, Polyethylene terephthalate; PLA, Polyactic acid; PP, Polypropylene; PS, Polystyrene; PUR, Polyurethane; PVC, Polyvinyl chloride; SR, Survival rate; \(\delta^{13} \mathrm{C}\), Stable carbon isotopes; TCA cycle, Tricarboxylic acid cycle; WB, Wheat bran.*

*Corresponding authors.*

*E-mail addresses: shanshanyang@hit.edu.cn (S.-S. Yang), billwu@stanford.edu (W.-M. Wu).*

Received 18 August 2023; Received in revised form 2 January 2024; Accepted 3 January 2024

Available online 6 January 2024

0304-3894/© 2024 Elsevier B.V. All rights reserved.

\section*{Introduction}

Polyethylene terephthalate (PET) is a ubiquitous plastic that is commonly known as "polyester" in the textile industry. PET products belong to aromatic polyesters, expressed as \(\left(\mathrm{C}_{10} \mathrm{H}_{8} \mathrm{O}_{4}\right)_{n}\) and consisting of polymerized units of monohydroxyethyl terephthalate (MHET). When used in textile-grade products, PET generally has an average molecular weight (MW) of about 20.00 kDa, while commercial PET products have 8.00 to 31.00 kDa of the weight-average molecular weight \(\left(\mathrm{M}_{w}\right)\) [1]. Annually, about 70 million tons of PET are produced for packaging and textiles with only a small fraction (<20\%) of that amount being recycled [2,3]. Consequently, PET pollution has been an environmental concern for years [4,5]. The degradation rate is extremely slow with a time frame of months to years. Some researchers predicted the general life time of PET to range from 16 to 48 years in the environment [6].

Since the last decade, a number of PET-degrading bacterial and fungal cultures have been enriched and isolated from environments to help researchers investigate PET biodegradation. The results have shown to be promising [7-9]. Enzymatic PET degradation has achieved significant progress [5]. Nowadays, PET hydrolyses from bacteria such as Actinomycetota phylum, Pseudomonadota phylum, and Bacteriodota phylum, as well as several fungi have been tested and engineered to decompose recycled PET [10,11]. Industrial-scale PET recycling based on enzymatic hydrolysis is underway, however, several technical (low efficient of monomer recovery, low prevalence of activity, low thermal and pH stability, impact of PET properties, e.g., crystallinity, particle size, etc., enzymatic properties and stability), engineering, and economic challenges still remain to be solved [7,10,12]. Furthermore, under ambient temperatures, research on PETase, MHETase, and cutinase remains limited to that of processing monomers or low-crystallinity/amorphous PET [10,13]. PET products for bottles and textiles are associated with high crystallinity of 30-40\%, which restricts the rate of enzymatic degradation [10,12,13]. For example, the half-life of high crystallinity PET (35\%, \(\mathrm{M}_{\mathrm{w}}\) 35.00 kDa) was still 372 hr with the treatment of leaf compost-cutinase enzyme (LCC-ICCG) [13].

Since 2015, insects belonging to families of the darkling beetles (Tenebrionidae) and pyralid moths (Pyralidae) have been found to be capable of biodegrading plastics [5,14-25]. Yellow mealworms, the larvae of Tenebrio molitor Linnaeus 1758, are observed around the world, and they are naturally present in forests with rotting wood, where they eat dried leaves and lignocellulosic materials [26]. Mealworms have been commercialized as animal feed all over the world and have also been suggested for use as a potential sustainable replacement for food protein for both human and animal consumption [27]. Mealworms have been found to have the novel ability to biodegrade polystyrene (PS) [28-30], polyethylene (PE) [31-34], polyvinyl chloride (PVC) [34,35], polypropylene (PP) [34,35], polyurethane (PUR) [36], and polylactic acid (PLA) [37]. Recently, we confirmed biodegradation of two commercial PET polymers with \(\mathrm{M}_{\mathrm{w}}\) of 39.33 and 29.43 kDa, and a crystallinity of 22.80\% and 18.00\%, respectively, resulting in an average mass reduction of 71.03\% and 73.28\% [38].

Physicochemical properties of polymers impact the biodegradation rate of plastics, with major factors including polymer type, surface hydrophobicity, molecular weight distribution (MWD), and three dimensional structure (e.g. crystallinity) [39]. Research results indicated that low-density PE (LDPE) and PS depolymerization and biodegradation by \(T\). molitor was significantly impacted by polymer MW [29,33]. For PS and LDPE, polymers with the low and medium MW are biodegraded via broad extent depolymerization (BD) pattern. Similar BD patterns were also observed in our previous study on biodegradation of two commercial PET polymers by \(T\). molitor larvae [38]. However, when PS and PE with high or extremely high MW were fed, a limited extent depolymerization (LD) pattern occurred, i.e., increase in \(\mathrm{M}_{\mathrm{w}}\) or \(\mathrm{M}_{\mathrm{n}}\) or both, indicating that polymers with high MW accumulated or are degraded at a much slower rate than polymers with lower MW [32,40]. It is unknown if different depolymerization patterns could also occur during biodegradation of commercial PET products with a higher MW than that of the two products tested [38].

The biodegradation of PET has been characterized through the identification of formation oxidized groups using Fourier transform infrared spectroscopy (FT-IR) and \({ }^{1} \mathrm{H}\) nuclear magnetic resonance \(\left({ }^{1} \mathrm{H}\right.\) NMR) [38], which are two tools commonly used for the characterization of biodegradation of PE, PP, PS, and PVC [40]. In this study, we tested the use of stable carbon isotope \(\left(\delta^{13} \mathrm{C}\right)\) values of residual PET as an indication for demonstrating PET biodegradation. This method is based on the fact that during biodegradation, organisms preferentially take up light \({ }^{12} \mathrm{C}\), and thus, the \({ }^{13} \mathrm{C}\) isotope in residual organic compounds increases. Monitoring stable isotope ratio has been successfully applied for several biodegradation tests, e.g., in vitro biodegradation of hydrocarbon vapor [41], and reductive dechlorination of trichloroethylene [42]. To date, this method has barely been applied for measuring the plastic biodegradation ability of \(T\). molitor and other insects when it comes to PET.

Previous studies indicated that biodegradation of distinct plastics in \(T\). molitor larvae implied a complexity of mechanisms [16,32,43]. The gut microbes are involved in plastics biodegradation, particularly depolymerization. Biodegradation of PE by \(T\). molitor and \(G\). mellonella did not stop even under antibiotic depression conditions [15,44], whereas the depolymerization and biodegradation of PS and PVC stopped or was severely inhibited under antibiotic suppression of gut microbes [16,28,32]. As for PET biodegradation, to date, the contribution of the gut microbiota and the symbiotic mechanism has not yet been described in details. To date, limited research has been done on the digestive enzymes secreted during plastic degradation by gut microbes and the insect hosts, i.e., \(T\). molitor, \(Z\). atratus, and \(G\). mellonella [15,19, 45,46], and such PET degradation mechanism remain unclear.

In the environment, the host and associated microbes within the intestine commonly cooperate and interact, eventually forming an


Based on previous studies described above, for biodegradation of PET in T. molitor larvae, we hypothesized: a) selective isotopic exchange between \({ }^{12} \mathrm{C}\) and \({ }^{13} \mathrm{C}\) occurred to enrich \({ }^{13} \mathrm{C}\) in residual PET; b) antibiotic suppression might have less impact on PET depolymerization than on non-hydrolysable polymers such as PS, LDPE, etc.; c) the phenotypic characteristics of the larvae, such as growth state, environmental adaptation, and PET degradation characteristics were closely interconnected with gut microbiome; d) the community structure and metabolic activities of intestinal microbes were impacted by physical properties of the PET polymers, especially MW; and e) the gut microbiota contributes to the digestion of ingested plastics with a supplementation of organic nutrients, especially nitrogen sources.

In this report, we aimed to use PET with three different MWs to examine the aforementioned five hypotheses in order to advance our understanding of the PET degradation mechanism by the associated gut microbiome. We characterized the commercial PET depolymerization with stable isotopic signatures and the impact of antibiotic suppression, and examined the impact of polymer MW on PET degradation, intestinal microbiome, host transcriptome, and intestinal metabolome.

\section*{2. Materials and methods}

\subsection*{2.1. T. molitor Larvae, PET materials, chemicals and feedstock preparation}

\(T\). molitor larvae (27.13 \pm 0.50 mg/larva) were obtained from a breeding farm in Dezhou, Shandong, China. They were fed with wheat bran (WB) prior to testing.

Commercial grade PET microplastics (MPs) powders (< 75.00 \(\mu \mathrm{m}\), crystallinity: \(18.00 \% \pm 2.25 \%\) ) were purchased from Zhongxin Plastic Factory (Guangdong, China), which was also used in previous studies [38] (Fig. S1a). The PET did not contain toxic additives or catalysts. Three high purity PET powders (chromatographically pure as standard) with respective \(M_{w}\) of 1.10, 27.10, and 63.50 kDa, \(M_{n}\) of 0.86, 15.40, and 30.00 kDa, and crystallinity 53.66\% \(\pm 3.31 \%\), 33.43\% \(\pm 3.01 \%\), and 4.25\% \(\pm 1.64 \%\), represented low, medium, and high MW PET polymers, which were named PET \(_{100}\), PET \(_{2700}\), and PET \(_{63500}\) in this study. PET \(_{1100}\) was irregularly shaped solid bulk material. PET \(_{2700}\) was fine powder (around 60.00 \(\mu \mathrm{m}\) ) and PET \(_{63500}\) were pellets with a size of 2.00-4.00 mm (Fig. S1b). These samples were purchased from the American Polymer Standards Corporation. The PET \(_{100}\) and PET \(_{63500}\) samples were ground to reduce particular size to less than 75.0 \(\mu \mathrm{m}\) prior to use. They were soaked in a liquid nitrogen bath for about 5 min and then mechanically ground by a stainless steel high-throughput tissue grinder (Wonbio-L, Shanghai Wonbio Biotechnology co., Ltd, Shanghai). Five types of antibiotics, including aminoglycosides, macrolides, amphenicol, tetracycline, and \(\beta\)-lactam antibiotics, i.e., gentamicin sulfate, erythromycin, chloramphenicol, tetracycline, ampicillin, and kana-mycin were purchased from Biotopped Co. (Beijing, China). They were mixed together for the suppression of gut bacteria.

WB was purchased from the same farm where the larvae were purchased, a breeding farm in Dezhou, Shandong, China. Agar was purchased from Aoboxing Biotechnology Co., Ltd (Beijing). All other chemicals were purchased from Macklin (Shanghai, China).

\subsection*{2.2. Biodegradation of PET in T. molitor larvae}

Before the test, the larvae were kept unfed for \(24 \mathrm{~h}\) to allow for digestion of previous feedstock. Then, they were reared with different diets in food-grade PP containers in an artificial climate incubator (Shanghai Shuli, Shanghai, China) under stable conditions (25 \({ }^{\circ} \mathrm{C}, 70 \%\) relative humidity, in the dark). All treatment groups were conducted in triplicate.

The test for biodegradation of commercial PET was performed with 500 larvae in each incubator (16 \(\times 11 \times 5 \mathrm{~cm})\) [38]. Three feeding conditions were employed: a) PET-fed; b) agar-fed; and c) unfed. The PET-fed larvae were fed with PET-agar gel (5:2, w/w) containing 5 g PET as the primary diet for 36 days. The agar-fed and unfed groups were designated as agar-fed and starvation controls. The larvae in the agar-fed control were fed with 2\% (agar: DI water, w/vol) agar gel. This method was previously used for the biodegradation of PVC and PLA powders [16,37]. Initially, 3 g of PET-agar gel or agar gel (on wet weight basis) was supplied to the incubators. Subsequently, the diets were supplemented daily based on their consumption rate. The supplemented mass was recorded to calculate the total consumption at the end of the test. The frass was collected to extract residual PET polymers for the analysis of \({ }^{13} \mathrm{C} /{ }^{12} \mathrm{C}\) ratio at the end of the test.

\subsection*{2.3. Effect of antibiotic suppression}

Antibiotic suppression tests were designed to examine the contribution of gut microbiota in the presence of severe inhibition of gut microbes. At first, the larvae (500) were fed with commercial PET-agar gel for two weeks. Afterward, PET-agar (5:2, w/w) with an equal proportion of the mixed five antibiotics i.e., gentamicin sulfate, erythromycin, chloramphenicol, tetracycline, ampicillin, and kanamycin (0.075 mg/ mg mixed antibiotics-agar) was prepared. The larvae were reared with the above diet for 5 days and then sampled to count gut bacterial numbers. Ten active larvae from each group were randomly selected and euthanized under sterile conditions. The gut tissues were collected, suspended, and homogenized in a \(10 \%\) phosphate buffered saline solution with a ratio of 1:50 (w/vol). The suspension was plated (dilution factors varied from \(10^{-1}\) to \(10^{-3}\) ) on a plate counting medium (medium composition in Supporting Information (SI M1) to count the number of bacterial colony forming units (CFUs). After five days, no CFUs were found in the plated gut suspensions from antibiotic-treated larvae. The larvae in the incubators were fed with the antibiotics containing PET-agar gel for another two weeks in order to examine PET degradation under antibiotic suppression versus PET degradation by the antibiotics-free control larvae.

\subsection*{2.4. Effect of PET MW on biodegradation}

The effect of MW on PET degradation was performed to determine the consumption of three high purity PET-agar gels with 60 larvae (SI M2). The incubation ended when the feedstocks were consumed completely. This period lasted for 13 days, 15 days, and 16 days for PET \(_{100}\), PET \(_{2700}\), and PET \(_{63500}\), respectively, while lasting 11 days for WB. The weight changes and survival rates of the larvae, as well as the utilization rate and depolymerization/biodegradation of PET by the larvae, were recorded at the end of the consumption duration.

\subsection*{2.5. Characterization of residual PET polymers in frass}

The depolymerization/biodegradation of ingested PET was characterized by Gel permeation chromatography (GPC), \(\delta^{13} \mathrm{C}, \mathrm{FT}-\mathrm{IR}\), and \({ }^{1} \mathrm{H}\) NMR according to established methods used in previous studies [33,37, 40], with detailed characterization information in SI M3. X-Ray Diffraction (Panalytical Empyrean, Netherlands) was utilized to analyze the crystallinity of PET polymers (SI M3). Detection of the ratio of

To assess the PET biodegradation mechanisms in the intestinal tract of the larvae, gut tissues from larvae fed with the low, medium, and high MW PET diets versus the WB diet were collected, frozen with liquid nitrogen, and saved at $-80^{\circ}$C prior to omics analysis. Gut microbiome analysis was performed using high-throughput 16S ribosomal RNA gene sequencing and metagenomic sequencing. For this, the collected samples were immediately rinsed with sterile water to remove soluble organic contaminants from the host. The larvae (25 in each group) were randomly selected in triplicate. The gut samples were sent to LC Bio Technology CO., Ltd (Hangzhou, China) for sequencing and analysis. DNA of gut microbiome samples was extracted using CTBA. The 16S rRNA gene was amplified by primers V3-V4 region 341 F (5'-CCTACGGGNGGCWGGCAG-3') and 805 R (5'-GACTACHVGGGTATC- TAATCC-3'). High-throughput sequencing of bacterial 16S rRNA genes was performed on the Illumina NovaSeq PE250 platform (Illumina NovaSeq PE250, Illumina Inc., San Diego, CA, USA).

Quality filtering on the raw reads was performed according to the fastqtrim (version 0.94). Chimeric sequences were filtered using Vsearch software (version 2.3.4). After dereplication using DADA2, Amplicon Sequence Variants (ASVs) feature table and feature sequences were obtained. Blast was used for sequence alignment, and the feature sequences were annotated with SILVA (Release 138) and NT-16S database for each representative sequence using a confidence threshold of 70%. Other diagrams were implemented using the R package (version 3.5.2).

Alpha diversity was applied in analyzing the complexity of gut bacterial species diversity for a sample through Chao1 and Shannon calculated with QIIME2. Beta diversity was analyzed by the PCoA to evaluate microbiome complexity between samples. The differences among treatments were examined using a Kruskal-Wallis test. Differences were considered statistically significant at $p < 0.05$.

The DNA of gut microbiome samples for metagenomic sequencing was extracted by the E.Z.N.A.® Stool DNA Kit (D4015–02, Omega, Inc., USA). Sequencing library construction was completed by the TruSeq Nano DNA LT Library Preparation Kit (FC-121–4001, Illumina). DNA was fragmented by dsDNA Fragmentase (NEB, M0348S) via incubation at $37^{\circ}$C for 30 min. Library construction began with fragmented cDNA. Blunt-end DNA fragments were generated by combining fill-in reactions and exonuclease activity. Narrow size selection was performed by sample purification beads. Then, an A-base was added to the blunt ends of each strand for ligating to the indexed adapters. Each adapter contained a T-base overhang for ligating the adapter to the A-tailed fragmented DNA. These adapters contained the full complement of sequencing primer hybridization sites for single, paired-end, and indexed reads. Single or dual-index adapters were ligated to the fragments, and the ligated products were amplified with PCR. Library purification, quantification, and quality control was followed by high-throughput sequencing carried out on the NovaSeq6000 platform (Illumina) and 150 bp paired end reads were generated.

Raw sequencing reads were processed to obtain valid reads for further analysis. In particular, sequencing adapters were removed from sequencing reads using cutadapt v1.9. Then, a sliding-window algorithm was used to trim low quality reads by fqtrim v0.94. Next, reads were aligned to the host genome using bowtie2 v2.2.0 to remove host contamination. After quality-filtered reads were obtained, they were de novo assembled to construct the metagenome for each sample by IDBA- UD v1.1.1. All coding regions (CDS) of metagenomic contigs were predicted by MetaGeneMark v3.26.

CDS sequences of all samples were clustered by CD-HIT v4.6.1 to obtain unigenes. Unigene abundance for a certain sample was estimated by TPM based on the number of aligned reads by bowtie2 v2.2.0. The lowest common ancestor taxonomy of unigenes was obtained by aligning them against the NCBI NR database by DIAMOND v 0.9.14. Similarly, the functional annotation (Gene ontology (GO), Kyoto encyclopedia of genes and genomes (KEGG) of unigenes were obtained. Based on the taxonomic and functional annotation of unigenes, along with the abundance profile of unigenes, a differential analysis was carried out at each taxonomic, functional, or gene-wise level by a Kruskal-Wallis test.

All RNA of larval gut tissue including bacterial biomass was extracted using Trizol reagent (Thermo fisher, 15596018) following the manufacturer's procedure. RNA quantity and quality were assessed by Bioanalyzer 2100 and RNA 6000 Nano LabChip Kit (Agilent, CA, USA) with RIN number $>7.0$. Poly-T oligo-attached magnetic beads used to isolate poly (A) mRNA from collected total RNA. After purification, the poly(A)-or poly(A) + RNA fractions were fragmented into small pieces using divalent cations under elevated temperature. Then the cDNA libraries were created using the mRNA-Seq sample preparation kit (Illumina, San Diego, CA, USA) following the protocol. The library was paired-end sequenced (PE150) on an Illumina Novaseq™ 6000 (LC Bio Technology CO., Ltd. Hangzhou, China) with 300 bp ($\pm$ 50 bp) following the vendor's recommended protocol.

We aligned reads of all samples to the Tenebrio molitor reference genome (https://www.ncbi.nlm.nih.gov/data/hub/genome/?taxon=70 67) using an HISAT2 (https://daehwankimlab.github.io/hisat2/, version: hisat2–2.2.1) package, which initially removed a portion of the reads based on quality information accompanying each read and then mapped the reads to the reference genome. The mapped reads of each sample were assembled using StringTie (http://ccb.jhu.edu/software/s tringtie/, version: stringtie-2.1.6) with default parameters. Then, all transcriptomes from all samples were merged to reconstruct a comprehensive transcriptome using gffcompare software (http://ccb.jhu. edu/software/stringtie/gffcompare.shtml, version: gffcompare-0.9.8). Expression level for all the transcripts was estimated by calculating the FPKM (fragment per kilobase of transcript per million mapped reads) value.

Untargeted metabolomic profiling was performed by a Vanquish Flex ultrahigh-performance liquid chromatography system (Thermo Fisher Scientific, Bremen, Germany) to help build a putative PET degradation pathway. An ACQUITY UPLC T3 column (100 mm $\times$ 2.1 mm, 1.8 $\mu$m, Waters, Milford, USA) was used for the reversed phase separation. The samples (100 mg) were first extracted using 1 mL of precooled 50% methanol, vortexed for 60 s, and incubated at room temperature for 10 min. Next, the samples were placed at $-20^{\circ}$C overnight to precipitate the proteins. After centrifugation (4000 g, 20 min), the cleared supernatant was transferred to sample vials for LC-MS analysis. The mobile phase consisted of solvent A (0.1% formic acid) and solvent B (acetoniitrile containing 0.10% formic acid). Gradient elution conducted with the flow rate was 0.40 mL/min. The elution conditions were set as: 0–0.5 min, 5% B; 0.5–7 min, 5% to 100% B; 7–8 min, 100% B; 8–8.1 min, 100% to 5% B; and 8.1–10 min, 5% B.

The generated raw data was preprocessed by the XCMS, CAMERA, and metaX toolbox implemented with the R software. The metabolites were searched in the online database of KEGG (http://www.genome.jp/kegg/) and HMDB (http://www.hmdb.ca/). During the data analysis, metabolic features detected at least 50% of QC samples and 80% biological samples were retained. In the normalized data matrix, variables with a relative standard deviation > 30% of QC samples were removed.

\section*{3. Results and discussion}

\subsection*{3.1. Confirmation of PET biodegradation using \(\delta^{13} \mathrm{C}\) signature}

The results of biodegradation of commercial PET MPs have been partially published in a previous companion paper [38] (Fig. S2). Biodegradation of PET in T. molitor larvae (500 individuals, \(n=3\) ) was conducted using commercial PET MPs (Mn 18.59, Mw 29.43 and Mz 39.58 kDa) (Fig. S3). The larvae consumed \(33.86 \pm 0.54 \mathrm{~g}\) PET over 36 days with an average specific consumption rate of 203.00 mg PET/100 larvae/d and a specific removal rate of 152.00 mg/100 larvae/d (Table S1). The survival rate of larvae fed with PET agar gel was slightly higher than that of unfed larvae (Fig. S2a), indicating PET agar gel provided an energy source to the larvae, similar to previous observations when PS, LDPE, PP, PUR, and PLA were fed to mealworms [16,31,36].

PET can be dissolved in 1,1,1,3,3-hexafluoroisopropanol (HFIP). During the test, water soluble, ethanol extracted, and HFIP extracted fractions in the frass of PET-fed larvae were determined (Fig. S2b and Table S2). The residual PET content in the egested frass was estimated using the content of HFIP extracted fractions. The HFIP extractable fraction (\%, w/w) decreased gradually over the 36-day test period. Mass balance of the ingested PET was calculated as an average 73.28\% mass reduction in ingested PET over 36 days (Fig. S2c). The water-soluble fraction contained hydrophilic, digested organic substances, and salts, a portion of which was gradually increased from \(22.94 \% \pm 2.13 \%\) on day 4 to \(26.36 \% \pm 2.12 \%\) on day 36. The ethanol-extracted fraction reflected lipophilic substances and accounted for \(2.88 \%\) to \(3.99 \%\) of the frass. The residual solid content increased gradually from \(33.26 \%\) on day 4 to \(60.12 \%\) on day 36. After extraction with HFIP, the residual solid in the frass included residual enzymes, microorganisms, insoluble metabolized components, and inorganic residues. The results indicated that significant soluble intermediates (>25\%) accumulated in the excrement.

As biodegradation leads to the enrichment of heavy natural isotopologues in the non-degraded residual fraction [49,50], in this study, stable carbon isotope \(\left(\delta^{13} \mathrm{C}\right)\) analysis confirmed that there was significant enrichment of \({ }^{13} \mathrm{C}\) in residual PET extracted from the frass. The results indicated that in the residual PET polymers, \(\delta^{13} \mathrm{C}\) value increased from \(-27.50 \% \pm 0.18 \%\) of virgin PET to \(-26.05 \% \pm 0.16 \%\) \((p<0.001)\) (Fig. 1a). This analysis confirmed biodegradation of PET polymers in larval gut because \({ }^{12} \mathrm{C}\) was preferentially metabolized by the host and gut microbes during PET biodegradation, thus increasing the \(\delta^{13} \mathrm{C}\) fraction.

The observed PET degradation rate and efficiency in the larvae is much higher than any reported results from microbial in vitro degradation or from degradation in the environment (Table S3). For example, the highest bacterial PET reduction reported was 75\% in 75 days by the PET-degrading bacterium Ideonella sakaiensis 201-F6 with low-crystallinity PET (1.90\%) [51]. On day 4 and day 36, the removal rate of high-crystallinity (18.00\%) commercial PET by T. molitor larvae was 99 times and then 266 times higher than that of bacterial culture (or a respective half-life of 0.39 and 0.15 d versus 38.50 d) (Table S3).

\subsection*{3.2. Effect of antibiotic suppression}

Antibiotic suppression on gut microbes in T. molitor larvae has been used to study the role of gut microbes in plastic, and to evaluate whether the gut microorganisms contribute to plastic biodegradation [16,35,52]. The tests of this study were performed to investigate whether the host could depolymerize PET under suppression of gut microbes.

After 5 days of antibiotics treatment, only a few CFUs (about \(26 \times 10^{6}\) ) were found in the plated samples of the antibiotic treatment group compared to the high CEUs \(\left(3.24 \pm 0.54 \times 10^{6}\right)\) in the non-treatment control, indicating that the gut microbes were severely depressed or inhibited to an extremely low level (Fig. S4). The larvae in the antibiotic treatment groups were then continuously fed with PET agar gel containing antibiotics for 15 days. No further CFU was found in the gut suspension. GPC analyses showed that PET depolymerization continued and that the reduction of \(\mathrm{M}_{\mathrm{n}}, \mathrm{M}_{\mathrm{w}}\), and \(\mathrm{M}_{\mathrm{z}}\) in the antibiotic treatment group was similar to the levels of the non-antibiotic treatment group without significant difference \((p>0.05)\) (Fig. 1b). This indicated that the larvae performed PET depolymerization independent of gut microbes, i.e., by the host itself. This observation was different from those during biodegradation of PS [5,28,32], LDPE [32,53], PVC [16], and PP [35] by T. molitor larvae, i.e., depolymerization of PS, PVC, and PP was severely inhibited or nearly stopped, and depolymerization of LDPE changed pattern. This observation also indicated that the T. molitor host has stronger ability to depolymerize/biodegrade PET than other

\section*{Fig. 1. (a) Confirmation of biodegradation of PET with \(\delta^{13} \mathrm{C}\) stable isotope signature of residual PET versus virgin PET MPs. The precision of \(\delta^{13} \mathrm{C}\) measurement was \(\pm 0.20 \%\). (b) Effect of antibiotic suppression by a mixture of the five antibiotics on depolymerization of commercial PET polymers by T. molitor larvae by comparison of \(\mathrm{M}_{\mathrm{n}}, \mathrm{M}_{\mathrm{w}}\), and \(\mathrm{M}_{\mathrm{z}}\) of virgin PET versus the residual polymers frass from the larvae fed with PET MPs in the absence and presence of antibiotics \((n=3\) samples/group) (Student's t tests, \(*: p<0.05, * *: p<0.01, * * *: p<0.001\), ns: not significantly).}

The weight of the larvae was increased by 52.10% ± 6.10% in WB-fed groups because WB provides abundant nutrients that supported larval development and allowed them to complete their life cycle. Compared to bran, PET lacks essential growth nutrients like proteins, vitamins, minerals, etc., and thus the weight increased less in larvae fed PET as their major food source. The increase in average weight of the PET27100 fed larvae (17.66% ± 3.50%) was similar to those fed PET63500 (17.59% ± 2.80%), and slightly higher than that of PET1100-fed groups (8.05% ± 2.10%) (Table S4). The lower weight change observed in the PET1100-fed larvae could also be attributed to the higher SR in the PET1100-fed group (69.44% ± 2.50%). For the same amount of diet, more larvae survived in the PET1100 group than in the PET27100- and PET63500-fed plastics under antibiotic suppression.

\subsection*{3.3. Effect of MW on PET degradation}

The effect of polymer MW or polymer size on PET biodegradation was investigated by feeding the three high purity PET MPs with \(\mathrm{M}_{\mathrm{w}}\) of 1.10, 27.10, and 63.50 kDa in agar gels (150.00 ± 2.00 mg) versus a normal diet of WB for 16 days \((n=3\) groups, individuals \(=60)\). PET1100 represented low-MW polymers, PET27100 fit the average MW range of commercial PET as medium-MW polymers, while PET63500 belonged to high-MW PET group that contributes only a small (e.g., <2\%) portion in commercial PET products. The \(T\). molitor larvae consumed all PET MPs.

\section*{Fig. 2. Characterization of biodegradation of three high purity PET polymers by \(T\). molitor larvae. Changes in \(\mathrm{M}_{\mathrm{w}}, \mathrm{M}_{\mathrm{n}}\), and \(\mathrm{M}_{\mathrm{z}}\) of three high purity PET MPs PET1100 (a), PET27100 (b), and PET63500 (c) before and after biodegradation by the \(T\). molitor larvae (Student's \(t\) tests, \({ }^{*}: p<0.05,{ }^{* *}: p<0.01,{ }^{* * *}: p<0.001\), ns: not significantly). All measurements were conducted in triplicate \((n=3\) samples/group); Distributions of MW of PET feedstock PET1100 (d), PET27100 (e), and PET63500 (f) before and after biodegradation by the \(T\). molitor larvae; Comparison of FT-IR spectra (g) and \({ }^{1} \mathrm{H}\) NMR spectra (h) of residual PET polymers in the frass fed with respective PET1100, PET27100, and PET63500 versus raw PET polymers.}

groups, which resulted in less diet digested per larva, and thus less weight gained.

The residual PET content in the frass was 4.27% ± 0.75%, 14.05% ± 1.20%, and 26.02% ± 2.01% (w/w) for the larvae fed with PET1100, PET27100, and PET63500, respectively, suggesting that the respective PET mass reduction was at least 95.7%, 86.0%, and 74.0% (Table S4) and that biodegradability of PET did decline as polymer MW increased. PET27100 represents the MW of most commercial PET and polyester products with MW ranging from 8 to 31 kDa [2]. For high MW, the mass reduction of PET63500 was still much greater than mass reduction (usually less than 70%) reported for other major plastics (PS, PE, PP and PVC) tested with T. molitor larvae [16,31–33,36].

The GPC analysis confirmed that depolymerization degree was impacted by MW. The MW of residual PET from frass fed with PET1100 was below the detection limit (−0.535 kDa in this study) (Fig. 2d). This could be attributed to the rapid biodegradation of PET1100 and the PET residue with low MW in the frass. This implies that the consumed PET1100 was almost completely degraded. M, M, and M of PET27100 were reduced by 44.23%, 21.83%, and 22.68%, respectively. However, for PET63500, M was increased by 37.33% (p < 0.001), and M and M were decreased by 4.71% (p < 0.01) and 22.86%, respectively (p < 0.001). The results demonstrated that PET1100 and PET27100 were degraded via BD, while PET63500 was degraded via LD (Fig. 2a, b, and c). BD pattern generally exhibited a decrease in all MW parameters of the plastic polymer, as well as a shift of MWD towards lower molecular weight (Fig. 2e), indicating a relatively rapid biodegradation of plastics. LD pattern indicated a selective breakdown of polymers of low MW, or an inability to scissor long molecular chains. For PET63500, short chain polymers were broken but longer chain polymers also reduced during depolymerization, resulting in narrower MWD after depolymerization (Fig. 2f). This explained the increase in M and decrease in M after degradation of PET63500. The different depolymerization patterns implied that lower MW PET was depolymerized more effectively than higher-MW. Based on observations of the MWD of the extracted polymers from the PET-fed frass, the MWD of medium-MW shifted towards the left side of lower MW and the high-MW polymer became narrower compared to the virgin PET. This suggests an overall decrease in MW after PET polymers passed through the larval intestinal tract, although the removal rate of high MW polymers was lower compared to smaller MW polymers (Fig. 2d, e and f).

According to the X-Ray Diffraction crystallinity analyses, the crystallinity degree of PET1100, PET27100, and PET63500 were 53.66% ± 3.31%, 33.43% ± 3.01%, and 4.25% ± 1.64%, respectively. As reported previously, both MW and crystallinity degree impacts enzymatic degradation [54]. Amorphous PET chains were more susceptible to microbial attacks than crystalline regions [12]. The results of this study indicated that MW had a higher negative impact on depolymerization/degradation of PET polymers than crystallinity degree during PET biodegradation by T. molitor. Similar results were observed in the previous study of PE degradation by T. molitor larvae, the factors affecting PE biodegradation extent was in the order of M > M > M > PE type > crystallinity degree [33]. Due to the limited availability of PET samples, we could not test PET with both high MW and crystallinity degree, although we hypothesized that it would be more persistent to biodegradation.

FTIR analysis further indicated that the three PET MPs were oxidized in the intestinal tract (Fig. 2g). The FTIR spectra of the three PET MPs showed that the peak at 951 cm−1 (aromatic summation band) disappeared and bands around 725 cm−1 (C–H) and 1260 cm−1 (C–O stretching from the ether aromatic group) decreased. In contrast, the intensity of oxidized groups increased. Higher intensities were induced and there was a shift of oxidized C=O functional groups from the band of 1719 cm−1 to 1670 cm−1. The formation of a hydrophilic (hydroxyl) functional group around 3500 to 3250 cm−1 suggests an increase in hydrophilicity of the polymers after PET biodegradation [55]. Also, 1H NMR analysis indicated that the PET residue in frass underwent surface chemical and structural changes (Fig. 2h). Two peaks located around δ 8.1 ad δ 4.7 ppm of virgin PET polymers corresponded to four protons from the terephthalate ring and the oxyethylene units in the main chain, respectively [56]. The 1H NMR spectra of residual PET from the frass showed new singlets related to oxidation, i.e., new peaks were detected in chemical shift regions associated with methylene protons (−CH2O) and hydroxyl (−OH) groups, indicating obvious evidence of transformations and modifications of PET molecular structure due to biodegradation.

\subsection*{3.4. Intestinal microbiome}

16 S rRNA gene amplicons were performed to evaluate the gut microbial community responses to the three high purity PET MPs (Mw 1.10, 27.10, and 63.50 kDa). The results suggested that feeding PET polymers shifted the gut microbial community structure of larvae remarkably (Fig. S6a and b), resulting in reduced gut community richness (Fig. S5a) and diversity (Fig. S5b-d) when compared to the WB-fed larvae (SI R1). The Sankey diagram (Fig. S7a) indicated that Firmicutes and Proteobacteria, two of the common phyla containing plastic degraders [57,58], were predominant in all groups but varied with different proportions. The abundance of Firmicutes was 21.65%, 81.07%, 50.39%, and 71.82% in the control-fed, PET1100-fed, PET27100-fed, and PET63500-fed groups, respectively. Proteobacteria comprised of 5536%, 10.94%, 15.22%, 14.43\% of the phyla in the control-fed, PET1100-fed, PET27100-fed, and PET63500-fed groups, respectively. To date, reported PET degrading bacteria were found in the phylum of Firmicutes (e.g., Bacillus gothetiili, B. cereus, B. muralis) and Proteobacteria (Ideonella sakaiensis 201-F6, Serratia proteamaculans, Vibrio sp.) [51,59]. Firmicutes was the most dominant phylum in the PET-fed groups, whereas Proteobacteria was the most dominant in the control. The results indicated that the gut microbiota of T. molitor larvae shifted towards new structures beneficial to PET degradation.

At the genera level (Figs. S6c and S7a), four ASVs including Cronobacter sp. (20.16\%), Lactococcus sp. (13.07\%), Spiroplasma sp. (12.16\%), and unclassified Enterobacteriaceae (10.29\%) were the dominant genera in the control group. However, in PETs-fed groups, an ASV of Lachnatalea sp., which was not reported in previous studies for PS, LDPE, and PP degradation in T. molitor larvae, exhibited the highest abundance, and comprised of 73.53\%, 37.18\%, and 62.66\% of the genera in PET1100-fed, PET27100-fed, and PET63500-fed groups, respectively. Although Lachnatalea sp. was found as a potential natural organic polymer-degrader isolated from lignin-enriched microflora [60], the PET-de

![Figure 3](attachment://figure3.png)

Fig. 3. Dominant gut microbial groups and function analyses of \( T\). molitor larvae fed with PET diets. The dominant microorganisms based on 16 S rRNA analyses in the samples of PETs-fed larvae versus those the WB fed group (control) by Linear discriminant analysis (LDA) coupled with effect size analysis. (a) PET1100 vs WB, (b) PET27100 vs WB, (c) PET63500 vs WB. Only top ten genus of bacteria in each group are presented. The species marked by the smiley faces were also appeared in the result of random forest analyses, which could be markedly differentiated key species associated with degradation of PET1100, PET27100, and PET63500, respective; (d) Relative abundances of annotated genes at PathwayDefinition of KEGG hierarchy which are more abundant in the PETs-fed samples in metagenomic sequencing libraries. PET1100, PET27100 and PET63500 are high purity PET polymer powders with respective MW of 1.10, 27.10 and 63.50 kDa.

\subsection*{3.5. Contributions of gut bacteria}

Metagenome sequencing was used to investigate the potential contribution of the gut bacteria to PET biodegradation pathways with a focus on genes related to PET degradation and metabolic pathways. Overall, the gene sets involved in the metabolism category of KEGG enrichment was predominant among the three PET-fed samples (Fig. 4a). Genes encoding enzymes related to the degradation of PET and its oligomers as well as enzymes responsible for modifying the PET surface, such as esterase/lipase [67,68] (K14731), carboxylesterase [69, 70] (K02170), and alkane-1 monoxygenase [57] (K00496), were found in the PET-fed groups (Fig. 4b). Esterase/lipase, which was reported as a PET surface-modifying enzyme, can initiate the biodegradation process attack on the ester of PET. Carboxylesterase was reported to be very active toward PET oligomers [70]. Alkane 1-monoxygenase can introduce oxygen to the alkane substrate [57], and could breakdown hydrocarbons by acting on the C-C bonds and transforming them to alcohols. Degradation of PET by alkane 1-monoxygenase has not been

\begin{figure}[h]
\centering
\includegraphics[width=0.8\textwidth]{figure4.png}
\caption{Metagenome sequencing analysis of PET-degrading metabolic functions of the gut microbes in \( T. molitor \). (a) Enriched KEGG genes of three PET-fed groups by metagenome sequencing analysis. (b) Heatmap plot of the esterase/lipase (K14731), carboxylesterase (K002170), and alkane-1 monooxygenase encoding-related genes (K00496); (c) Heatmap of CAZy function classification at level1 of CAZy hierarchy.}
\label{fig:figure4}
\end{figure}

The figure shows the results of metagenome sequencing analysis of PET-degrading metabolic functions of the gut microbes in \( T. molitor \). The analysis was performed on three PET-fed groups: PET100, PET7100, and PET63500. The results are presented in three parts:

(a) Enriched KEGG genes of three PET-fed groups by metagenome sequencing analysis. The KEGG (Kyoto Encyclopedia of Genes and Genomes) pathway enrichment analysis was performed to identify the pathways that were significantly enriched in the gut microbiota of \( T. molitor \) fed with PET. The results showed that the pathways involved in carbohydrate metabolism, lipid metabolism, and amino acid metabolism were significantly enriched in the PET-fed groups compared to the control group.

(b) Heatmap plot of the esterase/lipase (K14731), carboxylesterase (K002170), and alkane-1 monooxygenase encoding-related genes (K00496). The heatmap shows the expression levels of these genes in the three PET-fed groups. The expression levels of these genes were higher in the PET-fed groups compared to the control group, indicating that these genes are involved in the degradation of PET.

(c) Heatmap of CAZy function classification at level1 of CAZy hierarchy. The CAZy (Carbohydrate-Active Enzymes) database was used to classify the enzymes involved in the degradation of PET. The heatmap shows the expression levels of the enzymes in the three PET-fed groups. The expression levels of the enzymes were higher in the PET-fed groups compared to the control group, indicating that these enzymes are involved in the degradation of PET.

Overall, the results showed that the gut microbiota of \( T. molitor \) fed with PET is enriched in genes and enzymes involved in the degradation of PET. The degradation of PET was more efficient in the PET100 group compared to the PET7100 and PET63500 groups, which may be due to the lower molecular weight of PET100.

gut microbes contributed to degradation of PET metabolites. Thus, as discussed above, the gut bacteria had chemotaxis towards the PET metabolic intermediates under non-preferential dietary conditions during PET degradation.

The results indicated that the gut microbiota showed chemotaxis towards aromatic compounds (e.g., benzoate, amino benzoate, and fluorobenzoate) as frequently observed [57], and served as the secondary PET degrader. After the PET polymer chains are depolymerized or hydrolyzed outside the bacterial cells with the molecular size of the depolymerized products reduced or degraded to 10–50 carbon atoms [57], they could penetrate bacterial cells through transport proteins and be further metabolized [70]. The low abundance of transportation and chemotaxis-related genes in PET1100 could be due to the fact that the larvae possessed higher digestive ability toward low MW polymers and thus the gut microbes contributed less to the biodegradation of low MW PET than to the biodegradation of medium and high MW PET.

Genes involved in lipid metabolism were also abundant in PET27100- and PET63500-fed groups (Fig. 3d). Therein, FA is one of the main energy sources of the body and FA derivatives are important components of insect epidermis. Long chain FAs are the precursors of sphingolipids and glycolides, which are essential components of the cell membrane structure and are involved in a variety of cellular biological processes [76]. As a result, FA biosynthesis-related pathways contribute to many cellular biological processes and energy reserves during PET degradation in the larvae. Gut microbes also provide bioemulsifying factors to accelerate plastic degradation [53]. The genes enriched in pathways of FA metabolism and degradation implied that the degradation of FA from PET could be performed by gut bacteria [11]. This indicated energy reserves and FA degradation from PET could be performed by gut bacteria [11,53,76].

Besides PET degradation, pathways of nitrogen metabolism and biosynthesis of nutrients (cofactors and vitamins) were abundant in the PET27100- and PET63500-fed samples compared with the control group. Under nutritional deficiency, many insects harbor symbiotic bacteria that synthesize essential amino acids and/or vitamins for their hosts [77]. In this study, T. molitor larvae modified their gut microbiota to form a new community that could provide the source of nitrogen via nitrogen fixation in the larvae fed PET. However, compared with the high protein content in normal feed (e.g., WB), nitrogen provided by N-fixing bacteria was limited, and supplementation of nutrition-rich diets would be required for long-term survival because gut nitrogen-fixing bacteria could only provide the equivalent protein of 8.60 to 23.00 μg per day for each larva fed plastic only, e.g., PS foam [65].

\section*{a}

\section*{b}

\section*{GO Term}

GO:0022857 Transmembrane transporter activity Peptidase activity

GO:0082333 Peptidase activity

GO:0055114 Obsolete oxidation-reduction process

GO:006030 Chitin metabolic process

GO:006429 Lipid metabolic process

GO:004252 Serine-type endopeptidase activity

GO:008236 Serine-type peptidase activity

GO:0052689 Carboxylic ester hydrolase activity

GO:005351 Carbohydrate:proton symporter activity

GO:003536 Long-chain fatty-CoA metabolic process

GO:0080019 Fatty-acyl-CoA reductase (alcohol-forming) activity

GO:0016042 Lipid catabolic process

GO:0004175 Endopeptidase activity

GO:0019531 Oxidase transmembrane transporter activity

GO:0003993 Acid phosphatase activity

GO:0003333 Amino acid transmembrane transporter activity

GO:0016788 Hydrolase activity, acting on ester bonds

GO:0008970 Phospholipase A1 activity

GO:004568 Chitinase activity

GO:004620 Phospholipase activity

GO:0016407 Acetyltransferase activity

GO:0016787 Hydrolase activity

GO:0013267 Fatty acid elongation, saturated fatty acid

GO:0034625 Fatty acid elongation, monounsaturated fatty acid

GO:0034626 Fatty acid elongation, polyunsaturated fatty acid

GO:0016705 Oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen

GO:0016614 Oxidoreductase activity, acting on \(\mathrm{C} 1-\mathrm{OH}\) group of donors

GO:0012998 Lipase activity

\section*{Pathway name}

map02010 ABC transporters

map00620 Pyruvate metabolism

map00611 Fatty acid biosynthesis

map00230 Purine metabolism

map00010 Glycolysis / Gluconeogenesis

map00020 Citrate cycle (TCA cycle)

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine metabolism

map04080 Neuroactive ligand-receptor interaction

map00520 Amino sugar and nucleotide sugar metabolism

map00270 Cysteine and methionine

3.6. The transcriptome profile of T. molitor larvae during PET degradation

As antibiotic suppression tests demonstrated that PET depolymerization could also happened under inhabitation of intestinal microorganisms, transcriptome analysis was employed to evaluate the role of the larvae host in PET biodegradation by examining the digestive gene expression of T. molitor larvae. Principal component analysis (Fig. S8a) and Pearson correlation analyses (Fig. S8b) showed that the three PET-fed groups clustered together but far away from control samples. The results implied that the metabolism of the host on low, medium, and high MW PET exhibited similarity regardless of the MW of PET. Compared with the analyses result in Fig. S6a, the host was less sensitive to different MW of PET than its gut microbes. Similar to gut microbes, the most upregulated genes which related to the KEGG pathway of ABC transporters suggested that nutrient-deficient food could lead to increased nutrient acquisition capacity in the larvae [78] (Fig. 5b).

The transcriptome analysis indicated that the pathways for hydrolysis were induced after PET feeding. This provides a framework and set of concepts for describing the functions of gene products from all organisms. For the three PET-fed groups, high Gene Ontology (GO) enrichments were observed in carboxylic ester hydrolyase activity (GO:0016788), hydrolyase activity acting on ester bonds (GO:0016788), hydrolyase activity (GO:0016787), peptidase activity (GO:0008233), endopeptidase activity (GO:0004175), serine-type endopeptidase activity (GO:0004252), lipase activity (GO:0016298), cellulase (GO:0008810, GO:0030245), phospholipase activity, and serine-type peptidase activity (GO:0008236) (Fig. 5a), indicating that many gene families were activated for PET hydrolysis and metabolism in T. molitor. The GOs were probably responsible for enhancing the production of long-chain FAs, long-chain alcohols, or long-chain C-C or C-N linked carboxyls or carboxyls from PET. Particularly, overexpression of carboxylic ester hydrolyrases was observed, including that of lipase (EC:3.1.1.13) and carboxylesterase (EC:3.1.1.1, EC:3.1.1.7, EC:3.1.1.59) (Data S2), which are typical enzymes that act on ester bonds. This supports the possibility of PET hydrolysis in by the host. In this study, many genes encoding for carboxylesterases, which can cleave ester bonds (short-chain acyl ester) and facilitate the surface modification of PET [79], were found extensively in the PET-fed larvae. Although FPKM of lipase encoding genes was low, it was clearly activated in the PET-fed samples, compared with the control samples.

Oxidases were also overexpressed in the three PET-fed groups. Enzymes related to drug metabolisms (cytochrome P450 (CYP450) [45,76] and glutathione S-transferases) and laccase were activated in the gut tissue of larvae (Data S2). CYP450s belong to a superfamily of mono-oxygenases, which can catalyze diverse reactions, such as C-H hydroxylation, C-C double bond epoxidation, heteroatom oxygenation, and C-C bond cleavage in complex organic molecules [45]. A recent study conceived a CYP450-driven cascade enzymatic PE degrading pathway in synthetic bacteria to degrade PE [76]. Glutathione S-transferases can further transform intermediates by catalyzing the conjugation of the tripeptide glutathione to electrophilic centers of non-polar compounds, making them more water soluble [80]. Laccase is a widely reported oxidative enzyme known to decompose PE and aromatic compounds [67]. These findings suggested that the upregulated enzymes supported the observation of PET degradation by the host. Further research is needed to characterize roles of individual enzymes.

Similar to the genes of the gut bacteria, lipid metabolism related genes (FA degradation, FA biosynthesis, FA elongation, steroid biosynthesis, sphingolipid metabolism, glycerolipid metabolism, and cutin, suberine and wax biosynthesis) expanded in PET-fed groups (Fig. 5b), which is consistent with the response of other insects toward dietary restrictions [45,81]. Besides energy reservation and the assimilation of biodegraded intermediates into biomass, the upregulation of lipid metabolism related genes could also partially be due to the oxidative stress response to PET MPs ingested by the larvae host (SI R2, Fig. S9) [45,82]. LeMoine et al. [45] surmised that an important metabolic remodeling associated with lipid homeostasis happened when G. mellonella larvae were fed with LDPE. Thus, the change of lipid homeostasis could be due to multiple factors including assimilation, stress response, and other bioreactions.

3.7. Metabolome analysis

Untargeted metabolomics was performed to identify key metabolites during PET biodegradation in order to understand PET metabolic pathways. In the differential metabolite heatmap of the PET-fed groups versus agar-fed control group (Fig. S10), typical PET-degrading intermediates were abundant in the PET-fed samples, such as benzoic acid, protocatechuic acid, and catechol [55]. Compared with the control, 45, 45, and 65 identified metabolites were differentially accumulated in the three PET groups (fold change (FC) ≥ 2.00, p < 0.05). According to the KEGG mapping results, the significantly upregulated differential metabolites identified were classified into six specific MS2 classes: benzenoids, lipids, and lipid-like molecules, organic acids and derivatives, organic nitrogen compounds, organic oxygen compounds, and organo-heterocyclic compounds (Fig. S11). The metabolome results supported the analyses through the host and gut microbiomes. In detail, the upregulated nitrogen compounds in PETs-fed larvae further supported nitrogen fixation mechanism for insect survive on PET with low nutrient content (Fig. S11), as has been validated at the microbiome and metagenome levels. The most enriched KEGG in metabolic expression in all the three PETs-fed samples was glycerophospholipid metabolism (Fig. 6 a, b, c), which was closely linked to the maintenance of optimal physical properties and functions of cell membranes, and it also correlates with normal mitochondrial functions in the gut microbiome. It could be a stress response caused by microplastics. This is consistent with the above discussion about lipid homeostasis in host transcriptome, demonstrating consuming of PET triggered oxidative stress response in the host during the degradation of PET. Many of the significantly accumulated metabolites were enriched in the KEGG pathway of aromatic/polycyclic-aromatic hydrocarbon degradation, benzoate degradation, ethylbenzene degradation, and ether lipid metabolism (Fig. 6 a, b, and c), also supporting production of intermediates from PET degradation as analyses by microbiome and host transcriptome. In detail, based on the result of microbiome and transcriptome analyses, various hydroxyls that typically act on ester bonds supported the PET hydrolysis. This could result in production of hydrolysing products of PET, i.e., bis (2-hydroxy ethyl) terephthalate (BHET), mono (2-hydroxy ethyl) terephthalate (MHET), terephthalic acid, and ethylene glycol. Above metabolome analysis revealed the presence of benzoic acid, protocatechuic acid, and catechol in the PET-fed samples (Fig. S10), which are key products from PET biodegradation. During PET degradation, typical hydrolyzed products of PET (MHET), which was produced by various hydroxyls from hosts and gut microbes, could be metabolized into EG and terephthalic acid. Terephthalic acid (TPA) is usually metabolized into protocatechuic acid, which can be further degraded via catechol [1]. Further research is needed to identify the metabolic intermediates in detail via various analytical methods.

Based on the results from this study and the current knowledge, we propose a primary PET degradation pathway in T. molitor larvae performed by the host and gut microbes (Graphic Abstract and Fig. 7) and summarized in the following basic steps: (a) the larvae chew PET materials into smaller fragments (< 75 μm) and ingest them together with air (O2 and N2) into the foregut; (b) from the foregut to the midgut, PET fragments move and mix with digestive fluid, gut microbes, and extra-cellular enzymes secreted by the host and microbiota. In the midgut, PET is depolymerized by various hydroxylases and

# Figure 6

## Intestinal metabolic expression in PETs-fed larvae

### Scatter plot of KEGG enrichment of differential genes between PET1100-fed sample (a), PET27100-fed sample (b), and PET63500-fed sample (c) compared with the control group


\begin{figure}[h]
\centering
\includegraphics[width=0.7\textwidth]{figure7.png}
\caption{Putative PET degradation pathway contributed by the gut microbiomes (a) and the host \( T.molitor \) larvae only (b).}
\label{fig:figure7}
\end{figure}

The observed PET degradation rate and efficiency in the larvae is much higher than any reported results from microbial in vitro degradation or the environment degradation (Table S3). For example, the highest bacterial PET reduction reported was 75\% in 75 days by the PET-degrading bacterium Ideonella sakaiensis 201-F6 with low-crystallinity PET (1.9\%) [51]. On day 4 and day 36, the removal rate of high-crystallinity (18\%) commercial PET by \( T.molitor \) larvae was 99 and then 266 times higher than that of the bacterial culture (or a respective midgut, and nitrogen fixating bacteria convert \( \mathrm{N}_{2} \) to amino acids to supplement \( \mathrm{N} \) for enzyme synthesis, maintaining PET digestion and life activities.

# Conclusions and future studies

The results of this study indicate that PET polymers with MW ranges of commercial products even up to 63.5 kDa can be biodegraded effectively in Tenebrio molitor larvae using analytical evidences (mass reduction, GPC, FTIR, 1HMR etc.), and especially.

a) T. molitor larvae biodegrade the hydrolysable polymer PET with a half-life (0.15 to 039 day) that is much less than that of non-hydrolysable polymers such as PS and LDPE;

b) PET polymer size impacts mass reduction and depolymerization pattern, while the gut microbiome can adapt to a PET diet [28,30];

c) the host digestive system is capable of degrading PET independent of gut microbes, and the enzymes secreted by the T. molitor host plays a key role in PET biodegradation, as observed under antibiotic suppression;

and d) the gut microbiota contribute significantly to PET biodegradation, and the biodegradation of PET in T. molitor should be considered as symbiotic activities between the gut microbes and the host.

In addition, the gut microbiome for PET degradation could be likely due to ordinal gut microbes digesting lignocellulose in their natural diet. Methodically, this study also demonstrated that the increase in \(\delta^{13} \mathrm{C}\) of residual PET polymer can serve as an effective indication to access biodegradation. The results revealed that PET was synergistically biodegraded by host digestive enzymes together with gut microbes, although the host is able to depolymerize PET independent of gut microbes as well.

To date, low-crystallinity and amorphous PET materials have been widely tested to study enzymatic PET degradation [13,51,85] T. molitor is a model insect for research on plastic degradation. We hypothesize that similar mechanisms for PET biodegradation could also be present in the larvae of other darkling beetles such as Zophobas atratus, Tribolium castaneum, Plesiothalmus davidis, and Uloma etc. More research is needed to verify this hypothesis.

# Funding

The authors gratefully acknowledge the National Natural Science Foundation of China (Grant No. 52170131), National Engineering Research Center for Bioenergy (Harbin Institute of Technology, Grant No. 2021A001), the Open Project of State Key Laboratory of Urban Water Resource and Environment (Harbin Institute of Technology) (Grant No. 2021TS35), and the National Funded Postdoctoral Research Program of China (Certification Number, GZC20223543). Dr. W.-M. Wu appreciates the support by the Woods Institute for Environment at Stanford University (Award 1197667-10-WTAZB).

# CRediT authorship contribution statement

Ding Jie: Writing - review & editing, Supervision, Methodology, Funding acquisition, Conceptualization. Chen Cheng-Xin: Formal analysis, Conceptualization. He Lei: Writing - review & editing, Writing - original draft, Validation, Supervision, Methodology, Formal analysis, Conceptualization. Wu Wei-Min: Writing - review & editing, Supervision, Methodology, Conceptualization. Yang Shanshan: Writing - review & editing, Supervision, Methodology, Conceptualization. Pang Ji-Wei: Methodology, Formal analysis. Peng Bo-Yu: Methodology. Yang Fan: Formal analysis, Conceptualization. He Zhi-Li: Writing - review & editing, Validation. Ren Nan-Qi: Conceptualization. Zhang Yalei: Methodology, Formal analysis. xing De-Feng: Conceptualization.

# Declaration of Competing Interest

The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.

# Data Availability

Data will be made available on request.

# Acknowledgements

The authors appreciate Professor Craig S. Criddle, Stanford University, for his support and suggestion during this study. We gratefully acknowledge the support of the Heilongjiang Province Touyan Team, and the help in manuscript preparation by Ms. Andria T. Wu, DePaul University, and Ms. Julia T. Wu, University of Wisconsin, Madison.

# Supplementary material

The following files are available free of charge.

Additional file:

Supplementary Material : Including Methods S1 to S3\#; Results S1 to S2\#; Figs. S1 to S11\#; Tables S1 to S4\#.

Supplementary Data S1.

Supplementary Data S2.

# Appendix A. Supporting information

Supplementary data associated with this article can be found in the online version at doi:10.1016/j.jhazmat.2024.133446.

# References

[1] Farah, S., Kunduru, K.R., Basu, A., Domb, A.J., 2015. Poly(Ethylene terephthalate) based blends, composites and nanocomposites. William Andrew Publishing, Oxford, pp. 143–165. https://doi.org/10.1016/B978-0-323-31306-3.00008-7.

[2] Plastics - the facts 2022. Plastics Europe. [https://plasticeurope.org/kn owledge-hub/plastics-the-facts-2022/].

[3] Rahimi R, S., Nikbin, I.M., Allahyari, H., Habibi T, S., 2016. Sustainable approach for recycling waste tire rubber and polyethylene terephthalate (PET) to produce green concrete with resistance against sulfuric acid attack. J Clean Prod 126, 166–177. https://doi.org/10.1016/j.jclepro.2016.03.074.

[4] Kumar, A., Bano, N., Bag, S., Chaudhary, D., Jha, B., 2021. Transcriptome-guided insights into plastic degradation by the marine bacterium. Front Microbiol 12. https://doi.org/10.3389/fmicb.2021.751571.

[5] Yang, S.-S., Wu, W.-M., Pang, J.-W., He, L., Ding, M.-Q., Li, M.-X., Zhao, Y.-L., Sun, H.-J., Xing, D.-F., Ren, N.-Q., Yang, J., Griddle, C.S., Ding, J., 2023. Bibliometric analysis of publications on biodegradation of plastics: explosively emerging research over 70 years. J Clean Prod 428, 139423. https://doi.org/10. 1016/j.jclepro.2023.139423.

[6] Müller, R.-J., Kleeberg, I., Deckwer, W.-D., 2001. Biodegradation of polymers containing aromatic constituents. J Biotechnol 86, 87–95. https://doi.org/10.1016/S0168-1656(00)00407-7.

[7] Tournier, V., Duquesne, S., Guillamot, F., Cramail, H., Taton, D., Marty, A., André, I., 2023. Enzymes' power for plastics degradation. Chem Rev 123, 5612–5701. https://doi.org/10.1021/acs.chemrev.2c00644.

[8] Kushwaha, A., Goswami, L., Singhvi, M., Kim, B.S., 2023. Biodegradation of poly (ethylene terephthalate): mechanistic insights, advances, and future innovative strategies. Chem Eng J 457, 141230. https://doi.org/10.1016/j.cej.2022.141230.

[9] Cai, Z., Li, M., Zhu, Z., Wang, X., Huang, Y., Li, T., Gong, H., Yan, M., 2023. Biological degradation of plastics and microplastics: a recent perspective on associated mechanisms and influencing factors. Microorganisms 11, 1661. https://doi.org/10.3390/microorganisms11071661.

[10] Urbanek, A.K., Kosiorowska, K.E., Mironczuk, A.M., 2021. Current knowledge on polyethylene terephthalate degradation by genetically modified microorganisms. Front Bioeng Biotechnol 9, 771133. https://doi.org/1

\bibitem[Bombelli et~al.(2017)Bombelli, Howe, and Bertocchini]{Bombelli2017}
Bombelli, P., Howe, C., Bertocchini, F., 2017. Polyethylene bio-degradation by
  caterpillars of the wax moth \emph{Galleria mellonella}. Curr Biol 27,
  R292–R293. \doi{https://doi.org/10.1016/j.cub.2017.02.060}.

\bibitem[Bombelli et~al.(2019)Bombelli, Howe, Bertocchini, Kim, Huang, Lee, Kim,
  Jeon, Park, Bhak, Ryu, and Zhang]{Bombelli2019}
Bombelli, P., Howe, C., Bertocchini, F., Kim, H.-H., Chu, Y., Huang, J., Lee,
  S., Kim, H.-M., Jeon, S., Park, S.G., Bhak, J., Ryu, C.-M., 2019. The
  \emph{Galleria mellonella} holobionome supports microbiota-independent
  metabolism of long-chain hydrocarbon beeswax. Cell Rep 26, 2451–2464.e5.
  \doi{https://doi.org/10.1016/j.celrep.2019.02.018}.

\bibitem[Peng et~al.(2020)Peng, Chen, Chen, Hu, Criddle, Wu, and
  Zhang]{Peng2020}
Peng, B.-Y., Chen, Z., Chen, J., Yu, H., Zhou, X., Criddle, C.S., Wu, W.-M.,
  Zhang, Y., 2020. Biodegradation of polyvinyl chloride (PVC) in
  \emph{Tenebrio molitor} (Coleoptera: Tenebrionidae) larvae. Environ Int 145,
  106106. \doi{https://doi.org/10.1016/j.envint.2020.106106}.

\bibitem[Woo et~al.(2020)Woo, Song, and Cha]{Woo2020}
Woo, S., Song, I., Cha, H.-J., 2020. Fast and facile biodegradation of
  polystyrene by the gut microflora of \emph{Plesiophthalus davidsi} larvae.
  Appl Environ Microbiol 86, e03161–20. \doi{10.1128/AEM.01361-20}.

\bibitem[Kundungal et~al.(2022)Kundungal, Gangaraju, Sarangapani,
  Patchayappan, Devipriya, and Ramesh]{Kundungal2022}
Kundungal, H., Gangaraju, M., Sarangapani, S., Patchayappan, A., Devipriya,
  S., Ramesh, S., 2022. Role of pretreatment and evidence for the enhanced
  biodegradation and mineralization of low-density polyethylene films by
  greater wax worm. Environ Technol 42, 717–730. \doi{https://doi.org/10.1080/09593330.2021.1639325}.

\bibitem[Zhang et~al.(2022)Zhang, Nong, Xie, Hui, and Chu]{Zhang2022}
Zhang, S., Nong, W., Xie, Y., Hui, J.H.L., Chu, L.M., 2022. Long-term effect
  of growth on transcriptional response of mealworms (\emph{Tenebrio molitor}
  L.). Chemosphere 287, 132063. \doi{https://doi.org/10.1016/j.chemosphere.2021.132063}.

\bibitem[Zhang et~al.(2022)Zhang, He, Wu, Wang, Mai, Hu, Zhang, Huang, Tian,
  and Xia]{Zhang2022a}
Zhang, S., He, Z., Wu, C., Wang, Z., Mai, Y., Hu, R., Zhang, X., Huang, W.,
  Tian, Y., Xia, D., 2022. Complex bilateral interactions determine the fate of
  polystyrene micro- and nanoplastics and soil protists: implications from a
  soil amoeba. Environ Sci Technol 56, 4936–4949. \doi{10.1021/acs.est.1c06178}.

\bibitem[Khang et~al.(2022)Khang, Peng, Yang, Zhu, and Ju]{Khang2022}
Khang, Z., Peng, H., Yang, D., Zhu, G., Zhang, J., Ju, F., 2022. Polyvinyl
  chloride degradation by a bacterium isolated from the gut of insect larvae.
  Nat Commun 13, 5360. \doi{https://doi.org/10.1038/s41467-022-32903-y}.

\bibitem[Sanluis-Verdes et~al.(2022)Sanluis-Verdes, Colomer-Vidal,
  Rodriguez-Ventura, Bello-Villarmino, Spina-Amillaria, Ruiz-Lopez,
  Illanes-Vicioso, Castroviejo, Aiese, Cigliano, Montoya, Palaballa, Pesquera,
  Gonzalez-Legarreta, Arias-Palomino, Solis, Morrocho, Arias, and
  Bertocchini]{Sanluis-Verdes2022}
Sanluis-Verdes, A., Colomer-Vidal, P., Rodriguez-Ventura, F., Bello-Villarmino,
  M., Spina-Amillaria, M., Ruiz-Lopez, E., Illanes-Vicioso, R., Castroviejo,
  P., Aiese Cigliano, R., Montoya, M., Palaballa, P., Pesquera, C.,
  Gonzalez-Legarreta, L., Arias-Palomino, E., Solis, M., Morrocho, T., Arias,
  C.F., Bertocchini, F., 2022. Wax worm saliva and the enzymes therein are the
  key to polyethylene degradation by \emph{Galleria mellonella}. Nat Commun 13,
  5568. \doi{https://doi.org/10.1038/s41467-022-33127-w}.

\bibitem[Bulak et~al.(2021)Bulak, Proc, Pytlak, Puzaka, Gawdezik, and
  Bieganowski]{Bulak2021}
Bulak, P., Proc, K., Pytlak, A., Puzaka, A., Gawdezik, B., Bieganowski, A.,
  2021. Biodegradation of different types of plastics by \emph{Tenebrio molitor}
  insect. Polymers 13, 3508. \doi{https://doi.org/10.3390/polym13033508}.

\bibitem[Luo et~al.(2021)Luo, Wang, Guo, Yang, Qi, Zhao, Gao, and
  Zhou]{Luo2021}
Luo, L., Wang, Y., Guo, H., Yang, Y., Qi, N., Zhao, X., Gao, S., Zhou, A.,
  2021. Biodegradation of foam plastics by \emph{Zophobas aratus} larvae
  (Coleoptera: Tenebrionidae) associated with changes of gut digestive enzymes
  activities and microbiome. Chemosphere 282, 131006. \doi{https://doi.org/10.1016/j.chemosphere.2021.131006}.

\bibitem[Tsochatzis et~al.(2020)Tsochatzis, Lopes, Gika, and
  Theodoridis]{Tsochatzis2020}
Tsochatzis, E., Alberto~Lopes, J., G

[54] Genta, F.A., Dillon, R.J., Terra, W.R., Ferreira, C., 2006. Potential role for gut microbiota in cell wall digestion and glucoside detoxification in Tenebrio molitor larvae. J Insect Physiol 52, 593–601. https://doi.org/10.1016/j. jinsphys.2006.02.007.

[55] Wright, R.J., Bosch, R., Langille, M.G.I., Gibson, M., Christie-Oleza, J.A., 2021. A multi-OMIC characterization of biodegradation and microbial community succession within the PET plasmasphere. Microbiome 9, 141. https://doi.org/10.1186/s40168-021-01054-5.

[56] Wei, R., Breite, D., Song, C., Gräsing, D., Ploss, T., Hille, P., Schwerdtfeger, R., Matysik, J., Schulze, A., Zimmermann, W., 2019. Biocatalytic degradation efficiency of postconsumer polyethylene terephthalate packaging determined by their polymer microstructures. Adv Sci 6, 1900491. https://doi.org/10.1002/advs.201900491.

[57] Zampolli, J., Orro, A., Manconi, A., Ami, D., Natalello, A., Di Gennaro, P., 2021. Transcriptomic analysis of Rhodococcus opacus R7 grown on polyethylene by RNA-seq. Sci Rep 11. https://doi.org/10.1038/s41598-021-00525-x.

[58] Gambarini, V., Pantos, O., Kingsbury, J.M., Weaver, L., Handley, K.M., Lear, G., 2021. Phylogenetic distribution of plastic-degrading microorganisms. mSystems 6, e01112-20. https://doi.org/10.1128/mSystems.01112-20.

[59] Narciso-Ortiz, L., Coreno-Alonso, A., Mendoza-Olivares, D., Lucho-Constantino, C. A., Lizardi-Jiménez, M.A., 2020. Baseline for plastic and hydrocarbon pollution of rivers, reefs, and sediment on beaches in Veracruz state, México, and a proposal for bioremediation. Environ Sci Pollut Res 27, 23035–23047. https://doi.org/10.1007/s11356-020-09851-z.

[60] Sun, C., Zeng, X., Li, G., Du, Y., Wang, Z., Shao, Z., 2021. Diversity of anaerobic degrading bacteria for natural organic polymers in mangrove sediments and isolation of novel groups of bacteria. Acta Microbiol Sin 61, 987–1001. https://doi. org/10.13943/j.cnki.wxb.20200715.

[61] Eisenberg, T., Glaeser, S.P., Kämpfer, P., Schauerte, N., Geiger, C., 2015. Root sepsis associated with insect-dwelling Sebaellida termitidis in a lesser dwarf lemur (Cheirogaleus medius). Antonie Van Leeuwenhoek 108, 1373–1382. https://doi.org/10.1007/s10487-015-0590-4.

[62] Peydaei, A., Bagheri, H., Gurevich, L., de Jonge, N., Nielsen, J.L., 2021. Mastication of polyolefins alters the microbial composition in Galleria mellonella. Environ Pollut 280, 116877. https://doi.org/10.1016/j.envpol.2021.116877.

[63] Pivato, A.F., Miranda, G.M., Prichula, J., Lima, J.E.A., Ligabue, R.A., Seixas, A., Trenitz, D.S., 2022. Hydrocarbon-based plastics: progress and perspectives on consumption and biodegradation by insect larvae. Chemosphere 293, 133600. https://doi.org/10.1016/j.chemosphere.2022.133600.

[64] Loiret, F.G., Grimm, B., Hajirezaei, M.R., Kleiner, D., Ortega, E., 2009. Inoculation of sugarcane with Pantoee sp. increases amino acid contents in shoot tissues; serine, alanine, glutamine and asparagine permit concomitantly ammonium excretion and nitrogenase activity of the bacterium. J Plant Physiol 166, 1152–1161. https://doi.org/10.1016/j.jplph.2009.01.002.

[65] Yang, Y., Hu, L., Li, X., Wang, J., Jin, G., 2022. Nitrogen fixation and diazotrophic community in plastic-eating mealworms Tenebrio molitor L. Microb Ecol 13 (1). https://doi.org/10.1111/1365-2699.13248-021-01930-5.

[66] Navlekar, A.S., Osuji, E., Carr, D.L., 2023. Gut microbial communities in mealworms and Indianmeal moth larvae respond differently to plastic degradation. J Polym Environ 31, 2434–2447. https://doi.org/10.1007/s10924-023-0277-6.

[67] Temporiti, M.E.E., Nicola, L., Nielsen, E., Tosi, S., 2022. Fungal enzymes involved in plastics biodegradation. Microorganisms 10, 1180. https://doi.org/10.3390/ microorganisms10061180.

[68] Carniel, A., Valoni, É., Nicomedes, J., Gomes, A. da C., Castro, A.M. de, 2017. Lipase from Candida antarctica (CALB) and cutinase from Humicola insolens act synergistically for PET hydrolysis to terephthalic acid. Process Biochem 59, 84–90. https://doi.org/10.1016/j.procbio.2016.07.023.

[69] Billig, S., Oeser, T., Birkemeyer, C., Zimmermann, W., 2010. Hydrolysis of cyclic poly(ethylene terephthalate) trimers by a carboxylesterase from Thermobifida fusca KWS. Appl Microbiol Biotechnol 87, 1753–1764. https://doi.org/10.1007/s00253-010-2635-y.

[70] Wei, R., Zimmermann, W., 2017. Microbial enzymes for the recycling of recalcitrant petroleum-based plastics: How far are we? Microb Biotechnol 10, 1308–1322. https://doi.org/10.1111/1751-7915.12710.

[71] Giangeri, G., Morlino, M.S., De Bernardini, N., Ji, M., Bosaro, M., Pirillo, V., Antoniali, P., Molla, G., Raga, R., Treu, L., Campanaro, S., 2022. Preliminary investigation of microorganisms potentially involved in microplastics degradation using an integrated metagenomic and biochemical approach. Sci Total Environ 843, 157017. https://doi.org/10.1016/j.scitotenv.20