# Mixta tenebrionis sp. nov., isolated from the gut of the plastic-eating mealworm Tenebrio molitor L.

## Mengli Xia\(^1\), Jialei Wang\(^1\), Yi-Xin Huo\(^{1,2}\) and Yu Yang\(^1\)*

### Abstract

A bacterial strain, BIT-26\(^{\mathrm{T}}\), was isolated from the gut of plastic-eating mealworm Tenebrio molitor L. The taxonomic position of this new isolate was investigated by using a polyphasic approach. Cells of the strain were Gram-stain-negative, facultatively anaerobic, motile rods with peritrichous flagella. The 16S rRNA gene sequence (1412 bp) of strain BIT-26\(^{\mathrm{T}}\) showed the highest similarity (97.4\%) to Erwinia piriformigrans CFBP 5888\(^{\mathrm{T}}\), followed by Citrobacter sedlakii NBRC 105722\(^{\mathrm{T}}\) (97.3\%), Mixta calida LMG 25383\(^{\mathrm{T}}\) (97.3\%), Cronobacter muytjensii ATCC 51329\(^{\mathrm{T}}\) (97.2\%) and Mixta theicola QC88-366 \({ }^{\mathrm{T}}\) (97.2\%). The results of phylogenetic analyses, based on the 16S rRNA gene and concatenated sequences of four housekeeping genes (atpD, gyrB, infB and rpoB), placed strain BIT-26\(^{\mathrm{T}}\) within the genus Mixta of the family Erwiniaeae. This affiliation was also supported by the chemotaxonomic data. Strain BIT-26\(^{\mathrm{T}}\) had similar predominant fatty acids, including C\(_{12:0}\), C\(_{14:0}\), C\(_{16:0}\), C\(_{17:0}\) cyclo and C\(_{19:0}\) cyclo \(\omega 8 c\), to species of the genus Mixta. In silico DNA–DNA hybridization and average nucleotide identity calculations plus physiological and biochemical tests allowed the genotypic and phenotypic differentiation of strain BIT-26\(^{\mathrm{T}}\) from other species of the genus Mixta with validly published names. Therefore, strain BIT-26\(^{\mathrm{T}}\) is considered to represent a novel species, for which the name Mixta tenebrionis sp. nov is proposed. The type strain is BIT-26\(^{\mathrm{T}}\) (=CGMCC 1.17041\(^{\mathrm{T}}=\) KCTC 72449\(^{\mathrm{T}}\) ).

In 2016, the order `Enterobacteriales', a large and diverse group of bacteria within the class Gammaproteobacteria, was divided into seven families: Budviciaceae, Enterobacteriaceae, Erwiniaeae, Hafniaceae, Morganellaceae, Pectobacteriaceae, Thorselliaeae and Yersiniaceae [1]. At the time of writing, the novel family Erwiniaeae contains 14 genera: Benitsuchiphilus, Buchnera, Erwinia, Ishikawaella, Izhakiella, Mixta, Pantoea, Phaseolibacter, Purcelliella, Rosenkranzia, Siccibacter, Stammerula, Tatumella and Wigglesworthia [2]. Of these genera, Mixta was established in 2018 by reclassifying four species, Mixta calida, Mixta gaviniae, Mixta intestinalis and Mixta theicola, from the genus Pantoea [2]. Based on the results of multilocus sequence analysis (MLSA) and genome-based phylogenies, these four species of the genus Mixta form a monophyletic cluster, which was distinguished from all other genera within the Erwiniaeae [2]. Species of the genus Mixta have been isolated from a variety of environmental habitats. M. calida and M. gaviniae were first isolated from infant formula and an infant formula production environment [3]. M. calida is the type species representing the genus Mixta [2]. The other species, M. theicola and M. intestinalis, were isolated as endophytes from surface-sterilized leaves of black tea and a faecal sample of a healthy human, respectively [4, 5].

Recently, we found that mealworm (Tenebrio molitor L.) can eat Styrofoam as its sole diet and degrade the plastic after passage through the gut [6]. Furthermore, it has been demonstrated that symbiotic gut bacteria play an essential role in plastic degradation, and a plastic-degrading bacterium (Exiguobacterium sp. strain YT2) has been isolated from the mealworm's gut [7]. During the screening of the plastic-degrading gut bacteria associated with mealworms, strain BIT-26\(^{\mathrm{T}}\) was isolated from the gut of a plastic-eating mealworm from Beijing, PR China. In this study, strain BIT-26\(^{\mathrm{T}}\) was characterized on the basis of a taxonomic study using a polyphasic approach. We showed that the strain BIT-26\(^{\mathrm{T}}\) represents a novel species of the genus Mixta and proposed

Fig. 1. Maximum-likelihood tree based on 16S rRNA gene sequences of strain BIT-26 \({ }^{\mathrm{T}}\) and its closely related type species. Bootstrap values based on 1000 replicates are expressed as percentages. Asterisks indicate species names that have not previously been validly published. The 16S rRNA gene sequence of Brenneria alni NCPPB \({ }^{\mathrm{T}}\) was used as an outgroup. Bar, 0.005 substitutions per nucleotide position.


Fig. 2. Maximum-likelihood tree based on concatenated protein sequences of the *atpD, gyrB, infB* and *rpoB* genes of strain BIT-26* and recognized species of the genus *Mixa*, *Pantoena*, *Erwinia* and *Tatumella*. Similar protein sequences of *Brenneria alni* NCPPB* were used as an outgroup. Asterisks indicate species names that have not previously been validly published. Bootstrap values based on 1000 replicates are expressed as percentages. Bar, 0.04 substitutions per amino acid position.

71
Mixta calida 1400/07* [GQ367477, GQ367480, GQ367467, GQ367479]

Mixta gaviniae A18/07* [GQ367482, GQ367485, GQ367481, GQ367484]

Mixta tenebrionis BIT-26* [MN089590, MN089593, MN089596, MN089599]

Mixta intestinalis 2Y989B* [KP420529, KP420531, KP420525, KP420527]

Mixta thelicola QCSS-366* [AB914772, AB914773, AB914774, AB914775]

"Pantoena alhagi LTYR-11Z" [KX494125, KX494926, KX494927, KX494928]

Pantoena septica LMG 5345* [EU145256, EU145272, EU145288, EU145304]

Pantoena cyripedi LMG 2655* [FJ187827, FJ187832, FJ187837, FJ187842]

Pantoena eucrina LMG 2781* [EU145255, EU145271, EU145287, EU145303]

Pantoena beijingensis LMG 27579* [KC969201, KC969202, KC969203, KC969204]

Pantoena dispersa LMG 2603* [EF988731, EF988818, EF988904, EF988990]

Pantoena wallisii LMG 26277* [JF295021, JF295031, JF295041, JF295051]

Pantoena rodasii LMG 26273* [JF295013, JF295023, JF295033, JF295043]

Pantoena ananatis LMG 2665* [EF988737, EF988824, EF988910, EF988966]

Pantoena stewartii subsp. stewartii LMG 2715* [EF988744, EF988831, EF988917, EF988903]

Pantoena breneri LMG 5343* [EU145254, EU145270, EU145286, EU145302]

Pantoena conspicua LMG 24534* [EU145253, EU145269, EU145285, EU145301]

Pantoena deleyi LMG 24200* [EF988683, EF988770, EF988856, EF988942]

Pantoena anthophila LMG 2558* [EF988725, EF988812, EF988898, EF988984]

Pantoena vagans LMG 24199* [EF988681, EF988768, EF988854, EF988940]

Pantoena agglomerans LMG 1286* [EF988711, EF988798, EF988898, EF988970]

Pantoena eucalypti LMG 24197* [EF988675, EF988762, EF988848, EF988934]

Pantoena coffeiphila Ca04* [KM205369, KM205365, KM205362, KJ427825]

98

Erwinia pirillorinigrans CFBP 5888* [JF311469, JF311582, JF311695, JF311808]

Erwinia tasmaniensis Et 1/99* [MN089592, MN089595, MN089598, MN089601]

Erwinia uzeniensis YPPS 951* [KT073225, AB546201, KT073224, KT073223]

Erwinia amylovora LMG 2024* [HQ393596, HQ393608, HQ393620, HQ393632]

Erwinia pyrifoliae ICMP 14143* [HQ393597, HQ393609, HQ393621, HQ393633]

Erwinia aphidicola LMG 24877* [FN547378, FN547377, FN547373, FN547374]

Erwinia persicina LMG 11254* [HQ393598, HQ393610, HQ393622, HQ393634]

Erwinia raphontici LMG 2688* [EF988751, EF988838, EF988924, EF989010]

Erwinia toletana LMG 24162* [EU145258, EU145274, EU145290, EU145306]

Erwinia inieeta B120* [MN089591, MN089594, MN089597, MN089600]

Erwinia gerundensis EM595* [JN591389, JF617413, KP708035, KP708036]

Erwinia oleae DAPP-PG 531* [GU991653, GU991654, GU991655, GU991656]

Erwinia psidii LMG 7039* [HQ393594, HQ393606, HQ393618, HQ393630]

Erwinia tracheiphila LMG 2906* [HQ393591, HQ393603, HQ393615, HQ393627]

Erwinia billingeiae LMG 2613* [EU145259, EU145275, EU145291, EU145307]

Erwinia typographi DSM 22678* [HQ205359, KT073222, HQ2054

| Characteristic | 1 | 2 | 3 | 4 | 5 |
| --- | --- | --- | --- | --- | --- |
| Colony pigmentation | Yellow | White | White | Cream | White |
| Range for growth: |  |  |  |  |  |
| Temperature \(\left({ }^{\circ} \mathrm{C}\right)\) | 12–41 | 10–44 | 7–41 | 15–45 | 10–40 |
| pH | 4.5–9.0 | NA | NA | 4.0–9.0 | 5.0–9.0 |
| NaCl (\%, w/v) | 0–7.5 | NA | NA | 0–10.0 | <9.0 |
| Voges–Proskauer test | + | + | + | + | – |
| Indole production | – | – | – | – | – |
| Citrate utilization | – | + | + | NA | + |
| Enzyme activity: |  |  |  |  |  |
| Gelatinase | – | – | – | – | – |
| Urease | – | – | – | – | – |
| Arginine dihydrolase | – | – | – | NA | – |
| Lysine decarboxylase | – | – | – | NA | – |
| Ornithine decarboxylase | + | – | – | NA | – |
| Acid production from: |  |  |  |  |  |
| d-Sorbitol | – | + | – | – | – |
| Amygdalin | – | – | – | – | + |
| Salicin | + | + | + | – | + |
| Cellobiose | – | + | + | NA | + |
| Lactose | + | + | + | – | – |
| Sucrose | – | + | + | – | – |
| Raffinose | – | + | + | NA | – |
| d-Arabinol | – | – | – | NA | + |
| d-Fucose | w | – | – | + | + |
| Gluconate | w | + | + | NA | NA |
| 2-Keto-gluconate | w | + | + | NA | NA |
| 5-Keto-gluconate | w | + | – | NA | NA |
| DNA G+C content (mol\%) | 56.1 | 57.4 | 58.4 | 59.1 | 57.2 |

### 16S RNA Gene Phylogeny and MLSA

For 16S rRNA gene sequencing and phylogenetic analysis, genomic DNA extraction, PCR amplification and sequencing of the 16S rRNA gene were carried out according to the previously described method [5]. The closest phylogenetic neighbours and the corresponding similarity were determined by aligning the obtained 16S rRNA sequence against the bacterial type species recorded in the EzBioCloud database [8]. Sequence alignment of 16S rRNA gene sequence of the strain

| Fatty acid | 1 | 2 | 3 | 4 | 5 |
| --- | --- | --- | --- | --- | --- |
| C<sub>12:0</sub> | 2.16 | 2.5 | 4.1 | 3.9 | 5.9 |
| C<sub>14:0</sub> | 6.88 | 8.2 | 5.7 | 5.3 | 8.4 |
| C<sub>16:0</sub> | 33.08 | 35.7 | 29.8 | 29.4 | 32.5 |
| C<sub>17:0</sub> cyclo | 10.30 | 19.5 | 13.8 | 18.2 | 10.8 |
| C<sub>18:0</sub> | 0.23 | 0.5 | 0.4 | 0.6 | 1.2 |
| C<sub>19:0</sub> cyclo 08c | 1.18 | 4.3 | 1.4 | 4.2 | – |
| C<sub>19:0</sub> iso | – | – | – | – | 2.5 |
| Summed feature 2* | 8.0 | 10.0 | 9.8 | 9.0 | 9.0 |
| Summed feature 3* | 18.9 | 5.3 | 16.6 | 8.0 | 15.4 |
| Summed feature 8* | 18.3 | 11.3 | 17.3 | 18.0 | 14.3 |

*Summed features are combinations of fatty acids that cannot be separated by the MIDI system. Summed feature 2 comprises any combination of C<sub>12:0</sub> aldehyde, an unknown fatty acid of equivalent chain length 10.9525; summed feature 3 comprises C<sub>16:1ω7c</sub> and/or C<sub>16:1ω6c</sub>; summed feature 8 comprises C<sub>18:1ω7c</sub> and/or C<sub>18:1ω6c</sub>.

Xia et al., Int. J. Syst. Evol. Microbiol. 2020;70:790–796

tRNAscan-SE version 1.3.1 [15–17]. The in silico DDH tests between strain BIT-26T and the closest related species from the genus Mixta, Pantoaea, Erwinia and Tatumella were performed using the Genome-to-Genome Distance Calculator (GGDC) web server version 2.0 with the recommended settings [18]. The ANI calculation was carried out by using the online ANI calculator in the EzBioCloud [19].

A total of 0.999 Gb raw DNA sequence was generated with an average sequence length of 150 bp, which could be assembled into 73 scaffolds with an N50 value of 382 938 bp. The size of the assembled BIT-26T draft genome was at least 4 655 478 bp with a G+C content of 56.1%. In total, 4414 genes, two rRNA and 78 tRNA were predicted in the assembled draft genome. The results of in silico DDH tests (or ANI calculation) revealed that the highest genomes similarity was 29.2% (84.8%) between strain BIT-26T and M. gaviniae, followed by 29.1% (84.8%) between strain BIT-26T and M. calida, 26.6% (82.1%) between strain BIT-26T and M. theicola. The most similar genomes of the type species in the genus Erwinia, Pantoaea and Tatumella, were Erwinia amylovora with a similarity value of 22.8% (78.0%), Pantoaea eurina with 22.0% (78.4%) and Tatumella citrea with 19.9% (72.6%), respectively. The DDH values (or ANI values) between strain BIT-26T and other type species in the genus Mixta were lower than the generally accepted species-level boundary of 70% (or <95–96% ANI) [20]. Therefore, we can confirm that strain BIT-26T represents a novel species in the genus Mixta.

\section*{PHYSIOLOGY AND CHEMOTAXONOMY}

The properties of the isolate were investigated after 48 h growth on LB agar at \(30^{\circ} \mathrm{C}\). The Gram staining was performed with cells grown on the LB agar as described by Gerhardt et al. [21]. Growth under anaerobic atmosphere \(\left(\mathrm{N}_{2}: \mathrm{CO}_{2}: \mathrm{H}_{2}, 80: 10: 10\right.\), v/v) was determined after incubation in an anaerobic chamber on LB agar. Cell motility was examined by observing the spread of diffuse growth in test tubes containing semi-solid LB medium with 0.75% agar. Cell morphology and flagella were observed using SEM (SU8010, Hitachi) and TEM (JEM-1400, JEOL) after 12 h incubation on LB agar at \(30^{\circ} \mathrm{C}\), respectively. The software linked to SEM or TEM was used to measure the size and length of bacteria. Growth at different temperatures (4, 10, 20, 28, 30, 35, 37, 41 and \(44^{\circ} \mathrm{C}\) ) in LB medium and at various pH values (pH 4.0–11.0, prepared with appropriate biological buffers at intervals of 0.5 unit) in LB medium at \(30^{\circ} \mathrm{C}\) were determined after 48 h incubation. Salt tolerance was determined after 48 h incubation in LB medium supplemented with 0–15\% (w/v) \(\mathrm{NaCl}\) (at 1.5% intervals). Catalase activity was determined by the observation of bubble production in a 3\% (v/v) hydrogen peroxide solution, while oxidase activity was examined using 1\% (w/v) tetramethyl-p-phenylenediamine. Acid production from carbohydrates were examined with the API 50CH system (bioMérieux), while other biochemical and enzymatic tests were performed with the API 20E system (bioMérieux) according to the manufacturer's instructions.

Colonies of strain BIT-26T on LB agar were moist, yellow, convex and circular in diameter of 1.0–2.5 mm. Strain BIT-26T was Gram-stain-negative, facultatively anaerobic, rod-shaped and approx. 0.4–0.6×1.3–2.2 \(\mu \mathrm{m}\) in size (Fig. S3). Cells were observed to be motile by using peritrichous flagella (Fig. S4). Strain BIT-26T was able to grow at 12–41 \({ }^{\circ} \mathrm{C}\) (optimum, 20–40 \({ }^{\circ} \mathrm{C}\) ), at pH 4.5–9.0 (pH 6.0–7.5) and in the presence of 0–7.5\% (w/v) \(\mathrm{NaCl}(1–1.5 \%)\). The phenotypic characteristics of strain BIT-26T generally corresponded to the current description of the genus Mixta [2]. Differential phenotypic characteristics between strain BIT-26T and its closest related species of the genus Mixta are summarized in Table 1. Strain BIT-26T could be separated from its closest phylogenetic neighbours, M. calida and M. gaviniae, because it was able to produce ornithine decarboxylase, but unable to utilize citrate and to produce acids from cellobiose, sucrose and raffinose. Strain BIT-26T could be distinguished from M. intestinalis by its ability to produce acids from salicin. Additionally, strain BIT-26T differed from M. theicola in that it was positive for the Voges–Proskauer test and able to produce ornithine decarboxylase and acids from lactose, but unable to utilize citrate or to produce acids from amygdalin, cellobiose and d-arabitol. Other physiological and biochemical characteristics of strain BIT-26T are summarized in the species description.

For determination of the whole-cell fatty acid composition, strain BIT-26T was incubated on LB medium for 48 h at \(30^{\circ} \mathrm{C}\). The fatty acids were extracted and analysed according to the recommendations of the commercial identification system (Microbial Identification System, MIDI), and the whole fatty acid composition was determined using gas chromatography with Agilent 6890 N apparatus.

The whole-cell fatty acid profiles of strain BIT-26T and its closest related species in the genus Mixta are shown in Table 2. Strain BIT-26T shared the similarly predominant fatty acids, including \(\mathrm{C}_{12:0}, \mathrm{C}_{14:0}, \mathrm{C}_{16:0}, \mathrm{C}_{17:0}\) cyclo and \(\mathrm{C}_{19:0}\) cyclo o8c, with other closely related species of the genus Mixta. However, strain BIT-26T could be distinguished from these related species by its smaller ratio of \(\mathrm{C}_{12:0}\) and \(\mathrm{C}_{17:0}\) cyclo and its larger ratio of summed feature 8. The similarities and differences in the fatty acid profiles between strain BIT-26T and its closest related species of the genus Mixta further supported that strain BIT-26T belonged to the genus Mixta and represented a novel species.

In summary, on the basis of the phylogenetic results, genome relatedness, unique phenotypic characteristics and results of chemotaxonomic analyses, we suggest that strain BIT-26T represents a novel species of the genus Mixta, for which the name Mixta tenebrionis sp. nov. is proposed.

\section*{DESCRIPTION OF MIXTA TENEBRIONIS SP. NOV.}

Mixta tenebrionis (te.ne.bri.o'nis. N.L. gen. n. tenebrionis of the mealworm Tenebrio, the host from where the type strain was isolated).

Cells are Gram-stain-negative, facultatively anaerobic, motile rods (approx. 0.4–0.6×1.3–2.2 μm) with peritrichous flagella. Colonies on LB agar are circular, smooth, yellow and 1–2 mm in diameter within 48 h at 30°C. Growth occurs at 12–41°C (optimum, 20–40°C), at pH 4.5–9.0 (pH 6.0–7.5) and in the presence of 0–7.5% (w/v) NaCl (1–1.5%). Catalase activity, the Voges–Proskauer reaction, β-galactosidase and ornithine decarboxylase are positive, whereas oxidase activity, arginine dihydrolase, lysine decarboxylase, tryptophan deaminase, gelatinase, urease, citrate utilization, production of H2S and indole production are negative. In the API 50CH system, acids are produced from L-arabinose, ribose, D-xylene, galactose, glucose, fructose, mannose, rhamnose, inositol, mannotol, N-acetylglucosamine, arbutin, aesculin, salicinose, maltose, lactose and trehalose; weakly produced from glyceraldehyde, D-arabinose, melibiose, geranyl, D-fucose, gluconate, 2-keto-gluconate and 5-keto-gluconate; and are not produced from erythrose, L-xylene, adonitol, methyl β-D-xyllose, sorbose, dulcitol, sorbitol, methyl α-D-mannosidase, methyl α-D-glucoside, amygdalin, cellobiose, sucrose, inulin, melezitose, raffinose, starch, glycogen, xylitol, D-turanose, D-lyxose, D-tagatose, L-fucose, D-arabitol and L-arabitol. The predominant fatty acids are C12:0, C14:0, C16:0, C17:0 cyclo, C9:0 cyclo ω8c, summed feature 2 (comprising any combination of C12:0 aldehyde, an unknown fatty acid of equivalent chain length 10.9525), summed feature 3 (comprising C16:1ω7c and/or C16:1ω6c) and summed feature 8 (comprising C18:1ω7c and/or C18:1ω6c).

The type strain, BIT-26T (=CGMCC 1.17041T=KCTC 72449T), was isolated from the gut of the plastic-eating mealworm Tenebrio molitor L. in Beijing, PR China. The DNA G+C content of the type strain is 56.08%.

Funding information This work was supported by the National Natural Science Foundation of China (No. 51603004 and No. 31961133015), the Young Elite Scientist Sponsorship Program of the China Association of Science and Technology (No. 2017QNRC001) and the Beijing Institute of Technology Research Fund Program for Young Scholars (No. 3160011181804).

Acknowledgements We gratefully acknowledge the Institute of Food Science and Technology, Chinese Academy of Agriculture Science, for assistant in flagella observation using transmission electron microscopy.

Conflicts of interest The authors declare that there are no conflicts of interest.

References
1. Adeolu M, Alnajar S, Naushad S, S Gupta R. Genome-based phylogeny and taxonomy of the 'Enterobacteriales': proposal for Enterobacteriales ord. nov. divided into the families Enterobacteriaceae, Erwiniaeae fam. nov., Pectobacteriaceae fam. nov., Yersiniaceae fam. nov., Hafniaceae fam. nov., Morganellaceae fam. nov., and Budviciaceae fam. nov. Int J Syst Evol Microbiol 2016;66:5575–5599.
2. Palmer M, Steenkamp ET, Coetzee MPA, Avontuur JR, Chan WY et al. Mixta gen. nov., a new genus in the Erwiniaeae. Int J Syst Evol Microbiol 2018;68:1396–1407.
3. Popp A, Cleenwerck I, Iversen C, De Vos P, Stephan R. Pantoaea gaviniae sp. nov. and Pantoaea calida sp. nov., isolated from infant formula and an infant formula production environment. Int J Syst Evol Microbiol 2010;60:2786–2792.
4. Kato Tanaka Y, Horie N, Mochida K, Yoshida Y, Okugawa E et al. Pantoaea theicola sp. nov., isolated from black tea. Int J Syst Evol Microbiol 2015;65:3313–3319.
5. Prakash O, Nimmonkar Y, Vaishampayan A, Mishra M, Kumbhare S et al. Pantoaea intestinalis sp. nov., isolated from the human gut. Int J Syst Evol Microbiol 2015;65:3352–3358.
6. Yang Y, Yang J, Zhao J, Wu WM et al. Biodegradation and mineralization of polystyrene by plastic-eating mealworms. 1. confirmed evidence. Environ Sci Technol 2015;49:12080–12086.
7. Yang Y, Yang J, Wu WM, Zhao J, Song Y et al. Biodegradation and mineralization of polystyrene by plastic-eating mealworms: part 2. Role of gut microorganisms. Environ Sci Technol 2015;49:12087–12093.
8. Yoon SH, Ha SM, Kwon S, Lim J, Kim Y et al. Introducing EzBioCloud: a taxonomically United database of 16S rRNA gene sequences and whole-genome assemblies. Int J Syst Evol Microbiol 2017;67:1613–1617.
9. Larkin MA, Blackshields G, Brown NP, Chenna R, McGetigan PA et al. Clustal W and Clustal X version 2.0. Bioinformatics 2007;23:2947–2948.
10. Saitou N, Nei M. The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol Biol Evol 1987;4:406–425.
11. Felsenstein J. Evolutionary trees from DNA sequences: a maximum likelihood approach. J Mol Evol 1981;17:368–376.
12. Tamura K, Stecher G, Peterson D, Filipski A, Kumar S. MEGA6: molecular evolutionary genetics analysis version 6.0. Mol Biol Evol 2013;30:2725–2729.
13. Brady C, Cleenwerck I, Venter S, Vancanneyt M, Swings J et al. Phylogeny and identification of Pantoaea species associated with plants, humans and the natural environment based on multilocus sequence analysis (MLSA). Syst Appl Microbiol 2008;31:447–460.
14. Luo R, Liu B, Xie Y, Li Z, Huang W et al. SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler. Gigascience 2012;1:18.
15. Deicher AL, Bratke KA, Powers EC, Salzberg SL. Identifying bacterial genes and endosymbiont DNA with glimmer. Bioinformatics 2007;23:679–679.
16. Besemer J, Borodovsky M. GeneMark: web software for gene finding in prokaryotes, eukaryotes and viruses. Nucleic Acids Res 2005;33:W451–W454.
17. Lowe TM, Eddy SR. tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence. Nucleic Acids Res 1997;25:955–964.
18. Meier-Kolth