# Biodegradation of polystyrene by intestinal symbiotic bacteria isolated from mealworms, the larvae of *Tenebrio molitor*

## Ji-Won Park*, Minjun Kim*, Seo-Young Kim*, Jihye Bae*, Tae-Jong Kim*

*Department of Forest Products and Biotechnology, College of Science and Technology, Kookmin University, 77 Jeongneung-ro, Seongbuk-gu, Seoul 02707, Republic of Korea

## Article Info

### Keywords:
- *Acinetobacter septicus*
- Biodegradation
- Mealworm
- Micro-sprayed polystyrene particle
- Polystyrene

### Abstract
*Objective*: Polystyrene is a plastic that leads to environmental pollution. In particular, expanded polystyrene is very light and takes up much space, causing additional environmental problems. The aim of this study was to isolate new symbiotic bacteria which degraded polystyrene from mealworms.

*Methods*: The population of polystyrene-degrading bacteria was increased by enrichment culture of intestinal bacteria from mealworms with polystyrene as a sole carbon source. The degradation activity of isolated bacteria was evaluated by morphological change of micro-polystyrene particles and the surface change of polystyrene films.

*Results*: Eight isolated species (*Acinetobacter septicus*, *Agrobacterium tumefaciens*, *Klebsiella grimonii*, *Pseudomonas multiresinivorans*, *Pseudomonas nitroreducens*, *Pseudomonas plecoglossicida*, *Serratia marcescens*, and *Yokenella regensburgei*) were identified that degrade polystyrene.

*Conclusion*: Bacterial identification shows that a broad spectrum of bacteria decomposing polystyrene coexists in the intestinal tract of mealworms.

## Introduction

Plastic production has increased by about 10% every year since 1950, with total plastic production increasing from 1.3 million tons in 1950 to 245 million tons in 2006 [1]. And it increased to 350 million tons in 2017 [2]. The consumption of plastics has created a lot of waste, which is now causing enormous environmental pollution [3,4]. Approximately 6.3 billion tons of plastic waste was generated in 2015, of which only 9% was recycled; the rest had to be incinerated or landfilled [5]. The expanded polystyrene (EPS), used as insulations or packaging materials, accounted for 7% of the total plastic production in 2014 [6–8]. However, polystyrene does not decompose, unlike styrene monomers and oligomers, due to its large molecular weight and structural stability [9–11]. To solve this problem biologically, polystyrene degradation experiments using bacteria and fungi from soil and landfills have been proposed, but the degradation rate was less than 3% after four months [12–14].

The biological mechanism of styrene degradation has been proposed in *Pseudomonas* spp. [15,16], *Xanthobacter* sp. [17], and *Rhodococcus* sp [18]. Styrene degradation products of bacteria were identified by gas chromatography [19]. However, little research has been reported on the biodegradation mechanism of polystyrene, and it is also unknown whether polystyrene produces styrene as an intermediate metabolite when biodegraded [20]. Polystyrene degradation by mealworms, the larvae of *Tenebrio molitor*, is well-known

biodegradation [20–22], and through extensive studies on the mealworms, it will be possible to propose an efficient polystyrene biodegradation mechanism.

Many microorganisms, which play a significant role in the degradation of polystyrene, exist in the gut of mealworms [21,23–25]. After it became known that mealworms could grow by feeding only EPS, many studies reported polystyrene-degrading bacteria, such as Exiguobacterium sp., Pseudomonas sp., Bacillus sp., Aeromonas sp., Klebsiella pneumoniae from the intestines of mealworms [26–28]. Besides mealworms, superworms, the larvae of Zophobas morio, are also known to grow by feeding on EPS [28–30], but mealworm is known to biodegrade EPS faster than superworms [31].

In previous studies, some microorganisms and enzymes involved in the degradation of polystyrene have been suggested [26,28,32], but the overall degradation mechanism for polystyrene has not been revealed yet. In addition, the possibility of the presence of unidentified polystyrene degrading symbiotic bacteria from mealworms has been suggested in previous studies [25]. Finding and isolating more polystyrene-degrading microorganisms and revealing their degradation characteristics will help to elucidate the overall polystyrene biodegradation mechanism. In this study, we obtained polystyrene-degrading bacteria by culturing the microorganisms from the digestive tract of mealworms after feeding them with only EPS as the sole nutrient, and the degradation of polystyrene by the isolated and identified bacteria from mealworms was demonstrated.

\section*{2. Materials and methods}

\subsection*{2.1. Mealworms, chemicals, and solutions}

Mealworms, the larvae of \(T\). molitor, and the bran of wheat were purchased from 'nb Mealworm Insect Farm' (Yangju, Korea).

Expanded polystyrene (EPS) was provided without a charge from Woosung Resin Co., Ltd. (Gimhae, Korea). The volume of EPS is about 55 times that of polystyrene with a diameter of 0.4–1.7 mm and consisted of 93–96\% EPS (CAS number: 9003-53-6), 4–7\% pentane (CAS number: 109-66-0), and less than 1\% of 1,2,5,6,9,10-hexabromocyclododecane (CAS number: 3194-55-6). Polystyrene powder (catalog number: HI-LENE PSP-30, Sinwon Industrial Co., Ltd., Pyeongtaek, Korea) consisted of 95–100\% ethylbenzene homopolymer (CAS No. 9003-53-6), 0–5\% general plastic lubricant was provided from Sinwon Industrial Co., Ltd. (Pyeongtaek, Korea). Polystyrene film (catalog number: ST31-FM-000150, GoodFellow, Seoul, Korea) was an additive-free styrene polymer with a thickness of 0.05 mm. EPS was sterilized for 10 min at \(80^{\circ} \mathrm{C}\) and cooled at room temperature before feeding. Polystyrene powder was immersed in 75\% ethanol for 5 min, separated from ethanol using an aspirator and filter paper, and dried sufficiently with UV light on a clean bench before use.

A saline solution was prepared by mixing 0.85\% (w/v) sodium chloride and distilled water. Luria-Bertani (LB) medium was prepared by mixing 25 g of LB broth (catalog number: 244620, Becton, Dickinson and Company Korea, Seoul, Korea) in 1 L of distilled water. Tryptone yeast extract (TYE) medium was prepared by dispersing 10 g of tryptone (catalog number: 211705, Becton, Dickinson, and Company Korea) and 5 g of yeast extract (catalog number: 212750, Becton, Dickinson, and Company Korea) in 1 L of distilled water. LB and TYE agar plates were prepared by adding 15 g/L of agar (catalog number: 214010, Becton, Dickinson, and Company Korea) to the LB and TYE medium. A 50\% glycerol solution was prepared by mixing glycerin (catalog number: 4066-4400, Daejung Chemicals \& Metals Co., Ltd., Siheung, Korea) and distilled water in a volume ratio of 1:1. Mineral medium was prepared at pH 6.8 by mixing 0.7 g of \(\mathrm{KH}_{2} \mathrm{PO}_{4}, 0.7 \mathrm{~g}\) of \(\mathrm{K}_{2} \mathrm{HPO}_{4}, 1.0 \mathrm{~g}\) of \(\mathrm{NH}_{4} \mathrm{NO}_{3}, 0.005 \mathrm{~g}\) of \(\mathrm{NaCl}, 0.7 \mathrm{~g}\) of \(\mathrm{MgSO}_{4} \cdot 7 \mathrm{H}_{2} \mathrm{O}, 0.002 \mathrm{~g}\) of \(\mathrm{FeSO}_{4} \cdot 7 \mathrm{H}_{2} \mathrm{O}, 0.002 \mathrm{~g}\) of \(\mathrm{ZnSO}_{4} \cdot 7 \mathrm{H}_{2} \mathrm{O}\), and 0.001 g of \(\mathrm{MnSO}_{4} \cdot 7 \mathrm{H}_{2} \mathrm{O}\), and 0.001 g of \(\mathrm{MnSO}_{4} \cdot 7 \mathrm{H}_{2} \mathrm{O}\) in 1 L of distilled water according to ASTM G22-76 [27,33]. To prevent precipitation during the preparation of the mineral medium, \(\mathrm{KH}_{2} \mathrm{PO}_{4}, \mathrm{~K}_{2} \mathrm{HPO}_{4}, \mathrm{NH}_{4} \mathrm{NO}_{3}\), and \(\mathrm{NaCl}\) were dissolved in the distilled water and autoclaved at \(121{ }^{\circ} \mathrm{C}\) for 20 min. Solutions of \(\mathrm{MgSO}_{4} \cdot 7 \mathrm{H}_{2} \mathrm{O}, \mathrm{FeSO}_{4} \cdot 7 \mathrm{H}_{2} \mathrm{O}, \mathrm{ZnSO}_{4} \cdot 7 \mathrm{H}_{2} \mathrm{O}\), and \(\mathrm{MnSO}_{4} \cdot 7 \mathrm{H}_{2} \mathrm{O}\) were sterilized with a syringe filter of \(0.2 \mu \mathrm{m}\) pore size (catalog number: FJ25ASCCA002DL01, GVS Korea Ltd., Namyangju, Korea) separately. These filtered solutions were added to the autoclaved solution after cooling. The mineral agar plates were prepared by adding 1.5\% agar to the mineral medium. EPS solution was prepared by dissolving polystyrene film in chloroform (catalog number: 2548-4100, Daejung Chemicals \& Metals Co., Ltd.) at \(1 \%(\mathrm{w} / \mathrm{v})\). When the plastic solution was sprayed onto the mineral agar plates, it was sprayed at a pressure of 20 psi using an air spray gun vertically for 20 s at a height of 50 cm in a fume hood, according to a previous study (Shin et al., 2021). The prepared plates were used on a clean bench after UV sterilization for 30 min.

\subsection*{2.2. Mealworm breeding and extraction from the digestive tract}

A polypropylene box with an 8 L volume (catalog number: 57004, Komax Industrial Co., Ltd., Seoul, Korea) was used for mealworm breeding. Mealworms were fed with wheat bran for seven days. After dividing into groups with \(12.1 \pm 0.3 \mathrm{~g}\) of total weight of each group, mealworms were fed with EPS for fifteen days. They were bred at room temperature with a lid in the dark. The lid was opened and ventilated twice a day. Mealworm carcasses, cuticles, and pupae were removed daily to make EPS the sole energy source for mealworms.

After breeding, the

2.3. Separation and identification of symbiotic bacteria from the gut of mealworms

About 0.3 g of the separated digestive tract was inoculated into 80 mL of mineral medium with 1.0 g of polystyrene powder and incubated with a shaking speed of 150 rpm at 21 °C. After nine-day incubation, 80 μL of the culture medium was sub-cultured in 80 mL of mineral medium with 1.0 g of polystyrene powder and incubated with a shaking speed of 150 rpm at 21 °C for fifty-six days. After culture, the cell culture medium was serially diluted, and 100 μL of the diluted cell was spread on an LB agar plate and incubated at 21 °C for 40 h. After arbitrarily choosing a specific region on the plate, around 100 colonies were picked up regardless of the colony characteristics and streaked on a new LB agar plate. After incubation at 21 °C for 40 h, a single colony was picked up from each streaked plate, streaked again on the LB agar plate, and incubated at 21 °C for 40 h. A single colony obtained from the secondary streaked plate was inoculated into 5 mL of LB medium and incubated with a shaking speed of 200 rpm at 21 °C. After incubation for 40 h, the culture solution was stored at −80 °C with 25% glycerin.

The shape of the isolated bacteria was observed with an optical microscope (Zeiss Scope. A1, AXIO, Oberkochen, Germany). The separated bacteria were identified using a 16S rRNA sequence. Bidirectional 16S rRNA sequences of isolated bacteria were analyzed by Biofact Co., Ltd. (Daejeon, Korea). Bidirectional 16S rRNA sequencing was performed using two primers, 27F and 1492R. Polymerase chain reaction conditions for amplification were one cycle of pre-denaturation at 95 °C for 15 min, 30 cycles of amplification at 95 °C for 20 s, 56 °C for 40 s, and 72 °C for 90 s, and one cycle of post-elongation at 72 °C for 5 min. 16S rRNA sequences of 97 strains were obtained from 100 isolated symbiotic bacterial strains. Among them, three strains from Acinetobacter septicus and one from every nine species, a total of 12 strains, were selected for the further polystyrene degradation test. In species with two or more isolates, test strains were randomly selected using the 'RANDBETWEEN' function in Microsoft Excel. Phylogenetic analysis was performed with MEGA4 software using the 16S rRNA sequence of the 12 selected strains by the neighbor-joining method with bootstrap analysis of 1000 replicates [35].

2.4. Cell growth in LB and TYE medium

Twelve strains were individually streaked on LB agar and cultured at 21 °C for 40 h. The obtained single colony was inoculated into 20 mL of LB medium in a 250 mL baffled flask and cultured at 21 °C with 200 rpm for 40 h. The pre-cultured strain was inoculated with Abs600 = 0.05 cells in 20 mL of the main culture LB medium. The main culture was incubated for 24 h at 21 °C with 200 rpm, and absorbance was measured at 600 nm every 2 h.

In the case of measuring growth in the TYE medium, the same method described above was used, and the TYE medium was used instead of the LB medium. The experiment was carried out in three repetitions, and the mean value and standard deviation were calculated.

2.5. Attachment of microorganisms on the polystyrene film

Before inoculation, the film was sterilized with 95% ethanol and dried on a clean bench. After pre-culture of each strain in 5 mL of LB medium at 21 °C with the shaking speed of 200 rpm, the cells with Abs600 = 1 of the final concentration were inoculated in 5 mL of mineral medium. A polystyrene film with 14 mm × 14 mm was completely immersed in the medium solution and incubated at 21 °C for 1 h. The film was rinsed with 0.85% saline solution twice. The bacterial cells on the film's surface were observed using a microscope (Zeiss Scope. A1, AXIO) at a magnification of × 400.

2.6. Degradation of micro-sprayed polystyrene by microorganisms

For measuring micro-sprayed polystyrene degradation on the mineral agar plate, the strains stored at −80 °C were streaked on the LB agar plate and cultured at 21 °C for 40 h to obtain single colonies. The single colony was inoculated in 5 mL of LB medium and incubated at 21 °C with a shaking speed of 200 rpm for 40 h. The pre-culture medium was inoculated into 5 mL of fresh LB medium with Abs600 = 0.05 and incubated with a shaking speed of 200 rpm at 21 °C until Abs600 = 1.0. The cells in 1 mL of the main culture were rinsed three times with mineral medium using centrifugation (6,750 × g, 1 min) and then resuspended in 1 mL of mineral medium.

The resuspended cells with 10 μL were placed as a spot on the mineral agar plates, micro-sprayed with polystyrene, and incubated at 21 °C for fourteen days. The cell growth was measured by photographing the inoculated spot, and the decomposition of polystyrene particles was observed under an optical microscope (Zeiss Scope. A1, AXIO). As a control, the mineral medium was spotted on the same medium and cultured for fourteen days together, and Escherichia coli O157:H7 ATCC 43895 was used as a negative control.

2.7. Measuring polystyrene film degradation by scanning electron microscope

In order to observe changes in the surface of polystyrene by bacteria isolated from mealworms, the surface of polystyrene film was analyzed using a scanning electron microscope (SEM). The cells were cultured in the same manner described in the previous section, "Degradation of micro-sprayed polystyrene by microorganisms." The resuspended cells with 10 μL were placed on the mineral agar plates. After the spotted bacterial strains dried, a polystyrene film was covered over the cells. A polystyrene film was the only carbon source. After incubating at 21 °C for 30 days, the surface of the polystyrene film in contact with the strain was analyzed by SEM. In order to observe the surface change in the polystyrene film caused by bacteria, the film was incubated with 2% sodium dodecyl sulfate

# Results

## Isolation and identification of bacteria degrading polystyrene from intestine of mealworms

Previous studies have demonstrated that mealworms digest EPS, and it has been suggested that the intestinal symbiotic microbes play a significant function in EPS degradation [21,32,36]. This study was conducted assuming that the digestion of EPS alone increases the portion of EPS-degrading microorganisms in the intestine of mealworms. Four groups with about 50 mealworms per group were bred. The EPS intake by mealworms was confirmed by the irregular shape of EPS in Fig. 1. Mealworm weight changes before and after EPS feeding are shown in Table 1. On average, mealworms lost 10.8% of their weight after a fifteen-day EPS-only diet. This result confirmed that EPS is a dietary substance but not a good nutrient for mealworms, as in a previous study [37].

After a fifteen-day EPS-only diet, the digestive tract of mealworms was isolated to obtain total symbiotic microorganisms. Intestinal microorganisms were spread on LB agar plates after fifty-six-day enrichment culture. About 97 colonies were non-selectively obtained from an arbitrarily determined area and then taxonomically identified through 16S rRNA sequencing (Supplementary Table S1). The summary of identified taxonomic species is shown in Table 2.

To measure the polystyrene degradation activity, bacteria as many as the number in parentheses in Table 2 were randomly selected. The size and the shape of the selected twelve bacteria are shown in Fig. 2. All cells were rod-shaped, the length of the cells was 0.8–2.7 μm, and the thickness was 0.7–1.0 μm. The phylogenetic tree of the selected strain is shown in Fig. 3. There were 11 strains in the class of Gammaproteobacteria and 1 strain in the class of Alphaproteobacteria. Gammaproteobacteria were classified into five families, Enterobacteriaceae, Yersiniaceae, Aeromonadaceae, Pseudomonadaceae, and Moraxellaceae.

## Isolated bacterial growth in nutrient media

Before confirming the polystyrene degrading activity of the selected 12 candidate strains, the growth curves were observed in LB (Fig. 4) and TYE (Fig. 5) nutrient media to identify the growth characteristics of the bacteria. The TYE medium is a medium in which sodium chloride has been removed from the LB medium and was used to determine whether salt affects the growth of strains [38].

In the cell growth curve measured by absorbance at 600 nm, except for three strains of A. septicus (A. septicus MA21, A. septicus MA62, and A. septicus MA65), the cells grew well and showed typical bacterial cell growth. The three strains of A. septicus could not accurately measure their growth due to autoaggregation. This autoaggregation of A. septicus was observed microscopically in Fig. 6.

![Fig. 1](image1.png)
The shape of the expanded polystyrene (EPS) spheres destroyed by the ingestion of the mealworms. It was a perfectly spherical EPS with a diameter of about 0.4–1.7 mm before feeding, but the torn irregular shape confirms that the mealworms ingested it.

Table 1
Weight change of mealworms due to EPS diet.

| Group number | Weight before feeding (g) | Weight after feeding (g) | Reduced weight (g) | Relative weight change (%) |
|--------------|--------------------------|-------------------------|--------------------|---------------------------|
| 1            | 12.1                     | 10.6                    | -1.5               | -11.8                     |
| 2            | 11.9                     | 10.5                    | -1.4               | -11.4                     |
| 3            | 12.0                     | 10.9                    | -1.1               | -9.4                      |
| 4            | 12.2                     | 10.9                    | -1.3               | -10.6                     |
| Average      | 12.1 ± 0.13              | 10.7 ± 0.21             | 1.3 ± 0.17         | -10.8 ± 1.06              |


The summary of isolated bacterial strains obtained from the digestive tracts of mealworms that were fed only EPS for 15 days. Strains were identified through 16S rRNA sequence analysis and its BLASTN homology search in the nucleotide database of National Center for Biotechnology Information. The detail BLASTN homology search results are shown in Supplementary Table S1.

| Identified strain | Number of identified strains (Number of strains for the EPS degradation test) |
|---|---|
| Acinetobacter septicus | 69 (3) |
| Aeromonas hydrophila | 7 (1) |
| Agrobacterium tumefaciens | 8 (1) |
| Enterobacter soli | 1 (1) |
| Klebsiella grimmontii | 1 (1) |
| Pseudomonas multiresinivorans | 1 (1) |
| Pseudomonas nitroreducens | 7 (1) |
| Pseudomonas plecoglossicida | 1 (1) |
| Serratia marcescens | 1 (1) |
| Yokenella regensburgei | 1 (1) |

\section*{Fig. 2. Shape and size of twelve bacterial cells selected for evaluating polystyrene degradability. The scale bar on the lower right indicates \(10 \mu \mathrm{m}\). (a) E. soli MA04; (b) P. multiresinivorans MA11; (c) Y. regensburgei MA19; (d) A. septicus MA21; (e) A. hydrophila MA32; (f) S. marcescens MA42; (g) A. septicus MA62; (h) A. septicus MA65; (i) K. grimmontii MA76; (j) P. nitroreducens MA77; (k) A. tumefaciens MA92; (l) P. plecoglossicida MA93.}

The growth of \(P\). nitroreducens MA77 in the LB medium (Fig. 4B (-)) was much lower than in the TYE medium (Fig. 5B (-)). For K. grimmontii MA76 and \(P\). nitroreducens MA77, the doubling time in the LB medium was 139\% and 137\% longer, respectively, than that in the TYE medium. This observation suggested that K. grimmontii MA76 and \(P\). nitroreducens MA77 were sensitive to salt. Based on the doubling time in LB medium of 68.3 min in the exponential phase of E. coli, A. tumefaciens MA92, P. nitroreducens MA77, P. multiresinivorans MA11, K. grimmontii MA76, and \(P\). plecoglossicida MA93 showed a longer doubling time of 113.4 min, 106.8 min,

Fig. 3. Phylogenetic analysis of twelve bacterial cells selected for evaluating polystyrene degradability using their 16S rRNA sequence. Phylogenetic tree constructed by the neighbor-joining method with bootstrap analysis of 1000 replicates using MEGA4 software.

A.

\begin{tikzpicture}
    \begin{axis}[
        width=0.45\textwidth,
        height=0.3\textheight,
        xlabel={Incubation time (hrs)},
        ylabel={Cell density (Abs600)},
        xmin=0, xmax=24,
        ymin=0, ymax=12,
        xtick={0, 4, 8, 12, 16, 20, 24},
        ytick={0, 2, 4, 6, 8, 10, 12},
        legend pos=north west,
        ymajorgrids=true,
        grid style=dashed,
    ]
        \addplot[
            color=black,
            mark=square,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=blue,
            mark=triangle,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=red,
            mark=diamond,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=green,
            mark=star,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=orange,
            mark=triangle*,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=purple,
            mark=otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=yellow,
            mark=otimes*,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes otimes otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes otimes otimes otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes otimes otimes otimes otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24, 12)
            };
        \addplot[
            color=gray,
            mark=otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes otimes,
            ]
            coordinates {
                (0, 0)(4, 2)(8, 4)(12, 6)(16, 8)(20, 10)(24

\section*{3.3. Bacterial cell attachment on the polystyrene film}

Since bacteria's biodegradation of the polystyrene film starts by attaching to the film surface [39], the adhesion of selected bacterial cells to the polystyrene film surface was evaluated (Fig. 6). E. coli MA4 and A. hydrophila MA32 had less adhesion to polystyrene film like E. coli, the negative control. Y. regensburgei MA19, K. grimontii MA76, P. nitroreducens MA77, A. tumefaciens MA92, and P. plecoglossicida MA93 could adhere to polystyrene film. P. multiresinivorans MA11 and S. marcescens MA42 adhered well to the polystyrene film. Three strains of A. septicus (A. septicus MA21, A. septicus MA62, and A. septicus MA65) were attached to the polystyrene film surface by autoaggregation.

\section*{3.4. Degradation of micro-sprayed polystyrene by isolated bacteria}

Micro-sprayed polystyrene was used as a quick method to evaluate the polystyrene degradation activity of bacteria. The small particles increase the body surface area so that the degradation takes place in a short time [31,40]. Micro-sprayed polystyrene as the sole carbon source on the mineral medium was supplied to the agar plate, and \(10 \mu \mathrm{L}\) of the isolated bacteria were placed on a spot. The change was observed for fourteen days (Figs. 7-8).

As a result of observing the spot inoculated with the strain (Fig. 7), in the case of E. coli MA4, and A. hydrophila MA32 did not grow well on the polystyrene micro-sprayed plate like the negative control E. coli. Thick cell spots of the remaining 10 strains suggested that polystyrene was used as a nutrient for their growth. Interestingly, A. septicus MA62 had a thick spot after four days, but other A. septicus MA21 and A. septicus MA65 had a thick spot from one day of culture (Supplementary Fig. S1). This observation suggested that the degradation ability of polystyrene may be different even in the same A. septicus species.

After fourteen days of culture, the micro-sprayed polystyrene particles were observed through a microscope (Fig. 8). There was no significant difference in polystyrene particles of E. coli MA4 and A. hydrophila MA32, similar to the negative control inoculated with E. coli. It was observed that the size of the polystyrene particles was reduced, the number of particles was less, and the edge sharpness of


\section*{A.}

\section*{E. coli ATCC 43895}

\section*{B.}

\section*{E. coli ATCC 43895}

\section*{C.}

\section*{P. multiresinivorans MA11}

\section*{D.}

\section*{Y. regensburgei MA19}

\section*{E.}

\section*{A. septicus MA21}

\section*{F.}

\section*{A. hydrophila MA32}

\section*{G.}

\section*{S. marcescens MA42}

\section*{H.}

\section*{A. septicus MA62}

\section*{I.}

\section*{A. septicus MA65}

\section*{J.}

\section*{K. grimontii MA76}

\section*{K. nitroreducens MA77}

\section*{L.}

\section*{A. tumefaciens MA92}

\section*{M.}

\section*{P. plecoglossicida MA93}

\section*{Fig. 6. Attachment to the surface of the polystyrene film. Bacteria were incubated with polystyrene film for \(1 \mathrm{~h}\), and after rinsing, the attached bacterial cells were observed under a microscope. The scale bar on the lower right indicates \(20 \mu \mathrm{m}\). (a) E. coli ATCC 43895 as a negative control; (b) E. coli MA04; (c) P. multiresinivorans MA11; (d) Y. regensburgei MA19; (e) A. septicus MA21; (f) A. hydrophila MA32; (g) S. marcescens MA42; (h) A. septicus MA62; (i) A. septicus MA65; (j) K. grimontii MA76; (k) P. nitroreducens MA77; (l) A. tumefaciens MA92; (m) P. plecoglossicida MA93.}

the particles was decreased by Y. regensburgei MA19, A. septicus MA21, A. septicus MA62, A. septicus MA65, K. grimontii MA76, and P. nitroreducens MA77. In the case of four strains, P. multiresinivorans MA11, S. marcescens MA42, A. tumefaciens MA92, and P. plecoglossicida MA93, it was impossible to observe the change in polystyrene particles clearly under a microscope because there were too many bacteria on the surface of polystyrene particles. Among these four strains, P. multiresinivorans MA11, S. marcescens MA42, and P. plecoglossicida MA93 were identified as strains with a high degree of adhesion in adhesion experiments on polystyrene film (Fig. 6).

\subsection*{3.5. Destruction of surface integrity of polystyrene film by isolated bacteria}

The microscopic changes on the surface of the polystyrene film by isolated bacteria were observed by scanning electron microscopy after fourteen-day cell incubation (Fig. 9). Corrosion, holes, and pits were observed on the surface of the polystyrene film degraded by bacteria in previous studies [26,31]. The surface of the polystyrene film incubated in the culture medium without microorganisms was smooth and clean (Fig. 7A). E. coli (negative control), E. coli MA04, and A. hydrophila MA32 also showed a similar surface appearance to the polystyrene film without microorganism inoculation. P. multiresinivorans MA11, Y. regensburgei MA19, and S. marcescens MA42 produced micro-cracks on the polystyrene film surface. In the other 7 strains (A. septicus MA21, A. septicus MA62, A. septicus MA65,

\section*{Discussion}

The microbes in the digestive tract of mealworms are essential for the degradation of polystyrene [21,24]. The biological mechanisms of polystyrene degradation microorganisms will provide clues to the biological waste treatment of polystyrene. In order to quickly isolate the intestinal symbiotic microorganisms of mealworms, 97 bacteria were isolated from the intestines of mealworms after fifty-six-day enrichment culture. Among them, 12 strains were randomly selected based on species, and their degradation activity on polystyrene was evaluated.

As a result of cell adhesion to the polystyrene film surface, cell growth in the polystyrene particle layer, decomposition of microsprayed polystyrene particles, and surface destruction of polystyrene film, Y. regensburgei MA19, A. septicus MA21, A. septicus MA62, A. septicus MA65, K. grimontii MA76, and P. nitroreducens MA77 consistently exhibit polystyrene degrading activity. Although it was difficult to observe the changes in microsprayed polystyrene particles, P. multiresinivorans MA11, S. marcescens MA42, A. tumefaciens

\section*{Fig. 8. Degradation of micro-sprayed polystyrene by bacteria. The decomposition of micro-sprayed polystyrene particles can be evaluated as a decrease in the particle size, a decrease in the number of particles, and a change in the particle edges. The scale bar on the lower right indicates \(50 \mu \mathrm{m}\). (a) Area without bacterial inoculation as an original control; (b) E. coli ATCC 43895 as a negative control; (c) E. soli MA04; (d) P. multiresinivorans MA11; (e) Y. regensburgei MA19; (f) A. septicus MA21; (g) A. hydrophila MA32; (h) S. marcescens MA42; (i) A. septicus MA62; (j) A. septicus MA65; (k) K. grimontii MA76; (l) P. nitroreducens MA77; (m) A. tumefaciens MA92; (n) P. plecoglossicida MA93.}

MA92, and P. plecoglossicida MA93 were also considered bacteria that can degrade polystyrene.

In previous studies, P. nitroreducens and Rhizobium radiobacter (newly named A. tumefaciens) were found together in the intestine of Holotrichia parallela larvae [41]. Bacteria of the genera, Acinetobacter, Agrobacterium, Enterobacter, Klebsiella, and Pseudomonas, were also found in the intestine of Aedes albopictus [42]. Aeromonas sp. TM1 and S. marcescens were isolated from the intestines of T. molitor [28,32]. Y. regensburgei was also isolated from the intestine of Eurycantha calcarata [43]. The results of bacterial strains from mealworms in this study confirmed previous studies that the bacteria in this study live in the intestine of insects.

In our previous metagenomic study, more than half of the strains in the intestines of mealworms fed with EPS were two families, Streptococcaceae and Enterobacteriaceae [44]. The strains identified in this study accounted for \(80 \%\) of the families Moraxellaceae and Pseudomonadaceae. This is attributed to the difference in the aerobic culture conditions and the EPS enrichment culture for fifty-six days.

Acinetobacter sp. AnTc-1 isolated from the intestine of Tribolium castaneum degraded polystyrene [45]. Acinetobacter sp. NyZ450 isolated from the mealworms ingesting polyethylene mulching film as sole carbon also showed the ability to form biofilm and decompose polyethylene [46]. Aeromonas sp. TM1 isolated from the intestines of T. molitor and Z. morio showed the ability to degrade

\section*{Fig. 9. The surface structure change of micro-sprayed polystyrene by bacteria. Polystyrene degradation can be observed by debris and cracks. The scale bar on the lower right indicates \(10 \mu \mathrm{m}\). (a) No cell inoculated as a control; (b) E. coli ATCC 43895 as a negative control; (c) E. soli MA04; (d) P. multiresinivorans MA11; (e) Y. regensburgei MA19; (f) A. septicus MA21; (g) A. hydrophila MA32; (h) S. marcescens MA42; (i) A. septicus MA62; (j) A. septicus MA65; (k) K. grimontii MA76; (l) P. nitroreducens MA77; (m) A. tumefaciens MA92; (n) P. plecoglossicida MA93.}

\section*{References}

[28] Agrobacterium sp. can grow in the digestive tract of mealworms with a polystyrene diet [47]. The IS01 strain from partially degraded plastic waste found in a local rural market showed high hydrophobicity and the ability to degrade high-impact polystyrene, which was 99\% consistent with the Enterobacter genus [48]. Klebsiella oxytoca was found in the intestine of mealworms that digested polystyrene [25]. In addition, Klebsiella pneumoniae ZM1 was isolated from the intestines of T. molitor and Z. morio and showed the ability to degrade polystyrene [28]. P. plecoglossicida was isolated from the polyethylene plastic sample and can grow on paraffin as a carbon source [49]. Moreover, high-impact polystyrene composed of a mixture of polystyrene and polybutadiene can be degraded by an unclassified Pseudomonas sp. However, there was only a 10\% weight loss in the 200 mg high-impact polystyrene film [26]. Pseudomonas sp. DSM50071 isolated from the intestines of superworms degraded polystyrene by S-formylglutathione hydrolase and serine hydrolase [31]. S. marcescens was a representative strain of the intestinal microbiota of T. molitor that degrades polystyrene [32]. Serratia sp. WSW isolated from the intestine of Plesiothalamus davidis formed a biofilm on polystyrene film and showed the activity of degrading polystyrene [50]. Agrobacterium spp. was also found in the intestinal microbiota of mealworms fed with polystyrene [47]. In the case of the Yokenella genus, we could not find any research results on the degradation of plastic-related or polymer materials.

There have been previous studies on bacteria that decompose polystyrene at the genus level. However, bacterial cells that degrade polystyrene were obtained and identified down to the species level in this study.

A significant change in the comparison of the doubling times in the exponential growth phase of LB and TYE media suggested that salt affected the growth of two bacteria, K. grimontii MA76 and P. nitroreducens MA77. In the conditions for degrading polystyrene using these two strains, salt was an essential factor to be considered.

Autoaggregation was observed in three tested Acinetobacter strains in this study. The microorganisms of the genus Acinetobacter utilize the AtaA protein to cause autoaggregation and promote adhesiveness to abiotic surfaces [51]. We hypothesized that this adhesive ability to abiotic surfaces would provide an opportunity for Acinetobacter spp. to degrade EPS. A previous study [52] showed that A. johnsonii and A. junii had hydrophobic cell surfaces. If this hydrophobic cell surface characteristic is also present in the Acinetobacter isolated in this study, it will contribute to autoaggregation and adhesion to the polystyrene surface.

Scanning electron microscope for the surface change [9,31,50,54], biofilm formation on the polystyrene [9,32,55], gel-permeation chromatography for the size reduction [56], cross-polarization/magic angle spinning nuclear magnetic resonance spectroscopy for the structural change [31,34], and thermogravimetric analyzer coupled with a Fourier-transform infrared spectrometer for the mass change [9,22,34,57-59] were used for analyzing polystyrene degradation activity of microorganisms. In this study, the polystyrene decomposition ability of bacteria was confirmed by micro-sprayed polystyrene onto a solid mineral medium [40]. The small polystyrene particles increase the body surface area, making observing efficient degradation by microorganisms easier.

In further research, it is possible to propose a biochemical polystyrene degradation mechanism by analyzing the genomic sequence of each bacteria to identify genes that degrade polystyrene and comparing the identified polystyrene degradation genes between bacteria to derive common points. Although the bacteria identified in this study are symbiotic microorganisms in the intestinal tract of mealworms, studying the identified bacteria may provide a clue as to what environment and conditions should be created to promote the biodegradation polystyrene in nature.

\section*{5. Conclusions}

The symbiotic microorganisms were isolated from the intestine of the mealworms grown using expanded polystyrene as a sole nutrient, and 97 bacteria were isolated through enrichment culture for fifty-six days with polystyrene powder. Among them, twelve bacteria were selected, and the polystyrene decomposition ability was evaluated using micro-sprayed polystyrene as a sole nutrient to observe the growth, decomposition of micro-polystyrene particles, and degradation on the surface of polystyrene films. Ten bacteria, P. multiresinivorans MA11, Y. regensburgei MA19, A. septicus MA21, S. marcescens MA42, A. septicus MA62, A. septicus MA65, K. grimonii MA76, P. nitroreducens MA77, A. tumefaciens MA92, and P. plecoglossicida MA93, degraded polystyrene. By analyzing and comparing the genomic DNA of the isolated bacteria to identify the standard polystyrene degrading genes, a biological mechanism for polystyrene degradation could be proposed in further studies.

\section*{Additional information}

Supplementary content related to this article has been publish online at [URL].

\section*{Conflicts of interest/Competing interests}

The authors declare no competing interest.

\section*{Ethics approval and consent to participate}

Not applicable.

\section*{Availability of data and materials}

The datasets used and/or analyzed during the current study are available from the corresponding author on reasonable request.

\section*{Authors' contributions}

JB and TJK conceptualized the study. JWP, MK, SYK, and TJK designed the experiments. JWP, MK, SYK, and JB performed the experiments. JWP, MK, SYK, and TJK analyzed the data. JWP, MK, and TJ wrote the manuscript. All authors read and approved the final manuscript.

\section*{Authors' information}

None.

\section*{Declaration of competing interest}

The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.

# Acknowledgements

This study was carried out with the support of the “R&D Program for Forest Science Technology (Project No. 2019150B10-2323-0301)” provided by the Korea Forest Service (Korea Forestry Promotion Institute).

# Appendix A. Supplementary data

Supplementary data related to this article can be found at https://doi.org/10.1016/j.heliyon.2023.e17352.

# References

[1] A.K. Panda, R.K. Singh, D.K. Mishra, Thermolysis of waste plastics to liquid fuel: a suitable method for plastic waste management and manufacture of value added products-A world prospective, Renew. Sustain. Energy Rev. 14 (1) (2010) 233–248.

[2] E. Kosior, J. Mitchell, Chapter 6 - current industry position on plastic production and recycling, in: T.M. Letcher (Ed.), Plastic Waste and Recycling, Academic Press, 2020, pp. 133–162.

[3] C.M. Rochman, M.A. Browne, B.S. Halpern, B.T. Hentschel, E. Hoh, H.K. Karapanagioti, L.M. Rios-Mendoza, H. Takada, S. Teh, R.C. Thompson, Classify plastic waste as hazardous, Nature 494 (2013) 169–171.

[4] H. Zhang, S. Pap, M.A. Taggart, K.G. Boyd, N.A. James, S.W. Gibb, A review of the potential utilisation of plastic waste as adsorbent for removal of hazardous priority contaminants from aqueous environments, Environ. Pollut. 258 (2020), 113698.

[5] R. Geyer, J.R. Jambeck, K.L. Law, Production, use, and fate of all plastics ever made, Sci. Adv. 3 (7) (2017), e170078.

[6] PlasticEurope, Plastics – the Facts 2015: an Analysis of European Plastics Production, Demand and Waste Data, Association of Plastics Manufacturers, 2015.

[7] Y. Chae, S.H. Hong, Y.-J. An, Photosynthesis enhancement in four marine microalgal species exposed to expanded polystyrene leachate, Ecotoxicol. Environ. Saf. 189 (2020), 109936.

[8] Y. Xu, L. Jiang, X. Ju, Y. Li, Mechanical properties of expanded polystyrene lightweight aggregate concrete and brick, Construct. Mater. 27 (1) (2012) 32–38.

[9] C. Ebciba, A. Gnanamani, Detailed studies on microbial adhesion and degradation of polystyrene foam wastes (PSFW) for clean environment, Environ. Sci. Pollut. Res. 27 (35) (2020) 44257–44266.

[10] J. Purohit, A. Chattopadhyay, B. Teli, Metagenomic exploration of plastic degrading microbes for biotechnological application, Curr. Genom. 21 (4) (2020) 253–270.

[11] G. Lear, J.M. Kingsbury, S. Franchini, V. Gambarini, S.D.M. Maday, J.A. Wallbank, L. Weaver, O. Pantos, Plastics and the microbiome: impacts and solutions, Environ. Microbiome 16 (1) (2021) 2.

[12] M.C. Krueger, H. Harms, D. Schlosser, Prospects for microbiological solutions to environmental pollution with plastics, Appl. Microbiol. Biotechnol. 99 (21) (2015) 8857–8874.

[13] L. Hou, D. Kumar, C.G. Yoo, I. Gitsov, E.L.W. Majumder, Conversion and removal strategies for microplastics in wastewater treatment plants and landfills, Chem. Eng. J. 406 (2021), 126715.

[14] B.T. Ho, T.K. Roberts, S. Lucas, An overview on biodegradation of polystyrene and modified polystyrene: the microbial approach, Crit. Rev. Biotechnol. 38 (2) (2018) 308–320.

[15] G. Baggi, M.M. Boga, D. Catelani, E. Galli, V. Treccani, Styrene catabolism by a strain of Pseudomonas fluorescens, Syst. Appl. Microbiol. 4 (1) (1983) 141–147.

[16] K. Lee, D.T. Gibson, Stereospecific dihydroxylation of the styrene vinyl group by purified naphthalene dioxygenase from Pseudomonas sp. strain NCIB 9816-4, J. Bacteriol. 178 (11) (1996) 3353–3356.

[17] S. Hartmans, J.P. Smits, M.J. van der Werf, F. Volkerking, J.A. de Bont, Metabolism of styrene oxide and 2-phenylethanol in the styrene-degrading Xanthobacter strain 124X, Appl. Environ. Microbiol. 55 (11) (1989) 2850–2855.

[18] A.M. Warhurst, K.F. Clarke, R.A. Hill, R.A. Holt, C.A. Fewson, Metabolism of styrene by Rhodococcus rhodochrous NCIMB 13259, Appl. Environ. Microbiol. 60 (4) (1994) 1137–1145.

[19] J.H. Jiang, M. Hirai, M. Shoda, Styrene degradation by Pseudomonas sp. SR-5 in biofilters with organic and inorganic packing materials, Appl. Environ. Microbiol. 65 (3) (2004) 349–355.

[20] S.-S. Yang, A.M. Brandon, A.J.C. Flanagan, J. Yang, D. Ning, S.-Y. Cai, H.-Q. Fan, Z.-Y. Wang, J. Ren, E. Benbow, N.-Q. Ren, R.M. Waymouth, J. Zhou, C. S. Criddle, W.-M. Wu, Biodegradation of polystyrene wastes in yellow mealworms (larvae of Tenebrio molitor Linnaeus): factors affecting biodegradation rates and the ability of polystyrene-fed larvae to complete their life cycle, Chemosphere 191 (2018) 979–989.

[21] S.-S. Yang, W.-M. Wu, A.M. Brandon, H.-Q. Fan, J.P. Receveur, Y. Li, Z.-Y. Wang, R. Fan, R.L. McClellan, S.-H. Gao, D. Ning, D.H. Phillips, B.-Y. Peng, H. Wang, S.-Y. Cai, P. Li, W.-W. Cai, L.-Y. Ding, J. Yang, M. Zheng, J. Ren, Y.-L. Zhang, J. Gao, D. Xing, N.-Q. Ren, R.M. Waymouth, J. Zhou, H.-C. Tao, C.J. Picard, M. E. Benbow, C.S. Criddle, Ubiquity of polystyrene digestion and biodegradation within yellow mealworms, larvae of Tenebrio molitor Linnaeus (Coleoptera: Tenebrionidae), Chemosphere 212 (2018) 262–271.

[22] B.-Y. Peng, Y. Su, Z. Chen, J. Chen, X. Zhou, M.E. Benbow, C.S. Criddle, W.-M. Wu, Y. Zhang, Biodegradation of polystyrene by dark (Tenebrio obscurus) and yellow (Tenebrio molitor) mealworms (Coleoptera: Tenebrionidae), Environ. Sci. Technol. 53 (9

[33] ASTM International, Standard practice for determining resistance of plastics to bacteria (withdrawn 2002). vol ASTM G22-76 (1996).

[34] Y. Yang, J. Yang, W.-M. Wu, J. Zhao, Y. Song, L. Gao, R. Yang, L. Jiang, Biodegradation and mineralization of polystyrene by plastic-eating mealworms: Part 1. Chemical and physical characterization and isotopic tests, Environ. Sci. Technol. 49 (20) (2015) 12080–12086.

[35] K. Tamura, J. Dudley, M. Nei, S. Kumar, MEGA4: molecular evolutionary genetics analysis (MEGA) software version 4.0, Mol. Biol. Evol. 24 (8) (2007) 1596–1599.

[36] A.M. Brandon, S.-H. Gao, R. Tian, D. Ning, S.-S. Yang, J. Zhou, W.-M. Wu, C.S. Criddle, Biodegradation of polyethylene and plastic mixtures in mealworms (larvae of Tenebrio molitor) and effects on the gut microbiome, Environ. Sci. Technol. 52 (11) (2018) 6526–6533.

[37] L. Luo, Y. Wang, H. Guo, Y. Yang, N. Qi, X. Zhao, S. Gao, A. Zhou, Biodegradation of foam plastics by Zophobas atratus larvae (Coleoptera: Tenebrionidae) associated with changes of gut digestive enzymes activities and microbiome, Chemosphere 282 (2021), 131006.

[38] A. Conter, C. Gangneux, M. Suzanne, C. Gutierrez, Survival of Escherichia coli during long-term starvation: effects of aeration, NaCl, and the rpoS and osmC gene products, Res. Microbiol. 152 (1) (2001) 17–26.

[39] T. Volke-Sepulveda, G. Saucedo-Castañeda, M. Gutiérrez-Rojas, A. Manzur, E. Favela-Torres, Thermally treated low density polyethylene biodegradation by Penicillium pinophilum and Aspergillus niger, J. Appl. Polym. Sci. 83 (2) (2002) 305–314.

[40] G. Shin, S.-A. Park, J.M. Koo, M. Kim, M. Lee, J. Jegal, S.Y. Hwang, J. Park, D.X. Oh, A micro-spray-based high-throughput screening system for bioplastic-degrading microorganisms, Green Chem. 23 (15) (2021) 5429–5436.

[41] S. Huang, P. Sheng, H. Zhang, Isolation and identification of cellulolytic bacteria from the gut of Holotrichia parallela larvae (Coleoptera: Scarabaeidae), Int. J. Mol. Sci. 13 (3) (2012) 2563–2577.

[42] R. Tuanudom, N. Yurayart, C. Rodkhum, S. Tiawsirisup, Diversity of midgut microbiota in laboratory-colonized and field-collected Aedes albopictus (Diptera: Culicidae): a preliminary study, Heliyon 7 (10) (2021), e08259.

[43] M. Shelomi, I.R. Sitepu, K.L. Boundy-Mills, L.S. Kimsey, Review of the gross anatomy and microbiology of the phasmatodea digestive tract, J. Orthoptera Res. 24 (1) (2015) 29–40.

[44] J. Bae, H. Jung, J. Park, S. Yun, S. Ha, Y. Lee, T.-J. Kim, Changes in intestinal microbiota due to the expanded polystyrene diet of mealworms (Tenebrio molitor), Indian J. Microbiol. 61 (2) (2021) 130–136.

[45] Z. Wang, X. Xin, X. Shi, Y. Zhang, A polystyrene-degrading Acinetobacter bacterium isolated from the larvae of Tribolium castaneum, Sci. Total Environ. 726 (2020), 138564.

[46] C.-F. Yin, Y. Xu, N.-Y. Zhou, Biodegradation of polyethylene mulching films by a co-culture of Acinetobacter sp. strain NyZ450 and Bacillus sp. strain NyZ451 isolated from Tenebrio molitor larvae, Int. Biodeterior. Biodegrad. 155 (2020), 105089.

[47] S.W. Przemieniecki, A. Kosewska, S. Giesielska, O. Kosewska, Changes in the gut microbiome and enzymatic profile of Tenebrio molitor larvae biodegrading cellulose, polyethylene and polystyrene waste, Environ. Pollut. 256 (2020), 113265.

[48] V.C. Sekhar, K.M. Nampoothiri, A.J. Mohan, N.R. Nair, T. Bhaskar, A. Pandey, Microbial degradation of high impact polystyrene (HIPS), an e-plastic with decabromodiphenyl oxide and antimony trioxide, J. Hazard Mater. 318 (2016) 347–354.

[49] E. Puglisi, F. Romaniello, S. Galletti, E. Boccaleri, A. Frache, P.S. Coconcelli, Selective bacterial colonization processes on polyethylene waste samples in an abandoned landfill site, Sci. Rep. 9 (1) (2019), 14138.

[50] S. Woo, I. Song, H.J. Gha, Fast and facile biodegradation of polystyrene by the gut microbial flora of Plesiophthalmus davidi larvae, Appl. Environ. Microbiol. 86 (18) (2020), e01361-01320.

[51] M. Ishikawa, H. Nakatani, K. Hori, AtaA, a new member of the trimeric autotransporter adhesins from Acinetobacter sp. Tol 5 mediating high adhesiveness to various abiotic surfaces, PLoS One 7 (11) (2012), e48830.

[52] K. Phuong, K. Kakii, T. Nikata, Intergeneric coaggregation of non-flocculating Acinetobacter spp. isolates with other sludge-constituting bacteria, J. Biosci. Bioeng. 107 (4) (2009) 394–400.

[53] D.S. Ahmed, A. Mohammed, A.A. Husain, G.A. El-Hiti, M. Kadhom, B.M. Kariuki, E. Yousif, Fabrication of highly photostable polystyrene films embedded with organometallic complexes, Polymers 14 (5) (2022) 1024.

[54] H.S. Mansoor, R. Alsayed, M. Abdallah, D.S. Ahmed, M. Buraarosha, N. Saliehe, E. Yousif, Polystyrene doped with Schiff base to reduce outdoor plastic degradation for sustainable environment, S. Afr. J. Chem. Eng. 35 (2021) 53–59.

[55] D. Chauhan, G. Agrawal, S. Deshmukh, S.S. Roy, P. Priyadarshini, Biofilm formation by Exiguobacterium sp. DR11 and DR14 alter polystyrene surface properties and initiate biodegradation, RSC Adv. 8 (66) (2018) 37590–37599.

[56] Y. Yang, J. Wang, M. Xia, Biode