# Animal Gene
## Bacterial and fungal diversity in the gut of polystyrene-fed Alphitobius diaperinus (Insecta: Coleoptera)
### Claudio Cucini$^{a,*}$, Chiara Leo$^{b}$, Matteo Vitale$^{b}$, Francesco Frati$^{a}$, Antonio Carapelli$^{a}$, Francesco Nardi$^{a}$
$^{a}$ Department of Life Sciences, University of Siena, 53100 Siena, Italy
$^{b}$ Department of Life Sciences, Imperial College London, London, UK

## Article Info
### Keywords:
Insect gut
Lesser mealworm
Metagenomics
Microbiota
Plastics
Polystyrene breakdown

### Abstract
The use of plastics burgeoned in the last decades to become an essential component of our society. An environment-friendly method to dispose of plastic waste is not available yet, to the outcome that these accumulate in landfills or are scattered as microplastics. New researches reported that some coleopteran species are able to destroy plastics thanks to their chewing mouthparts and the metabolic activity of their gut microbiota. This study shows that the lesser mealworm Alphitobius diaperinus is capable of feeding on, and apparently degrading, polystyrene. The gut microbiota of polystyrene-fed larvae was characterized using an NGS metagenomic approach, targeting both bacteria and fungi. Several microbe taxa emerged as differentially between treatment and control groups (Cronobacter, Kocuria and Pseudomonas as bacteria, Aspergillus, Hyphodermella, Trichoderma as fungi). Some of them have been found in association with plastic compounds and/or have been proposed to be capable of plastic degradation. This research supports the notion that, although synthetic molecules, unlike most natural compounds, do not generally enter the natural food chain to be degraded by the environmental microbiota, some microbial communities may be able to decompose plastics. We speculate that, once identified, such communities may open to the possibility of devising bioreactors for plastic degradation.

## Introduction
Since 1950 the yearly plastic demand has risen at untenable rates, from 2 to the 360 million of tons produced in 2018, with a trend that is not expected to level over time. Polystyrene ((C9H8)n) is one of the most commonly used plastic materials world-wide, the seventh in terms of quantity. Food packaging, building insulation and electrical equipment are the main applications of this product (Plastics Europe, 2019).

Although new technologies enabled the development of several recycling strategies, still an important percentage of plastics end up in landfills with no chance of being recycled or processed to produce green energy (Geyer et al., 2017). Furthermore, synthetic debris are causing important concerns such as environmental and water pollution, as well as the accumulation of microplastic that are jeopardizing animal and human wellness (i.e. cellular chemical toxicity, physical damage of tissues, modification of protein's secondary structure, etc.) (Barnes et al., 2009; Goldstein and Goodwin, 2013; Jambeck et al., 2015; Rochman et al., 2015; Hollóczki and Gehrke, 2019).

Unlike most natural compounds, plastics are synthetic materials of relatively recent origin, and are not generally deteriorated. Nevertheless, several authors have recently identified insect species that are able to feed on and, eventually, degrade various types of plastics (Riudavets et al., 2007; Yang et al., 2014; Bombelli et al., 2017; Yang et al., 2015a; Yang et al., 2015b). While some Lepidoptera species were identified that are able to degrade polyethylene, the most studied group in terms of polystyrene degradation are tenebrionids. Mealworms and superworms are known to degrade up to 0.12 mg/d and 0.58 mg/d of polystyrene respectively (Yang et al., 2015a; Yang et al., 2020). It was also demonstrated that the insect gut's microbial community plays a key role in the process, as supported by the observation that the suppression of microbial gut communities with antibiotics leads to a disruption of plastic degradation (Yang et al., 2015a; Yang et al., 2015b; Yang et al., 2020; Peng et al., 2019; Przemieniecki et al., 2020). Multiple bacterial species appear to be involved in the process. Yang and colleagues (Yang et al., 2015a; Yang et al., 2015b; Yang et al., 2020) have identified several bacterial species that are able to degrade polystyrene, including Pseudomonas, Aspergillus, and Trichoderma. These species are able to degrade polystyrene by producing enzymes that break down the polymer into smaller molecules that can be absorbed by the insect gut. The degradation process is facilitated by the presence of the gut microbiota, which provides the necessary enzymes and metabolic pathways for the degradation of the plastic. The degradation of polystyrene by these bacterial species is a promising approach for the biodegradation of plastic waste, as it could potentially reduce the accumulation of microplastics in the environment.

2015b) identified Exiguobacterium sp. as capable of creating biofilms and degrading PS in Tenebrio molitor's gut. Later works on the same tenebrionid beetle indicated other Enterobacteriaceae as possible candidates for plastic consumption (Peng et al., 2019; Brandon et al., 2018).

Furthermore, many free-living or insects' gut-associated bacteria have been identified as capable of the breakdown of synthetic products (Brandon et al., 2018; Urbanek et al., 2020; Mohan et al., 2016). Generally, Pseudomonadaceae and Enterobacteriaceae are reported to be the main actors in insect's gut, although it is still not clear which environmental conditions may promote polystyrene degradation.

Besides, some authors have described free-living fungi that appear to be efficient in synthetic polymers degradation (Yamada-Onodera et al., 2002; Krueger et al., 2015; Gajendiran et al., 2016). While the possible role of fungi in the breakdown of polystyrene inside insects' guts has never, to our knowledge, been investigated in detail, synergic effects arising from the joint metabolic activities of bacterial and fungal communities may play a key role in polystyrene biodegradation within insects' gut (Nam and Kukor, 2000).

We focused on the lesser mealworm, Alphitobius diaperinus (Insecta: Coleoptera), which is a well-known pest of stored food grain products. Despite previous works tried to highlight the bacterial and fungal pathogens within this species, focusing on its possible role as an alternative food-source, a similar survey was never conducted in relation to plastic degrading (Stoops et al., 2017; Wynants et al., 2018). We initially confirmed, in an experimental setting, that A. diaperinus is actually capable of feeding on, and apparently degrading, polystyrene. Then, to better understand which metabolic processes may be involved, we investigated both bacterial and fungal communities harboured inside its gut. Finally, we compared our results with those from previous studies on other Tenebrionidae and free-living organisms, trying to further highlight which bacterial and fungal forms may be responsible for polystyrene breakdown.

\section*{2. Materials and methods}

\subsection*{2.1. Lesser mealworm sources and feedstock}

Alphitobius diaperinus last instar larvae (approximately 7-10 mm in length) were purchased from Agripet Garden (Padua, Italy; reared according to the factory protocol, see: https://www.agripetgarden.it/). Larvae were separated into two groups. One group (treatment group, henceforth PS; 350 g of larvae, corresponding to \(\sim 17,500\) larvae) was reared on \(20 \mathrm{~g}\) of polystyrene (see below); the second (control group, henceforth CT; 150 g of larvae, corresponding to \(\sim 7500\) individuals) was reared on \(12 \mathrm{~g}\) of commercial organic carrots, renewed every 2-3 days, as a combined source of energy and water as in (Björge et al., 2018). Larvae were thus subdivided to account for a higher mortality in the PS group (observed in preliminary experiments) and maintain the two colonies at comparable numbers for the most part of the period (i.e. no transfer between groups was done). Both groups were reared using the commercial substrate as in (Björge et al., 2018).

Both colonies were reared at the Department of Life Sciences of the University of Siena under controlled conditions \(\left(20 \pm 2{ }^{\circ} \mathrm{C} ; 12 \mathrm{~L}: 12 \mathrm{D}\right.\) photo-period; \(50-70 \% \mathrm{RH})\) for 30 days. The expanded polystyrene foam used as feedstock for the PS group was purchased from Toscoespansi s.r. l. (https://www.toscoespansi.it/). This product, known under the commercial name of Extir® (CAS 9003-53-6), had a density of \(0.01 \mathrm{~g} / \mathrm{cm}^{3}\) and, according to the manufacturer information, contained no extra additives or catalysts.

\subsection*{2.2. Survival and polystyrene consumption}

At time 0 and after 30 days of rearing, the total weight of larvae, from both treated and control groups, was assessed.

At the end of the experiment, undegraded polystyrene fragments were similarly weighted. In order to separate these latter from contaminating dead larvae, exuviae and fecal pellets, polystyrene fragments were washed in a \(0.2 \times\) SDS \(10 \% / 1 \times\) Tween 20 \({ }^{\circledR}\) solution at pH 3.5 in a magnetic stirrer for 10-15 min. The polystyrene was then dried in absorbent paper and incubated overnight at \(59{ }^{\circ} \mathrm{C}\) to remove all remaining humidity and carefully observed under a stereomicroscope to confirm the absence of organic materials (e.g. exuviae). Undegraded substrate, ideally composed of not eaten PS debris and fecal pellets, was sieved to separate the two components. The resulting PS fraction was weighted and added to the above figure.

\subsection*{2.3. DNA extraction of gut microorganisms, markers amplification and sequencing}

At the end of the experiment, total DNA was extracted from A. diaperinus guts to characterize its microbial communities. Lesser mealworm larvae were collected and the whole intestines dissected under a Leica Wild M3C stereomicroscope using Petri dishes filled with sterilized paraffin as support. Total DNA was extracted using the QIAamp PowerFecal \({ }^{\circledR}\) DNA Kit according to the manufacturer's protocol. Extractions were made in triplicates from each group, i.e. three individual larvae were used to have triplicate observations from the PS group and three from the control group. DNA extractions were used for a metabarcoding analysis of the fungal and bacterial communities using the ITS1 and the 16S v3-v4 region as molecular marker, respectively (details in Protocol S1; Supplementary material).

\subsection*{2.4. Data analysis}

Raw sequences were demultiplexed based on their indices and primer sequences were removed. Quality trimming/filtering was performed in trimmomatic v.0.39 (Bolger et al., 2014) and then processed using the Quantitative Insights Into Microbial Ecology v.2 software (QIIME2 - v.2019.7) (Bolyen et al., 2019) as described in Protocol S2 (Supplementary material).

Diversity analyses were performed in QIIME2 and data for both markers were rarefied at the minimum sampling depth observed among samples prior to analyses. Good's coverage, Simpson and Shannon diversity indices ( \(\alpha\) diversity), were calculated for both markers, using the Kruskal-Wallis test to compare values across groups. Bray-Curtis Principal Coordinates Analysis (PCoA) was performed on both datasets. The phylogeny-based Unifrac distances (weighted and unweighted) PCoA were applied to the bacteria dataset only ( \(\beta\) diversity). QIME2 workflows are provided as Protocol S3 (bacteria) and Protocol S4 (fungi) (Supplementary material).

Differential abundance of OTUs was evaluated for both datasets (on the original non rarefied datasets) using the Phyloseq and DESeq2 packages in R v.3.6.1 (R Core Team 2018, http://www.R-project.org/) (Love et al., 2014; McMurdie and Holmes, 2013), and OTUs with an associated adjusted \(p\)-value (padj) \(<0.05\) were retained. The custom R script for this procedure is supplied as Protocol S5 of Supplementary material.

\section*{3. Results}

\subsection*{3.1. A. diaperinus survival rate and polystyrene weight loss}

Alphitobius diaperinus underwent a reduction of 77\% and 89\% of total weight in the CT and PS group, respectively. Reduction was more severe in the PS compared to the CT group (1.2×).

The weight of the polystyrene decreased during the experiment from 20 g (time zero) to 16 g (day 30) \((s=2.83)\). Henceforth, 4 g (20\%) of polystyrene was presumably degraded during 30 days of incubation.

\subsection*{3.2. Diversity of bacteria}



The taxa bar plot showed the presence of six predominant bacterial phyla, mostly with different relative frequencies: Proteobacteria (76% and 57% in PS and CT, respectively), Bacteroidetes (3% and 26%), Firmicutes (13% and 8%), Actinobacteria (4% and 8%), Fusobacteria (3% and \(<1 \%)\) and an unassigned phylum (<1% in both groups) (Fig. 1a).

Clustering at the level of classes identified eight predominant entities, often represented with different relative frequencies: Gammaproteobacteria (76% and 57%), Bacteroidia (3% and 25%), Bacilli (13% and 7%), Actinobacteria (4% and 8%), Fusobacteria (3% and 1%), Alphaproteobacteria (<1% and 1%), Erysipelotrichia (<1% both) and an unassigned class (<1% both) (Fig. 1b).

Shared OTUs count indicated that \(\sim 30 \%\) OTUs were shared among treatment and control groups, \(\sim 50 \%\) were observed in control only and \(\sim 20 \%\) in treatment only (Fig. 2).

The Good's coverage index, an estimate of the representativeness of the data, produced values higher than 0.99 for both PS and CT groups, thus indicating that the sequencing depth could be considered appropriate to provide an adequate estimation of the bacterial diversity in the samples. The Shannon and Simpson indices (Table 1) produced high values overall, with the only exception of one PS sample (library PS3). Based on the Simpson index, species richness appeared to be slightly lower in PS samples (0.43–0.87) compared to CT samples (0.88–0.90). The Shannon index provided comparable results, with PS samples displaying a more limited diversity (1.54–3.60) than CT samples (3.72–4.71). This disparity in richness between PS and CT groups emerged as statistically supported by the Kruskal-Wallis test (p-value \(<0.05)\).

Based on beta-diversity measures, PS and CT samples appeared in PCoA plots as separate clusters according to all estimators, nevertheless this difference was not recovered as statistically significant (p-value \(>0.05\) ) (Fig. 3a-c).

Four bacterial families appeared as differentially abundant between PS and CT samples (Fig. 4a). Pseudomonadaceae, Micrococcaceae, Bacillaceae and Enterobacteriaceae were recovered as significantly more abundant in PS samples than in CT samples, whereas all others appeared to be more abundant in CT (i.e. depleted in PS samples). At the genus level, one Gram-positive (i.e. Kocuria) and two Gram-negative bacteria (i.e. Cronobacter and Pseudomonas, with two species) were found to be significantly over-represented in PS samples, with a Log2-fold change between 4.1 and 6.5. Two Gram-positive bacteria (i.e. Pseudogracilicibacillus and Virgibacillus) were similarly over-represented in PS samples, with Log2-fold change of 4.8 and 5.8, respectively (Table 2). Noteworthy, two different OTUs from both these latter genera were also under-represented in PS samples (Log2-fold change of \(-7.4\) and \(-7.3\) respectively) (Fig. 4).

The resampling analysis (see Result R1, Supplementary materials), performed to support the identification of differentially abundant OTUs in PS specimens, showed that the two Pseudomonas species resulted by overrepresented in all resamples (100\%), whereas Pseudogracilicibacillus and Kocuria were recovered in 75–80\% of resamples. At variance, Cronobacter and Virgibacillus were recovered in a mere 25\% of resamples,

Fig. 3. PCoA based on β diversity indices. (a) bacteria, Bray-Curtis; (b) bacteria, unweighted Unifrac distances; (c) bacteria, weighted Unifrac distances; (d) fungi, Bray-Curtis. Black dots represent treated samples, white dots controls.

3.3. Diversity of fungi

The resulting sequencing data are available in Result R1 (Supplementary material).

The taxa bar plot showed that, at the phylum level, Ascomycota were predominant overall over Basidiomycota. On average, Ascomycota accounted for 70% of taxa in PS samples and 85% in CT samples (Fig. 5a). Basidiomycota were also well represented in PS samples (29% on average), although with large differences among replicates, while appearing to be utterly rare in CT samples, where a single library showed a mere 0.4%. A significant portion of detected fungal entities remained unidentified. These accounted for 14% on average in CT groups and as much as 39% in one single CT sample (CT1), while unidentified taxa were rare in PS samples (<1%).

Among these groups, six predominant classes were recorded (Agaricomycetes, Saccharomycetes, Sordariomycetes, Dothideomycetes, Eurotiomycetes, Leotiomycetes), plus one unidentified taxon (Fig. 5b). Good's coverage indices higher than 0.99 for both CT and PS samples indicated an adequate estimation of fungal diversity. Both the Shannon and Simpson indices were generally higher in PS samples (Simpson: 0.58–0.69 in PS, 0.19–0.63 in CT; Shannon: 1.65–2.72 in PS, 0.73–1.67 in CT), with the exception of one replicate (CT2) which had values in line with PS specimens (Table 1). The differences observed in both indices were nevertheless recovered as non-significant by the Kruskal-Wallis test (p > 0.05).

Shared OTUs count indicated that ~10% OTUs were shared among treatment and control groups, ~30% were observed in control only and ~60% in treatment only (Fig. 2).

Non-phylogenetic β diversity, as described by the Bray-Curtis PCoA, suggested a marked difference in OTUs presence and abundance between the CT and PS groups (Fig. 3d). Nevertheless, the β diversity measures resulted as non-significant in a PERMANOVA analysis (p-value > 0.05), most likely as an outcome of the small number of included samples.

The differential abundance analysis performed with DESeq2 identified five genera that appeared to be overrepresented in PS samples compared to CT samples: Trichoderma, Penicillium, Hypoderma, Aspergillus plus an unidentified genus belonging to the Agaricomycetes class, with a Log2-fold change between 11.8 and 18 (Fig. 6a; Table 2). Among them, only Aspergillus, Hypoderma and the unidentified agaricomycete were recovered with a sizeable frequency in resamples (80%), whereas Penicillium and Trichoderma were recovered rarely (25%), possibly questioning the significance of this result (Fig. 6b).


\section*{Table 2}

Species that appeared to be over-represented in PS vs. CT groups.

\begin{tabular}{llllll}
\hline Phylum & Class & Species & Log2FC & padj \\
\hline Bacteria & Proteobacteria & Gammaproteobacteria & Pseudomonas sp. & 6.5 & 2.31 \\
& & & & $\times$ & $10^{-3}$ \\
& Proteobacteria & Gammaproteobacteria & Pseudomonas sp. & 4.1 & 1.89 \\
& & & & $\times$ & $10^{-2}$ \\
& Proteobacteria & Gammaproteobacteria & Cronobacter sp. & 5.8 & 4.73 \\
& & & & $\times$ & $10^{-2}$ \\
Actinobacteria & Actinobacteria & Kocuria sp. & 4.2 & 2.48 \\
& & & & $\times$ & $10^{-2}$ \\
Bacilli & Bacillaceae & Virgibacillus sp. & 5.8 & 4.14 \\
& & & & $\times$ & $10^{-2}$ \\
Bacilli & Bacillaceae & Pseudogracilbacillus & 4.8 & 2.41 \\
& & & & $\times$ & $10^{-2}$ \\
Fungi & Ascomycota & Eurotiomycetes & Aspergillus sp. & 13.9 & 1.33 \\
& & & & $\times$ & $10^{-2}$ \\
& Ascomycota & Eurotiomycetes & Penicillium sp. & 16.3 & 2.54 \\
& & & & $\times$ & $10^{-3}$ \\
& Ascomycota & Agaricomycetes & Unidentified & 18.0 & 8.66 \\
& & & & $\times$ & $10^{-4}$ \\
& Ascomycota & Agaricomycetes & Hyphodermella sp. & 12.0 & 4.00 \\
& & & & $\times$ & $10^{-2}$ \\
& Ascomycota & Sordariomycetes & Trichoderma sp. & 11.8 & 4.00 \\
& & & & $\times$ & $10^{-2}$ \\
\hline
\end{tabular}4. Discussion

The fairly high reduction observed in larvae weight during the experiment may be dependent on the overall stressful rearing conditions, exacerbated in the PS group by the use of polystyrene as the only source of food (Bjorge et al., 2018). Nevertheless, both groups survived for the entire length of the experiment and, in the following weeks, several larvae underwent pupation and moult into adult forms (data not shown). However, further and more focused eco-physiological experiments are needed to evaluate the ability of A. diaperinus, alongside alternative plastic-degrading insects, to actually survive for more than one generation on a non-conventional food source such as polystyrene. The use of a combined source of energy and water in the PS treatment might improve the insect's physiological resistance to the stress caused by the use of an unnatural food source (Yang et al., 2018). This possibility may be considered in following experiments if the total PS degradation rate is to be maximised, but did not seem the primary choice in this initial phase where the differences between PS and control groups was to be maximised.

Alpha-diversity estimates (Table 1), as well as the count of shared OTUs, highlighted that the bacterial microbiota harboured in A. diaperinus' gut of PS samples is significantly impoverished compared to CT specimens (Figs. 1-2). This may be the outcome of the critical conditions encountered, that may not be viable for a portion of bacterial species naturally occurring in A. diaperinus' gut. Alternatively, it might be the result of the reliance on one single and simple source of carbon, that possibly favoured only one degradation pathway at the expense of others. Alpha-diversity estimates of the fungal microbiota, on the other hand, indicate that PS samples display a richer microbial diversity than CT (although differences between the two groups appear not to be statistically significant). This observation may be associated with the almost total absence, in CT samples, of Basidiomycota, a phylum of fungi that is is encountered at fairly high frequency in PS samples.

Apart from sheer species richness, some differences in the presence and relative abundance of individual taxa (i.e., \(\beta\) diversity) are highlighted by PCoA plots of PS compared to CT samples (Fig. 3), although these differences are not recovered as statistically supported. In fact, all distance measures produced plots where PS and CT samples are separated over at least one of the major axes. This difference is further supported by visual observation of the taxa bar plots (Fig. 1 for bacteria, 5 for fungi). In some instances, differences tend to emerge from the presence of specific taxa in some samples that are absent in others, such as, in the fungi dataset, Basidiomycota that are present at sizable frequency in at least 2 PS samples and nearly absent in all CT samples. Similarly, in the bacterial dataset, Bacteroidetes are well represented in CT samples and nearly absent in PS ones. In other instances, a large increase in relative frequency of a given taxon, as in the case of Proteobacteria in PS samples, produces a proportional shrinkage in the

\begin{figure}[h]
\centering
\includegraphics[width=0.8\textwidth]{figure5.png}
\caption{Fungal OTUs diversity. Taxa bar plots of: a) phyla and b) classes of fungi. Taxa are colour coded.}
\end{figure}

\begin{figure}[h]
\centering
\includegraphics[width=0.7\textwidth]{figure6.png}
\caption{Fungal OTUs differential abundance. (a) DESeq2 differential abundance of fungi clustered by genus. Taxa are colour coded. Species recovered as significantly more/less abundant in PS with respect to CT samples correspond to positive/negative of Log2-fold change, respectively. Asterisks indicate taxa that are observed at a frequency above 0.1\% in the original data. (b) DESeq2 differential abundance in resampled data set. Columns are colour coded according to the frequency a taxon is recovered significantly over/under expressed in the resamples. Red error bars indicate the range of Log2-fold changes in the resamples}
\label{fig:figure6}
\end{figure}

Given that the main focus of this study is the identification of microbial species that are associated with polystyrene-fed insects, and hence potentially involved in plastic degradation, a detailed OTUs differential abundance analysis was performed (Figs. 4, 6; Table 2). The bacteria genera Pseudomonas, Kocuria, Cronobacter, Pseudogracillibacillus and Virgibacillus were generally found to be significantly more abundant in PS compared to CT samples and henceforth possibly associated and/or involved in polystyrene degradation in the beetles' gut.

Pseudomonas has already been identified in the context of studies on plastic degradation and has been proposed to be strictly associated with polystyrene consumption (Urbanek et al., 2020; Mohan et al., 2016; Galgali et al., 2002; Atiq et al., 2010; Syranidou et al., 2017) as well as with the degradation of other plastic materials (Howard et al., 1999; Vega et al., 1999; Kyaw et al., 2012; Shimpi et al., 2012). Following from the observation that this mechanism of plastic degradation would require an initial breakdown of the polymer, previous studies tried to identify the enzyme classes that, based on their structure and functional parameters, may be responsible for the degradation of several different types of polystyrene and other synthetic polymers (Mohan et al., 2016; Shimpi et al., 2012; Ho et al., 2018; Yoon et al., 2012; Mooney et al., 2006). As an example, Azotobacter beijerinckii HM121 is known to produce a Hydroquinone Peroxidase capable of polystyrene degradation in a dichloromethane-water system (Nakamiya et al., 1997). In Pseudomonas the alkB gene was identified as the key enzyme in alkane biodegradation during polyethylene (PE) breakdown (Yoon et al., 2012). Furthermore, Mooney et al. (2006) reconstructed the entire process of styrene breakdown, identifying styrene monoxygenase as the first enzyme along the styrene degradation pathway, describing its subunit structure (styA and styB) and its regulation by styS and styR. In addition, an unknown type of esterase produced by Pseudomonas and Bacillus was associated with high impact polystyrene (HIPS) degradation (Mohan et al., 2016). Yet, Pseudomonas seems to be strictly associating with lesser mealworm larvae as assessed in previous metabarcoding researches on A. diaperinus, indicating the constant presence and the resilience of this microorganism even in different rearing conditions (Stoops et al., 2017; Wynants et al., 2018).

The genus Kocuria, found to be significantly overrepresented in the gut of polystyrene-fed larvae has never been recorded as associated with plastic consumption. Nevertheless, Parshetti et al. (2006) demonstrated that K. rosea is able to degrade malachite green, a chemical compound that is used as a textile dye or as an antimicrobial in aquacultures. Interestingly, malachite green shows a styrene core structure, a molecular substructure that is also present in polystyrene. It can therefore be envisioned that Kocuria may be able to produce enzymes that, using the styrene core as substrate, may be effective in the degradation of polystyrene.

The Gram-negative Cronobacter, similarly overrepresented in PS, is known to create biofilms on plastic surfaces, but it has never been recorded as involved in plastic breakdown (Torlak et al., 2015). However, Cronobacter is a member of the Enterobacteriaceae, a bacterial family that includes a number of species that were shown to degrade HIPS, PHA and EPS as free-living organisms (Mohan et al., 2016; Sekhar et al., 2016; Atiq et al., 2010; Volova et al., 2010) or as part of insects' gut microbiota (Yang et al., 2014; Peng et al., 2019; Brandon et al., 2018). The presence of similar enzymatic pathways, involved in polystyrene breakdown, is therefore possible.

Although bacteria are probably more abundant and diverse, they are not the only microbial component characterizing a particular environment. As such, since no previous work (to our knowledge) focused on insect's intestinal fungi as related to plastic consumption, we deem the inventory of the fungal community in A. diaperinus' gut of significant interest for the characterization of its microbiome at large. At variance with bacteria, \(\alpha\) diversity measures, alongside taxa bar plots (Fig. 5; Table 1), show a higher intrinsic variability of fungi in PS compared to CT samples. It has been shown that fungal species are able to produce a variety of readily used vitamins (Fraenkel and Blewett, 1943) amino acids and sterols (Noda and Koizumi, 2003; Gibson and Hunter, 2010). This capability may provide fungi with a substantial advantage under

stringent conditions, where a single and uniform source of carbon and no food source with a high-water content are available. It is therefore possible to speculate that the added nutritional role of these compounds may explain the higher abundance of fungi observed in the gut of polystyrene-fed larvae compared to control.

Five genera plus an unidentified form belonging to the Agaricomycetes class appear to be over-represented in PS compared to CT samples (Fig. 6; Table 2).

Agaricomycetes, especially abundant in PS samples, is a very diverse and species rich class (at least 21,000 species according to Kirk et al. (2008)) but no study seems to associate species of this group to plastic breakdown. No reference relevant to plastic breakdown is similarly available for Hyphodermella.

On the other hand, relevant information is available for Aspergillus, Trichoderma and Penicillium, three genera that have variously been associated with plastic degradation. All of them were found to be over-represented in PS samples. Some Aspergillus species were reported that are capable of degrading several plastic materials (Mathur et al., 2011; Raaman et al., 2012; Maeda et al., 2005). Trichoderma viride and Aspergillus nomius were both screened as soil free-living fungi able to attack and destroy LDPE (Munir et al., 2018). Yamada-Onodera and colleagues (Yamada-Onodera et al., 2002) described the capacity of Penicillium simplicissimum to break the C—C bond of polyethylene. More recently, Penicillium variabile was shown to mineralize polystyrene after long incubation (Tian et al., 2017). These two fungal species have been already described as associated with A. diaperinus in a previous study (Wynants et al., 2018) and may be part of the natural insect associated microbiota.

In full evidence, while association studies are definitely relevant to identify microbial species possibly associated with plastic degradation, these do not provide a proof of their functional role in the process. Culturing studies are underway in order to establish whether these selected microbial species are (or are not) culturable. Chemical and biochemical studies may be warranted in the near future, in line with previous works on Tenebrio and Zophobas atratus (Yang et al., 2015a; Yang et al., 2015b; Yang et al., 2020; Peng et al., 2019; Przemieniekiew et al., 2020; Brandon et al., 2018; Urbanek et al., 2020). Moreover, translational studies to deploy the microbial chain that naturally emerged as capable of plastic degradation in the insect's gut may be functional for the possible future development of artificial, sustainable and eco-friendly waste degradation technologies. In addition, due to the cannibalistic capability of tenebrionids in sub-optimal rearing condition, an improved rearing protocol should be adopted in further works to eliminate this further breeding variable. Lastly future studies should focus on other commensals or parasites that are known to be present in the Coleoptera's gut (i.e. gregarines (Nocciolini et al., 2018)).

\section*{5. Conclusion}

In this study we compared the gut microbial community of three A. diaperinus larvae reared on polystyrene with three control larvae fed on vegetable food. Several bacterial and fungal entities were significantly modulated in polystyrene fed animals. Some of these correspond to taxa already reported in this context in other organisms, some had never been reported but display biochemical features that suggest a possible role in this process. Future studies on A. diaperinus should focus on a chemical analysis of faeces to confirm plastic degradation and characterize the chemical reactions taking place. Moreover, since it is not known if these species are culturable, microbiological studies to assess the necessary culturing parameters are required in order to assess the possibility to culture these microbes in biochemical reactors aimed at degrading synthetic polymers.

\section*{Declaration of Competing Interest}

The authors declare that they have no conflict of interest.

\section*{Acknowledgments}

The authors appreciate the help provided from Toscoespansi s.r.l. (Loc. Cusona 53037 San Gimignano (Siena)) in the polystyrene concession and Alessandro Donati for providing computational resources for data analysis. The authors wish to thank Prof. Romano Dallai and Prof. Laura Marri for useful discussions. This work was partially supported by internal University of Siena program PSR.

\section*{Appendix A. Supplementary data}

Supplementary data to this article can be found online at https://doi.org/10.1016/j.ange.2020.200109.

\section*{References}

Aitq, N., Ahmed, S., Ali, M.I., Andleeb, S., Ahmad, B., 2010. Isolation and identification of polystyrene biodegrading bacteria from soil. Afr. J. Microbiol. Res. 4, 1537-1541.

Barnes, D.K.A., Galgani, F., Thompson, R.C., Barlaz, M., 2009. Accumulation and fragmentation of plastic debris in global environments. Philos. Trans. R. Soc. Lond. Ser. B Biol. Sci. 364, 1985-1998. https://doi.org/10.1098/rstb.2008.0205.

Bjorge, J.D., Overgaard, J., Malte, H., Gianotten, N., Heckmann, L.H., 2018. Role of temperature on growth and metabolic rate in the tenebrioid beetles Alphitobius diaperinus of the wax moth Galleria mellonella. J. Insect Physiol. 107, 89-96. https://doi.org/10.1016/j.jinsphys.2018.02.010.

Bolger, A.M., Lohse, M., Usadel, B., 2014. Trimomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30, 2114-2120. https://doi.org/10.1093/ bioinformatics/btu170.

Bolyen, E., Dillon, M., Bokulich, N., Abnet, C., Al-Ghailith, G., Alexander, H., Alm, E., Arumugam, M., Asnicar, F., Bai, Y., Biazen, J., Bittinger, K., Brejnrod, A., Brislawn, C., Brown, T., Callahan, B., Chase, J., Cope, E., Dorrestein, P., Douglas, G., Durall, D., Duvall, E., Edwards, C., Ernst, M., Estaki, M., Fouquier, J., Gauglitz, J., Gibson, D., Gonzalez, A., Gorlick, K., Guo, J., Hillmann, B., Holmes, S., Holste, H., Huttenhower, C., Hutley, G., Janssen, S., Jarmusch, A., Jiang, L., Kaehler, B., Keefe, C., Keim, P., Kelley, S., Knights, D., Koester, I., Koscielak, T., Kreps, J., Lee, J., Ley, R., Liu, Y.X., Loofffeld, E., Lozupone, C., Maher, M., Marotz, C., Martin, B., McDonald, D., McIver, L., Melnik, A., Metcalf, J., Morgan, S., Morton, J., Navas-Molina, J., Orchanian, S., Pearson, T., Peoples, S., Petras, D., Pruesse, E., Rivers, A., Robeson, M., Rosenthal, P., Segata, N., Shaffer, M., Shiffer, A., Sinha, R., Spear, J., Swafford, A., Thompson, L., Torres, P., Trinh, P., Tripathi, A., Turnbaugh, P., Ul-Hasan, S., Vargas, F., Vogtmann, E., Walters, W., Wan, Y., Wang, M., Warren, J., Weber, K., Willis, A., Zanevedel, J., Zhang, Y., Zhu, Q., Knight, R., Caporaso, G., 2019. QIIME 2: reproducible, interactive, scalable, and extensible microbiome data science. Nat. Biotechnol. 37, 852-857. https://doi.org/10.1038/s41587-019-0209-9.

Bomb

Jambeck, J.R., Geyer, R., Wilcox, C., Siegler, T.R., Perryman, M., Andrade, A., Narayan, R., Law, K.L., 2015. Plastic waste inputs from land to the ocean. Science 347, 768–771. https://doi.org/10.1126/science.1260352.

Kirk, P.M., Cannon, P.F., Minter, D.W., Stalpers, J.A., 2008. Ainsworth & Bisby's Dictionary of the Fungi, 10th ed. CABI International, p. 12.

Krueger, M.C., Hofmann, U., Moeder, M., Schlosser, D., 2015. Potential of wood-rotting fungi to attack polystyrene sulfonate and its depolymerisation by Gloeophyllum trabeum via hydroquinone-driven Fenton chemistry. PLoS One 10, 1–17. https://doi.org/10.1371/journal.pone.0131773.

Kyaw, B.M., Champakalakshmi, R., Sakharkar, M.K., Lim, C.S., Sakharkar, K.R., 2012. Biodegradation of low-density polythene (LDPE) by Pseudomonas species. Indian J. Microbiol. 52, 411–419. https://doi.org/10.1007/s12088-012-0250-6.

Love, M.I., Huber, W., Anders, S., 2014. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 1–21. https://doi.org/10.1186/s13059-014-0550-8.

Maeda, H., Yamagata, Y., Abe, K., Hasegawa, F., Machida, M., Ishioka, R., Gomi, K., Nakajima, K., 2005. Purification and characterization of a biodegradable plastic-degrading enzyme from Aspergillus oryzae. Appl. Microbiol. Biotechnol. 67, 778–788. https://doi.org/10.1007/s00253-004-1853-6.

Mathur, G., Mathur, A., Prasad, R., 2011. Colonization and degradation of thermally oxidized high-density polyethylene by Aspergillus niger (ITCC No. 6052) isolated from plastic waste dumpsite. Bioremediat. J. 15, 69–76. https://doi.org/10.1080/10898686.2011.570281.

McMurtrie, P.J., Holmes, S., 2013. Phyloseq: an R package for reproducible interactive analysis and graphics of microbiome census data. PLoS One 8, e61217. https://doi.org/10.1371/journal.pone.0061217.

Mohan, A.J., Sekhar, V.C., Bhaskar, T., Nampoorthiri, K.M., 2016. Microbial assisted high impact polystyrene (HIPS) degradation. Bioreosour. Technol. 213, 204–207. https://doi.org/10.1016/j.biortech.2016.03.021.

Mooney, K., Ward, P.G., O'Connor, K.E., 2006. Microbial degradation of styrene: biochemistry, molecular genetics, and perspectives for biotechnological applications. Appl. Microbiol. Biotechnol. 72, 1. https://doi.org/10.1007/s00253-006-0443-1.

Munir, E., Harefa, R.S.M., Priyani, N., Suryanto, D., 2018. Plastic degrading fungi Trichoderma viride and Aspergillus nigerius isolated from local landfill soil in Medan. IOP Conf. Ser. Earth Environ. Sci. 126, 6–13. https://doi.org/10.1088/1755-1315/126/1/012145.

Nakamiya, K., Sakasita, G., Ooi, T., Kinoshita, S., 1997. Enzymatic degradation of polystyrene by hydroquinone peroxidase of Azotobacter beijerinckii HM121. J. Ferment. Bioeng. 84, 480–482. https://doi.org/10.1016/S0922-338X(97)80213-2.

Nam, K., Kukor, J.J., 2000. Combined ozonation and biodegradation for remediation of mixtures of polymeric aromatic hydrocarbons in soil. Biodegradation 11, 1–9. https://doi.org/10.1023/A:1026592324693.

Nocciolini, C., Cucini, C., Leo, C., Francardi, V., Dresassi, E., Carapelli, A., 2018. Assessing the efficiency of molecular markers for the species identification of gregarious isolates from the mealworm and super worm midgit. Microorganisms 6, 119. https://doi.org/10.3390/microorganisms6040119.

Noda, H., Koizumi, Y., 2003. Sterol biosynthesis by symbiotes: cytochrome P450 sterol C-22 desaturase genes from yeastlike symbiotes of rice planthoppers and anobid beetles. Insect Biochem. Mol. Biol. 33, 649–658. https://doi.org/10.1016/S0965-1748(03)00056-0.

Parshetti, G., Kalme, S., Saratate, G., Govindwar, S., 2006. Biodegradation of malachite green by Kocuria rosea MTCC 1532. Acta Chim. Slov. 53, 492–498.

Peng, B.Y., Su, Y., Chen, Z., Chen, J., Zhou, X., Benbow, M.E., Criddle, C.S., Wu, W.M., Zhang, Y., 2019. Biodegradation of polystyrene by dark (Tenebrio obscura) and yellow (Tenebrio molitor) mealworms (Coleoptera: Tenebrionidae). Environ. Sci. Technol. 53, 5256–5265. https://doi.org/10.1021/acs.est.8b06963.

Plastics Europe, 2019. Plastics-the facts. In: An Analysis of European Plastics Production, Demand and Waste Data. www.plasticeurope.org/.

Przemieniecki, S.W., Kosewska, A., Ciesielski, S., Kosewska, O., 2020. Changes in the gut microbiome and enzymatic profile of Tenebrio molitor larvae biodegrading cellulose, polyethylene and polystyrene waste. Environ. Pollut. 256, 113265. https://doi.org/10.1016/j.envpol.2019.113265.

Raaman, N., Rajitha, N., Jayshree, A., Jegadeesh, R., 2012. Biodegradation of plastic by Aspergillus spp. isolated from polystyrene polluted sites around Chennai. J. Acad. Ind. Res. 1, 313–316.

Riudavets, J., Salas, I., Pons, M.J., 2007. Damage characteristics produced by insect pests in packaging film. J. Stored Prod. Res. 43, 564–570. https://doi.org/10.1016/j.jspr.2007.03