# Nitrogen Fixation and Diazotrophic Community in Plastic-Eating Mealworms Tenebrio molitor L

## Yu Yang1, Lin Hu1, Xiaoxi Li1,2, Jialei Wang1, Guishan Jin3

Received: 15 June 2021 / Accepted: 12 November 2021 / Published online: 21 January 2022

© The Author(s), under exclusive licence to Springer Science+Business Media, LLC, part of Springer Nature 2022

## Abstract

Mealworms, the larvae of a coleopteran insect \(Tenebrio molitor\) L., are capable of eating, living on, and degrading non-hydrolyzable vinyl plastics as sole diet. However, vinyl plastics are carbon-rich but nitrogen-deficient. It remains puzzling how plastic-eating mealworms overcome the nutritional obstacle of nitrogen limitation. Here, we provide the evidence for nitrogen fixation activity within plastic-eating mealworms. Acetylene reduction assays illustrate that the nitrogen-fixing activity ranges from \(12.3 \pm 0.7\) to \(32.9 \pm 9.3 \mathrm{nmol}\) ethylene-h \({ }^{-1} \cdot \mathrm{gut}^{-1}\) and the corresponding fixed nitrogen equivalents of protein are estimated as \(8.6\) to \(23.0 \mu \mathrm{g}\) per day per mealworm. Nature nitrogen isotopic analyses of plastic-eating mealworms provide further evidence for the assimilation of fixed nitrogen as a new nitrogen source. Eliminating the gut microbial microbiota with antibiotics impairs the mealworm's ability to fix nitrogen from the atmosphere, indicating the contribution of gut microbiota to nitrogen fixation. By using the traditional culture-dependent technique, PCR and RT-PCR of \(n i f H\) gene, nitrogen-fixing bacteria diversity within the gut was detected, and the genus Klebsiella was demonstrated to be an important nitrogen-fixing symbiont. These findings first build the relationship between plastic degradation (carbon metabolism) and nitrogen fixation (nitrogen metabolism) within mealworms. Combined with previously reported plastic-degrading capability and nitrogen-fixing activity, mealworms may be potential candidates for up-recycling of plastic waste to produce protein sources.

## Keywords

Nitrogen fixation · Plastic waste · Biodegradation · Mealworm · Klebsiella

## Introduction

An increasing accumulation of non-hydrolyzable vinyl plastic wastes, such as polystyrene (PS), polyethylene (PE), polypropylene (PP), and polyvinyl chloride (PVC), has become a severe environmental and social concern [43, 56]. In 2015, mealworm, the larva of a coleopteran insect \(Tenebrio molitor\) L., was first reported to be able to eat PS foam (Styrofoam) as sole diet and lived as well as those fed with a normal food (wheat bran) over a period of 1 month [57, 58]. Ingested Styrofoam was demonstrated to be efficiently degraded after passage through the larval gut within a retention time of less than \(24 \mathrm{~h}\) and further mineralized to \(\mathrm{CO}_{2}\) and incorporated into lipids [57]. Subsequently, the ability of mealworm to live on and degrade Styrofoam was ubiquitously confirmed by expanding assessments from 22 countries [55]. Additionally, multiple recent studies showed that mealworm was also capable of surviving by eating other synthetic vinyl plastics, such as PE, PVC, PP, and styrene-butadiene rubber (SBR), as sole diet, and rapidly degrading these ingested vinyl plastics within the larval gut [1, 7, 37, 54, 56]. These findings have provided implications for future applications of mealworms in plastic waste management. However, there is a controversy that vinyl plastics of PS, PE, PVC, PP, and SBR are carbon-rich but nitrogen-deficient diet, which could not meet the nitrogen requirements for the biosynthesis of proteins, nucleic acids, and other metabolites within mealworms. Therefore, it is intriguing to know how mealworms overcome the nutritional obstacle of nitrogen limitation when they are fed with vinyl plastic as sole diet.

Many insects, like termite, shipworm, stag beetle, and red turpentine beetle, intrinsically feed on nitrogen-deficient wood diet [4]. These xylophagous insects have been extensively demonstrated to be capable of supplementing their diets by fixing nitrogen from atmosphere based on the acetylene reduction assays or nitrogen isotopic tracers [3, 5, 6, 8, 11, 23, 27, 36, 40, 44, 47, 53]. The nitrogen-fixing activity of xylophagous insects could be abolished by feeding with antibiotics, indicating that gut bacteria may contribute to nitrogen fixation [6, 8, 11]. These findings promote a search for insect gut bacteria that are involved in nitrogen fixation [22, 30, 35]. By applying the traditional culture-dependent technique of growing the bacteria on a nitrogen-free medium, a few cultures of nitrogen-fixing bacteria, identified as the species from the genera of Azotobacter, Spirochaeta, Citrobacter, Enterobacter, Treponema, Klebsiella, Beijerinckia, Clostridium, and Raoultiella, have been isolated from the guts of various nitrogen-fixing insects [5, 9, 11, 15, 18, 20, 26, 28, 39, 42, 50]. Nevertheless, the culture-dependent method may detect only a limited number of diazotrophic species because of the inability to culture most of environmental isolates. As the gene nifH encoding dinitrogen reductase is genetically conserved [59], the culture-independent molecular techniques based on the amplification of nifH gene has been used to detect an unexpected diversity of nitrogen-fixing bacteria within guts of these nitrogen-fixing insects [3, 12, 14, 16, 33, 34, 52]. However, the successful amplifications of nifH genes do not always indicate that the encoding dinitrogen reductase is actually expressed by the corresponding bacteria, because the expressions of nifH genes are regulated at the transcriptional and post-transcriptional level [59]. Thus, detection of mRNA by a reverse transcription polymerase chain reaction (RT-PCR) is usually used to further support the expression of nifH genes and determine the actual activity of specific gut microbes in nitrogen fixation [3, 12, 14, 31, 32].

Like xylophagous insects, mealworm's gut also provides an ecological niche for the resident of a complex mixed microbial community [51, 58]. The composition of gut bacterial community within mealworms from different sources has been investigated using both culture-dependent and culture-independent methods [21, 25, 41, 45, 48, 49, 51, 58]. The culture-independent pyrosequencing-based analyses show that Proteobacteria, Bacteroidetes, Firmicutes, Actinobacteria, and Tenericutes are the dominant phylum [21, 25, 41, 45, 48, 49]. The detected species belong to phylum Proteobacteria or Firmicutes, mainly Citrobacter spp., Klebsiella spp., Enterobacter spp., Pantoea spp., Pseudomonas spp., or Clostridium spp., are known potential nitrogen-fixing bacteria [4]. These available data provide circumstantial evidence that nitrogen fixation may occur in mealworms. However, the direct evidence for the nitrogen-fixing activity of vinyl plastic-eating mealworm and the involvement of gut nitrogen-fixing bacteria is still lacking.

Therefore, the specific aims of this study are to (i) determine whether the gut bacteria-mediated nitrogen fixation takes place within mealworms when they were fed with Styrofoam as sole diet by using the acetylene reduction assays and stable nitrogen isotope composition analyses, (ii) to uncover the diazotrophic diversity of the gut microbial symbiont by culture-dependent technique and culture-independent amplification of nifH genes, and (iii) to illustrate whether the nifH genes encoding dinitrogen reductase are actually expressed by gut nitrogen-fixing bacteria. The overall aim is to extend our present understanding that plastic-eating mealworms can adapt the nitrogen-deficient plastic diet by employing the nitrogen-fixing capacity of gut microbiota.

\section*{Materials and Methods}

\section*{Collection and Culture of Mealworms}

Mealworms (growth age at approximately \(3-4\) instars) were purchased from six different insect-breeding plants in northern (Beijing), eastern (Shanghai and Ningbo), western (Chengdu), southern (Guangzhou), and central (Luoyang) region of China, respectively. The distributions of body weight and size of the sampled mealworms are shown in SI Tables S1. The conventional diet for these sampled mealworms was bran.

Sampled mealworms from each source were divided into two groups. One group (denoted as normal group, \(n=3\) ) were reared with Styrofoam blocks as a sole diet for 28 days. Other group (labeled as Antibiotic group, \(n=3\) ) were first fed with the antibiotic diet (60 mg gentamicin per 1 g bran) until the number of active gut bacteria became approximately zero. The number of active gut bacteria was detected by the series dilution method of plate counting on solid nitrogen-free plates (1.5\% agar) [10]. These antibiotic-treated mealworms were subsequently fed with Styrofoam for 28 days. Both of "normal" or "antibiotic" groups were used for further investigations.

\section*{Acetylene Reduction Assays}

Ten mealworms were randomly selected from each group and were quickly surface sterilized by immersion in 75\% ethanol for 1 min and then rinsed 2 times with sterile phosphate saline buffer (PBS) prior to gut excision. Their guts were steriley drawn out and pooled into a Hungate tube containing \(5 \mathrm{~mL}\) of sterile PBS buffer. Then, Hungate tubes were stoppered with rubber plugs, and the inside atmosphere was exchanged with argon gas (Ar). Acetylene was

Ten mealworms were randomly selected from each group and lyophilized, ground to a homogeneous powder. The powder (about 1.0 mg) was packed in tin capsules and combusted to CO<sub>2</sub> and N<sub>2</sub> at 1000 °C in an on-line combustor. Subsequently, N<sub>2</sub> was separated in a He carrier stream on a Carbosieve G column and further introduced to an isotopic ratio mass spectrometer (Finnigan MAT 253, Thermo Electron, Waltham, MA). Nitrogen isotope compositions were reported as δ<sup>15</sup>N values relative to atmospheric nitrogen (a pure N<sub>2</sub> gas reference, 0.00‰). The system was calibrated using multiple runs of a powdered working standard (ammonium sulfate and sucrose mixture with δ<sup>15</sup>N versus air of 1.33‰), and the detection method was periodically calibrated against international isotope standards (IAEA N1 and N3).

Isolation and Characterization of Nitrogen-Fixing Bacteria

Thirty mealworms from each group were randomly collected and quickly surface sterilized by immersion in 75% ethanol for 1 min and then rinsed twice with sterile PBS. Their guts were drawn out and pooled in a 10-mL centrifuge tube containing 5 mL sterile PBS. The solution was roiled on a vortex mixer for 5 min in order to adequately release the microbial contents from the guts. After that, the clearing gut tissues were carefully picked out with a pipette, while the solution containing gut contents was used as gut microbial inoculum, which was tenfold diluted and inoculated on the plates containing semi-solid nitrogen-free media (0.5% agar) [10]. These inoculated plates were incubated for 5 days at 30 °C. Individual colonies were randomly picked and streaked repeatedly on Luria–Bertani (LB) agar to obtain pure cultures. Genomic DNA extraction, PCR amplification, sequencing, and phylogenetic analysis of the 16S rRNA gene from these pure cultures were carried out according to the previously described method [51].

For acetylene reduction assays, pure cultures were inoculated in LB media and incubated in agitation at 30 °C for 24 h, with a well-known nitrogen-fixing bacterium Azospirillum brasilense was used as positive control [38]. The LB cultures were centrifuged at 4 °C, 12,000 rpm for 5 min, and then the bacterial pellets were collected and washed 3 times and re-suspended with sterile PBS. The OD<sub>600</sub> values of all bacterial suspensions were determined and adjusted to the same value (OD<sub>600</sub> = 1.0, 10×dilution). Then, 100 μL of each suspension was inoculated in 25 mL test tubes containing 15 mL of semi-solid nitrogen-free media. The test tubes stopped with cotton plugs were incubated for 72 h at 30 °C. Subsequently, the cotton plugs of test tubes were replaced with rubber plugs, and 1 mL of acetylene was injected into the tubes. Ethylene production was measured periodically by GC as described above.

PCR amplifications of the nitrogenase (nifH) gene (about 360 bp) from the genomic DNA of the nitrogen-fixing isolates were performed using a two-stage nest PCR procedure with a pair of universal primers (SI Table S2). Genomic DNA extraction of the pure cultures was carried out according to the previously described method [51]. In the first round, the genomic DNA extract (1 μL, 300 ng·μL<sup>-1</sup>), 1 μL of forward nifFo and 1 μL of reverse nifRe primer, was added to 11 μL of the PCR mixtures (2×M5 Taq HiFi PCR mix, Mei5 Biotechnology, Beijing). The final volume of PCR mixtures was 20 μL by adding double-distilled H<sub>2</sub>O (ddH<sub>2</sub>O). PCR was carried out with one denaturation step of 5 min at 95 °C, 40 cycles of 94 °C for 11 s, 92 °C for 15 s, 54 °C for 8 s, 56 °C for 30 s, 74 °C for 10 s, and 72 °C for 10 s and a final extension step of 4 min at 72 °C. In the second round, the amplification products of the first round (1 μL, 300 ng·μL<sup>-1</sup>), 1 μL of forward polF and 1 μL of reverse polR primer, were added to 11 μL of the PCR mixtures (2×M5 Taq HiFi PCR mix, Mei5 Biotechnology, Beijing). The final volume of PCR mixtures was 20 μL by adding double-distilled H<sub>2</sub>O (ddH<sub>2</sub>O). PCR was carried out with one denaturation step of 5 min at 95 °C, 40 cycles of 94 °C for 1 min, 55 °C for 1 min, and 72 °C for 30 s and a final extension step of 4 min at 72 °C. The final amplification products were analyzed by electrophoresis in 1.5% (w/v) agarose gels in 1×TAE buffer. PCR products of the expected size were gel-purified using a GeneJET Gel Extraction Kit (Thermo Scientific, USA). Nucleotide sequencing was done

Nitrogen Fixation and Diazotrophic Community in Plastic-Eating Mealworms Tenebrio molitor... 267 using ABI 3730XL genetic analyzer (Applied Biosystems, CA, USA).

DNA Extraction, PCR Amplification, Cloning, and Sequencing of nifH Genes

Fifty active mealworms were randomly picked out from each group and were surface sterilized with the same method described above. Their guts were drawn out and pooled into a 50 mL centrifuge tube containing 25 mL sterile PBS on ice, respectively. After being roiled on a vortex mixer for 5 min, the microbial contents were released from the guts, and the empty guts were carefully removed from the tube by using a pipette. The solution containing gut microbial contents was further centrifuged at 4 °C, 12,000 rpm for 5 min. The supernatants were discarded, and the precipitates were used for the DNA extraction by using the TIANamp Stool DNA Kit (DP 328, TIANGEN, Beijing, China) according to the manufacturer's instructions. The average concentration of yielded genomic DNA was determined by NanoDrop One Microvolume UV-Vis Spectrophotometer (Thermo Scientific, USA).

An internal fragment of the nifH gene (about 360 bp) was amplified with using each set of previously reported primer pairs (Table S2). The genomic DNA extract (1 μL, 300 ng·μL−1), 1 μL of forward and 1 μL of reverse primer, was added to 11 μL of the PCR mixtures (2×M5 Taq HiFi PCR mix, Mei5 Biotechnology, Beijing). The final volume of PCR mixtures was 20 μL by adding double-distilled H2O (ddH2O). Negative controls without DNA were also included to test for contamination. PCR was carried out with one denaturation step of 8 min at 95 °C, 35 cycles of 94 °C for 0.5 min, 47 °C for 0.5 min, and 72 °C for 1 min and a final extension step of 7 min at 72 °C. Amplification products were analyzed by electrophoresis in 1.5% (w/v) agarose gels in 1×TAE buffer. PCR products of the expected size were gel-purified using a GeneJET Gel Extraction Kit (Thermo Scientific, USA). PCR products were ligated into pJET1.2/ blunt cloning vector and transformed into Escherichia coli XL-10-Gold cells using the CloneJET PCR Cloning Kit (Thermo Scientific, USA). Plasmid DNA was isolated from individual clones and purified using TIANprep Mini Plasmid Kit (DP103, Tiangen, Beijing, China). Nucleotide sequencing was done using ABI 3730XL genetic analyzer (Applied Biosystems, CA, USA).

RNA Extraction and RT-PCR

Gut microbial contents were prepared with the same method as described above for DNA extraction. Total RNA was extracted from gut contents by using TRIzol™ Reagent (Invitrogen, USA). RNA samples were purified using RapidOut DNA Removal Kit (Thermo Scientific, USA) to remove DNA. Reverse transcription reactions were performed using GoScript™ Reverse Transcription System (Promega, USA). After reverse transcription, nifH genes were amplified in cDNA samples using PCR experimental conditions as described in the previous section. Each set of previously reported primer pairs was tested in the PCR reaction (Table S2). RNA samples without reverse transcription step were used as controls to check for residual DNA in the RNA preparations. RT-PCR and PCR negative controls were also included to test for reagent contamination. The generation of clone libraries and sequencing of RT-PCR amplified nifH products were performed by the same method described in the above section.

Phylogenetic Analysis of nifH Clones

All detected nifH gene nucleotide sequences were translated into amino acid sequences by using ExPASy bioinformatics resource tool [19]. Amino acid sequence with at least one amino acid change was considered as a unique operational taxonomic units (OTUs). The closest phylogenetic neighbors were obtained by aligning the representative amino acid sequence of each OTU against NCBI database using protein Basic Local Assignment Search Tool (BLASTp) [2]. Sequence alignment of all amino acid sequences was performed by using clustal×2.0 software [24]. Aligned sequences were used to reconstruct the maximum-likelihood (ML) phylogenetic trees using the program MEGA 6.0 [17, 46], with bootstrap values based on 1000 replications.

Re-infection Treatment

For the re-infection treatment, the nitrogen-fixing isolate was grown in liquid nutrient broth (NB) medium for 12 h, respectively. The cells were collected via centrifugation (10,000 rpm) and rinsed with sterile salt water to remove the residual medium. Next, all collected cells were re-suspended in sterile salt water to obtain a mixed cell suspension of approximately 108 cells per mL. New sterile bran diet was prepared and supplemented with bacterial cell suspension with a ratio of 1:1 (w/v) and then was dried at 35 °C for 1 h. Antibiotic-treated mealworms were fed with the bacteria-inoculated diet for 10 days. Then, the re-infected mealworms as well as normal and antibiotic-treated mealworms (300 as a group, n = 3) were fed with Styrofoam blocks as sole diet for 16 days. The survival rate (SR) and the average dry body weight (ADBW) of each group were determined.

Statistical Analysis

Statistical analyses were performed in Excel (version 2016). To assess differences in acetylene-reducing activity and δ15N values between three groups (normal, antibiotic,

# Results

## Nitrogen-Fixing Activity of Styrofoam-Eating Mealworm

We used acetylene reduction assay to examine the nitrogen-fixing activity of mealworm, because both acetylene and nitrogen are unsaturated compounds with covalent triple bonds and the conversion of acetylene to ethylene has been widely used as an indicator of nitrogen-fixing activity for both terrestrial and aquatic systems [29]. As shown in Fig. 1, ethylene production from acetylene was detected by Styrofoam-eating mealworms (normal group) from six different sources, with the rate from 12.3 ± 0.7 to 32.9 ± 9.3 nmol ethylene-h⁻¹·gut⁻¹. The acetylene-reducing (nitrogen-fixing) activity of Styrofoam-eating mealworm was similar to that of individual field-collected Mediterranean fruit fly (18.0 ± 1.5 nmol ethylene-h⁻¹·gut⁻¹) [5] but was one or two orders of magnitude higher than that of termite (1.25 ± 0.37 nmol acetylene-h⁻¹·gut⁻¹) and stag beetle (0.03 nmol acetylene-h⁻¹·gut⁻¹) [8, 23]. When mealworms were fed with antibiotic diet (60 mg gentamicin per g bran), all detectable acetylene-reducing (nitrogen-fixing) activity decreased to the range of 8.7 ± 0.5 to 19.0 ± 0.4 nmol ethylene-h⁻¹·gut⁻¹ (Fig. 1), correlating with the suppression of gut bacterial numbers detected by the serial dilution method of plate counting (SI Fig. S1). Statistical analyses with a two-way ANOVA (SI Table S3–S4) showed that the effect of antibiotic treatment was significant ($F$ = 17.05 > $F_{crit}$, $p$ = 0.009 < 0.05), while the effect of tested mealworms population was non-significant ($F$ = 3.33 < $F_{crit}$, $p$ = 0.11 > 0.05) on the acetylene-reducing (nitrogen-fixing) activity. These results indicated that nitrogen fixation really occurred in Styrofoam-eating mealworms and nitrogen-fixing bacteria were present within their guts.

Moreover, we also determined the acetylene-reducing (nitrogen-fixing) activity of mealworms fed with the conventional diet of bran (rich in carbohydrates and proteins). The bran-feeding mealworms had an acetylene-reducing (nitrogen-fixing) activity of 8.1 ± 2.1 to 12.2 ± 0.3 nmol ethylene-h⁻¹·gut⁻¹, which was significantly lower than that of Styrofoam-eating mealworms (two-way ANOVA, $F$ = 13.76 > $F_{crit}$, $p$ = 0.014 < 0.05) (SI Table S5–S6), agreeing with the previously known phenomenon that the available nitrogen in the diet would repress the nitrogenase synthesis of nitrogen-fixing systems [8].

## Assimilation of Fixed Nitrogen into Styrofoam-Eating Mealworm's Tissues

According to the acetylene-reducing rate, the fixed nitrogen equivalents of protein were estimated as 8.6 to 23.0 μg per day per Styrofoam-eating mealworm using the previous reported method [5]. The investigation of natural stable nitrogen isotope compositions in organisms also can provide


Further evidence for the incorporation of fixed nitrogen as a source of new nitrogen, because the $\delta^{15}$N value of atmospheric N<sub>2</sub> is 0‰ and nitrogen-fixing organisms often have $\delta^{15}$N close to 0‰ [47]. Thus, we analyzed the $\delta^{15}$N values of Styrofoam-eating mealworms, antibiotic-treated mealworms, and bran-feeding mealworms that were sampled from six different sources.

As seen in Fig. 2, all Styrofoam-eating mealworms had $\delta^{15}$N values between 1.7 and 1.9‰, which were significantly lower than that of antibiotic-treated mealworms (2.3–2.4‰) (two-way ANOVA, $F$ = 160.0 > $F_{crit}$, $p$ = 0.00005 < 0.05) (SI Table S7–S8). Moreover, the $\delta^{15}$N values of bran-feeding mealworms were also analyzed and found to be in the range of 1.9 to 2.1‰, which were significantly higher than that of Styrofoam-eating mealworm (two-way ANOVA, $F$ = 19.62 > $F_{crit}$, $p$ = 0.0068 < 0.05) (SI Table S9–S10). These results were consistent with the decreased acetylene-reducing (nitrogen-fixing) activities within antibiotic-treated and bran-feeding mealworms in comparison to Styrofoam-eating mealworms. The low $\delta^{15}$N values of Styrofoam-eating mealworms could be attributed to the presence of gut nitrogen-fixing bacteria that have fix more nitrogen from atmosphere nitrogen and provided a substantial new nitrogen source for the mealworm's metabolism.

\section*{Nitrogen-Fixing Bacterial Cultures Isolated from Mealworm's Gut}

To assess the level of potential nitrogen-fixing bacteria in gut microbiota of Styrofoam-eating mealworms, gut contents from Styrofoam-eating mealworms were serially diluted and inoculated in a semi-solid nitrogen-free enrichment medium for the growth of nitrogen-fixing bacteria. The typical colonies of nitrogen-fixing bacteria came out on all plates inoculated with gut contents of Styrofoam-eating mealworms from six different sources (SI Fig. S1). The number of potential nitrogen-fixing bacteria was estimated at \(5.5 \times 10^{4} \mathrm{CFU}\) per gut. In contrast, the size of the total cultivable fraction on the a general LB medium, inoculated with the same dilution series, was estimated to be \(5.0 \times 10^{6} \mathrm{CFU}\) per gut [58]. Therefore, the level of functional nitrogen fixers was estimated at about 1.1\% of the total bacterial population within the gut of Styrofoam-eating mealworm.

To identify the specific gut microbes responsible to nitrogen fixation, we conducted extensive streaking isolations using nitrogen-free medium. Colonies isolated from the nitrogen-free plates were identified by amplification and sequencing of the almost full-length 16S rRNA gene. About 110 pure colonies were obtained, but some were duplicates based on their similar colony morphologies and identical 16S rRNA sequences. After removing duplicates, there were only 24 distinct isolates, and their acetylene-reducing abilities were determined. As shown in Fig. 3a, four isolates, stains BIT-1, BIT-56, BIT-96, and BIT-105, were able to reduce acetylene into ethylene. In comparison to a positive control of a known free-living diazotrophs Azospirillum brasilense ATCC 29,145, stain BIT-96 and BIT-56 showed a comparable nitrogen-fixing rate (t test, \(p>0.05\) ), while stains BIT-1 and BIT-105 had a relative lower rate (t test, \(p<0.05\) ). To further ascertain the nitrogen-fixing ability of these isolates, PCR amplifications of \(n i f H\) gene from the genomic DNA of these isolates were performed using a

\section*{Fig. 2 \(\delta^{15} \mathrm{~N}\) values [N isotopic ratio, per mil (\%) ] of normal Styrofoam-eating mealworms and antibiotic-treated mealworms from six different sources. All results are shown as mean \pm standard deviation (SD) \((n=3\) samples, 10 worms as a sample). Statistical analyses using a two-way ANOVA were performed to evaluate the effects of the tested mealworm population and antibiotic treatment on the \(\delta^{15} \mathrm{~N}\) values of mealworms (Table S7-S8). The results showed a significant main effect for antibiotic treatment \(\left(F=160.0>F_{\text {crit }}\right.\) \(p=0.0005<0.05)\) but not for the tested mealworm population \(\left(F=0.52<F_{\text {crit }}, p=0.76>0.05\right)\)}

(a)
\begin{figure}[h]
\centering
\includegraphics[width=0.45\textwidth]{fig3a.png}
\caption{Nitrogen-fixing bacteria isolated from the mealworm's gut. a Nitrogen-fixing activity measured by acetylene reduction assays for bacterial isolates obtained from the gut of mealworms. Azospirillum brasilense ATCC 29,145, a known free-living nitrogen-fixing bacterium, was used as a positive control. Results are shown as mean ± standard deviation (SD) (n = 3). The t test with Tukey's correction was used to assess differences in acetylene-reducing activity between nitrogen-fixing bacteria. b PCR products of nifH gene from the genomic DNA of potential nitrogen-fixing gut isolates. c The 16S rRNA gene-based maximum-likelihood (ML) phylogenetic tree of nitrogen-fixing bacteria isolated from the gut of plastic-eating mealworms. The 16S rRNA sequence of Pseudomonas aeruginosa DSMZ 50,071 was used as outgroup. Scale bar indicates 2% estimated sequence divergence}
\label{fig:fig3a}
\end{figure}


(a)
\begin{tabular}{cccccc}
M & Beijing & Chengdu & Ningbo & Shanghai & M \\
500 bp & & & & & \\
250 bp & & & & & \\
\end{tabular}

(b)
\begin{tabular}{c}
\begin{tabular}{c}
Cluster I \\
\end{tabular} \\
\hline
\end{tabular}

Expression of nifH Genes in Styrofoam-Eating Mealworm's Gut

To demonstrate whether the nifH genes encoding dinitrogenase reductase were actually expressed by gut nitrogen-fixing bacteria within Styrofoam-eating mealworms, transcribed nifH mRNA in the gut microbial communities was analyzed using RT-PCR. Amplifications were detected in the cDNA samples of gut contents from six different Styrofoam-eating mealworms but not in the control RNA sample of gut contents without the RT reaction step (Fig. 5), indicating that the RNA preparation did not have any residual DNA after the DNase treatment and nifH transcripts virtually presented in the total RNA pool extracted from Styrofoam-eating mealworms' guts. In comparison, no amplifications were observed in the cDNA samples of gut contents from antibiotic-treated and bran-feeding mealworms (SI Fig. S2), in line with their relative weaker acetylene-reducing (nitrogen-fixing) activities than Styrofoam-eating mealworms.

Clone libraries were constructed from nifH cDNA amplified by RT-PCR from total RNA. We recovered two unique representatives from the RT-PCR cDNA clone libraries. Both of them clustered within group 1 of cluster I and were exclusively Klebsiella phylotypes (Fig. 4b and SI Table S12). Given the high nitrogen fixation rates measured for the Klebsiella isolates BIT-96 (Fig. 3a) and the apparent consistence of Klebsiella nifH genes in PCR and RT-PCR clone libraries (Figs. 4 and 5), it is likely that the genus Klebsiella, belonging to the order of Enterobacteriales, is an important nitrogen-fixing symbiont of Styrofoam-eating mealworms.

\section*{Fig. 5 RT-PCR amplification of nifH gene mRNA from the gut microbiota of plastic-eating mealworms. The nifH-specific primer PolF-PolR was used in the PCR reaction. RNA preparations treated with DNase in the absence of RT did not show the presence of PCR products, indicating there are no DNA contaminant in the RNA extracts}


# Impact of Nitrogen-Fixing Gut Bacteria on Styrofoam-Eating Mealworm's Fitness

To further examine the influence of nitrogen-fixing bacteria on mealworms' fitness, we re-infected the antibiotic-treated mealworms with nitrogen-fixing gut isolate K. spallanzanii BIT-96 and subsequently compared their survival rates (SR) and the average dry body weights (ADBW) with that of Styrofoam-eating mealworms (normal group) and antibiotic-treated mealworms. As shown in Fig. 6a, the SRs of all three groups were not significantly different during a 16-day incubation with Styrofoam as sole diet (one-way ANOVA, \( p = 0.89 > 0.1 \)). By contrast, the ADBW of antibiotic-treated groups was significantly lighter than that of re-infected groups (\( t \) test, \( p < 0.01 \)), even though the ADBW of re-infected groups was significantly lighter than that of normal groups (\( t \) test, \( p < 0.01 \)) (Fig. 6b). As antibiotic treatment may eliminate both of nitrogen-fixing gut bacteria and other functional bacteria, the highest ADBW loss of antibiotic-treated mealworms may be resulted from the damage of nitrogen-fixing capability and other functions. Re-infection with nitrogen-fixing gut isolate K. spallanzanii BIT-96 may help relieve the partial weight loss of antibiotic-treated mealworms by the restoration of the nitrogen-fixing capability. Hence, we could infer that the presence of nitrogen-fixing gut bacteria was important to the biological performance of Styrofoam-eating mealworms.

\section*{Discussion}

Since mealworm was first introduced as a plastic eater and digester in 2015, this coleopteran insect larva has been proven to survive by eating a series of non-hydrolyzable vinyl plastics, such as PS, PE, PP, and PVC, as sole diet and effectively degrade them [1, 7, 37, 54, 56, 57]. In the perspective of nutrient element content, the petroleum-derived vinyl plastics were carbon-rich but nitrogen-deficient diet. However, from a biology viewpoint, nitrogen was an indispensable element for the biosynthesis of proteins, nucleic acids, and other metabolites in insect [4]. Hence, there was a pending question how mealworms can survive on nitrogen-deficient vinyl plastic as sole diet as well as that feed on normal nutritionally adequate diet (such as bran). Inspired by that the xylophagous insects were capable of supplementing their diets by fixing nitrogen from atmosphere, we attempted to reveal whether mealworms could also fix atmosphere nitrogen.

In the present study, we have provided comprehensive evidence for the occurrence of nitrogen fixation in Styrofoam-eating mealworms from three aspects proposed by Bar-Shmuel et al. [4]. Firstly, our results explicitly demonstrated that Styrofoam-eating mealworm has nitrogen-fixing activity by using the acetylene reduction assays. Secondly, the analyses of natural stable nitrogen isotope compositions in Styrofoam-eating mealworm has provided evidence for the assimilation of atmosphere nitrogen into the biomass. The fixed nitrogen equivalents of protein were estimated as 8.6 to 23.0 \(\mu \mathrm{g}\) per day per mealworm based on the

\section*{Fig. 6 Influence of nitrogen-fixing ability on the biological performance of mealworms. a Survival rate (SR), b the average dry body weight (ADBW). The one-way ANOVA and \( t\) test with Tukey's correction was used to assess differences in SR and ADBW between the three groups, respectively}

\[
\text { (a) }
\]

\[
\text { (b) }
\]


The four identified nitrogen-fixing gut bacteria isolates from mealworm were found from the genera of Klebsiella, Mixta, Kluyvera, and Citrobacter within the order of Enterobacteriales. The detected eight distinct nifH gene clones from DNA or cDNA of gut microbiota had the greatest sequence similarity to the genera Klebsiella and Pseudomonas and the nifH gene from uncultured nitrogen-fixing bacteria in soil. These nitrogen-fixing bacterial species found in the gut of mealworms were also retrieved from the guts of other nitrogen-fixing insects, such as wood-eating termites [15, 18, 39], the fruit fly Ceratitis capitata [5], and the red turpentine beetle Dendroctonus valens LeConte [28]. All this suggests that many nitrogen-fixing insects may share the similar gut nitrogen-fixing bacteria regardless of the specific host.

Given that the gut microbiota of mealworms have been demonstrated to be responsible for the plastic degradation in our previous studies [58] and nitrogen fixation within mealworm in the present study, the bacteria from the gut of mealworms possess two properties, plastic degradation and nitrogen fixation. Hence, engineering a synthetic microbial consortium or a combined cell factory, which can degrade plastic waste and fix atmosphere nitrogen at the same time, would be a possible way for up-recycling plastic waste to cell proteins without the additional feed of nitrogen nutrient.

The online version contains supplementary material available at https://doi.org/10.1007/s00248-021-01930-5.

HL, LX, and WJ conducted acetylene reduction assays, isolation of gut nitrogen-fixing bacteria, and PCR and RT-PCR experiments. JG conducted bulk stable isotope measurements on mealworms. YY and HL analyzed data and wrote the manuscript. YY conceived and designed the research.

This work was supported by grants from the National Natural Science Foundation of China (No. 51603004 and No. 31961133015), the Young Elite Scientist Sponsorship Program of the China Association of Science and Technology (No. 2017QNRC001), and the Beijing Institute of Technology Research Fund Program for Young Scholars (No. 3160011181804).

The GenBank accession numbers for 16S rRNA of strains Kluyvera sp. BIT-1, Mixta sp. BIT-56, Klebsiella sp. BIT-96, and Citrobacter sp. BIT-105 were MW422732, MW422733, MW422734, and MW422751, respectively. All detected nifH gene nucleotide sequences were deposited in GenBank with the accession numbers MW442362-MW442370 and OK631825-OK631827.

This article does not contain any studies with human participants or animals performed by any of the authors. All authors read and approved the manuscript.

The authors declare no competing interests.

1. Aboelkheir MG, Visconte LY, Oliveira GE, Toledo Filho RD, Souza FG (2019) The biodegradative effect of Tenebrio molitor Linnaeus larvae on vulcanized SBR and tire crumb. Sci Total Environ 649:1075–1082
2. Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ (1990) Basic local alignment search tool. J Mol Biol 215:403–410
3. Ayayee P, Rosa C, Ferry JG, Felton G, Saunders M, Hoover K (2014) Gut microbes contribute to nitrogen provisioning in a wood-feeding cerambid. Environ Entomol 43:903–912
4. Bar-Shmuel N, Behar A, Segoli M (2020) What do we know about biological nitrogen fixation in insects? Evidence and implications for the insect and the ecosystem. Insect Sci 27(3):392–403
5. Behar A, Yuval B, Jurkevitch E (2005) Enterobacteria-mediated nitrogen fixation in natural populations of the fruit fly Ceratitis capitata. Mol Ecol 14(9):2637–2643
6. Benemann JR (1973) Nitrogen fixation in termites. Science 181:164–165
7. Brandon AM, Gao SH, Tian R, Ning D, Yang SS, Zhou J, Wu WM, Criddle CS (2018) Biodegradation of polyethylene and plastic mixtures in mealworms (larvae of Tenebrio molitor) and effects on the gut microbiome. Environ Sci Technol 52:6526–6533
8. Breznak JA, Brill WJ, Mertins JW, Coppel HC (1973) Nitrogen fixation in termites. Nature 244:577–580
9. Bridges JR (1981) Nitrogen-fixing bacteria associated with bark beetles. Microb Ecol 7:131–137
10. Burdman S, Jurkevitch E, Schwartzburd B, Hampel M, Okon Y (1998) Aggregation in Azospirillum brasilense: effects of chemical and physical factors and involvement of extracellular components. Microbiology 144(7):1989–1999
11. Carpenter EJ, Culliney JL (1975) Nitrogen fixation in marine shipworms. Science 187:551–552
12. Ceja-Navarro JA, Nguyen NH, Karaoz U, Gross SR, Herman DJ, Andersen GL, Bruns TD, Pett-Ridge J, Blackwell M, Brodie EL (2014) Compartmentalized microbial composition, oxygen gradients and nitrogen fixation in the gut of Odontotaenius disjunctus. ISME J 8:6–18
13. Chun J, Oren A, Ventosa A, Christensen H, Arahal DR, da Costa MS, Rooney AP, Yi H, Xu XW, De Meyer S, Trujillo ME (2018) Proposed minimal standards for the use of genome data for the taxonomy of prokaryotes. Int J Syst Evol Microbiol 68:461–466
14. Desai MS, Brune A (2012) Bacteroidales endosymbionts shape the nitrogen-fixing community in dry-wood termites. ISME J 6:1302–1313
15. Doolittle M, Raina A, Lax A, Boopathy R (2008) Presence of nitrogen fixing Klebsiella pneumoniae in the gut of the Formosan

16. Du X, Li X, Wang Y, Peng J, Hong H, Yang H (2012) Phylogenetic diversity of nitrogen fixation genes in the intestinal tract of Reticulitermes chinensis Snyder. Curr Microbiol 65:547–551
17. Felsenstein J (1981) Evolutionary trees from DNA sequences: a maximum likelihood approach. J Mol Evol 17:368–376
18. French JRJ, Turner GL, Bradbury JB (1976) Nitrogen fixation by bacteria from the hindgut of termites. Microbiology 95:202–206
19. Gasteiger E, Gattiker A, Hoogland C, Ivanyi I, Appel RD, Bairoch A (2003) ExPASy: the proteomics server for in-depth protein knowledge and analysis. Nucleic Acids Res 31:3784–3788
20. Gomathi V, Ramalakshmi A, Ramasamy K (2005) Isolation of nitrogen fixing bacteria from fungus termites. Entomol Res 35:75–78
21. Jung J, Heo A, Park YW, Kim YJ, Koh H, Park W (2014) Gut microbiota of Tenebrio molitor and their response to environmental change. J Microbiol Biotechnol 24(7):888–897
22. Kneip C, Lockhart P, Voss C, Maier UG (2007) Nitrogen fixation in eukaryotes–new models for symbiosis. BMC Evol Biol 7(1):55
23. Kuranouchi T, Nakamura T, Shimamura S, Kojima H, Goka K, Okabe K, Mochizuki A (2006) Nitrogen fixation in the stag beetle, Dorcus (Macro dor cus) rectus (Motschulsky) (Col., Lucanidae). J Appl Entomol 130(9–10):471–472
24. Larkin MA, Blackshields G, Brown NP, Chenna R, McGettigan PA, McWilliam H, Valentin F, Wallace IM, Wilm A, Lopez R, Thompson JD, Gibson TJ, Higgins DG (2007) Clustal W and Clustal X version 2.0. Bioinformatics 23:2947–2948
25. Li L, Xie B, Dong C (2016) Can closed artificial ecosystem have an impact on insect microbial community? A case study of yellow mealworm (Tenebrio molitor L.). Ecol Engng 86:183–189
26. Lilburn TG, Kim KS, Ostrom NE, Byzek KR, Leadbetter JR, Breznak JA (2001) Nitrogen fixation by symbiotic and free-living sprochetes. Science 292:2495–2498
27. Morales-Jiménez J, Zúñiga G, Villa-Tanaca L, Hernández-Rodríguez C (2009) Bacterial community and nitrogen fixation in the red turpentine beetle, Dendroctonus valens LeConte (Coleoptera: Curculionidae: Scolytinae). Microb Ecol 58(4):879–891
28. Morales-Jiménez J, Vera-Ponce de León A, García-Domínguez A, Martínez-Romero E, Zúñiga G, Hernández-Rodríguez C (2013) Nitrogen-fixing and uricolytic bacteria associated with the gut of Dendroctonus rhi zophagus and Dendroctonus valens (Curculionidae: Scolytinae). Microb Ecol 66:200–210
29. Myrold DD (2007) Quantification of nitrogen transformations. In Manual of Environmental Microbiology, ASM 3:687–696
30. Nardi JB, Mackie RI, Dawson JO (2002) Could microbial symbionts of arthropod guts contribute significantly to nitrogen fixation in terrestrial ecosystems? J Insect Physiol 48(8):751–763
31. Noda S, Ohkuma M, Usami R, Horikoshi K, Kudo T (1999) Culture-independent characterization of a gene responsible for nitrogen fixation in the symbiotic microbial community in the gut of the termite Neotermes koshunensis. Appl Environ Microbiol 65:4935–4942
32. Noda S, Ohkuma M, Kudo T (2002) Nitrogen fixation genes expressed in the symbiotic microbial community in the gut of the termite Coptotermes formosanus. Microbes Environ 17:139–143
33. Ohkuma M, Noda S, Usami R, Horikoshi K, Kudo T (1996) Diversity of nitrogen fixation genes in the symbiotic intestinal microflora of the termite Reticulitermes speratus. Appl Environ Microbiol 62:2747–2752
34. Ohkuma M, Noda S, Kudo T (1999) Phylogenetic diversity of nitrogen fixation genes in the symbiotic microbial community in the gut of diverse termites. Appl Environ Microbiol 65:4926–4934
35. orona Tamayo D, Heil M (2015) N fixation in insects: its potential contribution to N cycling in ecosystems and insect biomass. In: de Bruijn FJ (ed) Biological Nitrogen Fixation. John Wiley & Sons Inc, Hoboken, pp 1141–1148
36. Pandey S, Waller DA, Gordon AS (1992) Variation in acetylene-reduction (nitrogen-fixation) rates in Reticulitermes spp. (Isoptera; Rhinotermitidae). Va J Sci 43(3):333–338
37. Peng BY, Chen Z, Chen J, Yu H, Zhou X, Criddle CS, Wu WM, Zhang Y (2020) Biodegradation of polyvinyl chloride (PVC) in Tenebrio molitor (Coleoptera: Tenebrionidae) larvae. Environ Int 145:106106
38. Pinto-Tomás AA, Anderson MA, Suen G, Stevenson DM, Chu FS, Clandaw WW, Weimer PJ, Currie CR (2009) Symbiotic nitrogen fixation in the fungus gardens of leaf-cutter ants. Science 326:1120–1123
39. Potrikus CJ, Breznak JA (1977) Nitrogen-fixing Enterobacter agglomerans isolated from guts of wood-eating termites. Appl Environ Microbiol 33:392–399
40. Prestwich GD, Bentley BL (1981) Nitrogen fixation by intact colonies of the termite Nasutitermes corniger. Oecologia 49(2):249–251
41. Przemieniecki SW, Kosewska A, Ciesielski S, Kosewska O (2020) Changes in the gut microbiome and enzymatic profile of Tenebrio molitor larvae biodegrading cellulose, polyethylene and polystyrene waste. Environ Pollut 256:113265
42. Rísal CP, Ozawa T (2002) Isolation and characterization of diazotrophs from the intestinal tract of an earthworm (Phoretina vitata). Soil Sci Plant Nutr 48:101–103
43. Ru JK, Huo YX, Yang Y (2020) Microbial degradation and valorization of plastic wastes. Front Microbiol 11:422
44. Sapountzis P, de Verges J, Rousk K, Cilliers M, Vorster BJ, Poulsen M (2016) Potential for nitrogen fixation in the fungus-growing termite symbiosis. Front Microbiol 7:1993
45. Stoops J, Crauwels S, Waud M, Claes J, Lievens B, Van Campenhout L (2016) Microbial community assessment of mealworm larvae (Tenebrio molitor) and grasshoppers (Locusta migratoria migr

55. Yang SS, Wu WM, Brandon AM, Fan HQ, Receveur JP, Li Y, Wang ZY, Fan R, McClellan RL, Gao SH, Ning D, Phillips DH, Peng BY, Wang H, Cai SY, Li P, Cai WW, Ding LY, Yang J, Zheng M, Ren J, Zhang YL, Gao J, Xing D, Ren NQ, Waymouth RM, Zhou J, Tao HC, Picard CJ, Benbow ME, Criddle CS (2018) Ubiquity of polystyrene digestion and biodegradation within yellow mealworms, larvae of Tenebrio molitor Linnaeus (Coleoptera: Tenebrionidae). Chemosphere 212:262–271

56. Yang Y, Wang J, Xia M (2020) Biodegradation and mineralization of polystyrene by plastic-eating superworms Zophobas atratus. Sci Total Environ 708:135233

57. Yang Y, Yang J, Wu WM, Zhao J, Song Y, Gao L, Yang R, Jiang L (2015) Biodegradation and mineralization of polystyrene by plastic-eating mealworms. 1. chemical and physical characterization and isotopic tests. Environ Sci Technol 49:12080–12086

58. Yang Y, Yang J, Wu WM, Zhao J, Song Y, Gao L, Yang R, Jiang L (2015) Biodegradation and mineralization of polystyrene by plastic-eating mealworms. 2. role of gut microorganisms. Environ Sci Technol 49:12087–12093

59. Zehr JP, Jenkins BD, Short SM, Steward GF (2003) Nitrogenase gene diversity and microbial community structure: a cross system comparison. Environ Microbiol 5:539–554